{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 88,
   "id": "fa998a5a",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import json\n",
    "import random"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 89,
   "id": "c22546b8",
   "metadata": {},
   "outputs": [],
   "source": [
    "success_df = pd.read_pickle(\"merged.df\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 90,
   "id": "f289dae8",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import uuid\n",
    "\n",
    "# Assuming the DataFrame is already defined as df\n",
    "\n",
    "# Get unique genotypes from the 'genotype' column\n",
    "unique_genotypes = success_df['genotype'].explode().unique().tolist()\n",
    "\n",
    "# Create a dictionary to store the genotype ID pairs\n",
    "genotype_id_dict = {}\n",
    "\n",
    "# Assign a unique ID to each genotype\n",
    "for genotype in unique_genotypes:\n",
    "    genotype_id = str(uuid.uuid4())\n",
    "    genotype_id_dict[genotype] = genotype_id"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 91,
   "id": "bb3db9c5",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'WT': [(151, 153), (612, 614)], 'KO': [(193, 195), (638, 640)]}"
      ]
     },
     "execution_count": 91,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "success_df.at[0, 'index_dict']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 92,
   "id": "53d27f99",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import json\n",
    "import random\n",
    "\n",
    "# Assuming the DataFrame is already defined as df\n",
    "\n",
    "# Generate unique IDs for each row\n",
    "success_df['id'] = [str(random.randint(100000, 999999)) for _ in range(len(success_df))]\n",
    "\n",
    "# Create a new DataFrame with the desired columns\n",
    "new_df = success_df[['id', 'Project ID', 'Project Name', 'Title', 'Description', 'genotype', 'index_dict']].copy()\n",
    "\n",
    "# Rename the columns to match the desired JSON format\n",
    "new_df.columns = ['id', 'project_id', 'project_name', 'project_title', 'content', 'genotype', 'annotations']\n",
    "\n",
    "# Convert the genotype column to a list of dictionaries\n",
    "new_df['genotype'] = new_df['genotype'].apply(lambda x: [{'id': genotype_id_dict[g], 'value': g} for g in x])\n",
    "\n",
    "# Convert the index_dict column to a list of dictionaries\n",
    "new_df['annotations'] = new_df['annotations'].apply(lambda x: [{'start': g, 'end': a, 'tag_name': 'GENOTYPE', 'value': k} for k, v in x.items() for g, a in v if v])\n",
    "\n",
    "# Convert the DataFrame to JSON\n",
    "json_data = new_df.to_json(orient='records')\n",
    "\n",
    "# Parse the JSON string\n",
    "parsed_data = json.loads(json_data)\n",
    "\n",
    "# Create a dictionary with the 'data' key and the parsed data\n",
    "result = {'data': parsed_data}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 93,
   "id": "49429143",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'id': '265085',\n",
       " 'project_id': 'PRJNA986078',\n",
       " 'project_name': 'Evi1 governs Kdm6b-mediated histone demethylation to finely regulate the Laptm4b-driven mTORC pathway in vivo [LK]',\n",
       " 'project_title': 'Evi1 governs Kdm6b-mediated histone demethylation to finely regulate the Laptm4b-driven mTORC pathway in vivo [LK]',\n",
       " 'content': 'Ecotropic viral integration site 1 (EVI1/MECOM) overexpression is common in myeloid malignancies. We present a new Evi1 transgenic mouse model with inducible expression in hematopoietic stem cells (HSCs) and progenitor cells (HPCs) at lower levels. Upon exogenous Evi1 induction, mice displayed anemia, thrombocytopenia, lymphopenia, and dysplasia in erythroid and megakaryocyte cells with a significant expansion of committed myeloid progenitor cells, resembling human Myelodysplastic syndrome/Myeloproliferative neoplasm (MDS/MPN)-like disease. Methods: Lin-C-Kit+ cells were isolated from 3 pairs of WT and Evi1 overexpressing mice. Around 1X106 Lin-C-Kit+ cells were harvested from each mouse. Then the cells were lysated by Trizol and total RNA was extracted by phenol-chloroform. Results: Molecular pathways altered in Lin-C-Kit+ cells from Evi1-OE mice. Conclusions: Multiple molecular pathways changed in Evi1 overexpressing hematopoietic stem and progenitor cells. Overall design: 3 pairs of WT and Evi1 overexpressing mice at the same age and gender were sacrificed and the BM cells were harvested.',\n",
       " 'genotype': [{'id': 'd5ffd8e8-0229-469b-ba10-1462f309718b', 'value': 'WT'},\n",
       "  {'id': '7fb070ca-b785-4d70-a797-36163abf09a5',\n",
       "   'value': 'Evi1 overexpressing'}],\n",
       " 'annotations': [{'start': 603,\n",
       "   'end': 605,\n",
       "   'tag_name': 'GENOTYPE',\n",
       "   'value': 'WT'},\n",
       "  {'start': 1001, 'end': 1003, 'tag_name': 'GENOTYPE', 'value': 'WT'},\n",
       "  {'start': 610,\n",
       "   'end': 629,\n",
       "   'tag_name': 'GENOTYPE',\n",
       "   'value': 'Evi1 overexpressing'},\n",
       "  {'start': 913,\n",
       "   'end': 932,\n",
       "   'tag_name': 'GENOTYPE',\n",
       "   'value': 'Evi1 overexpressing'},\n",
       "  {'start': 1008,\n",
       "   'end': 1027,\n",
       "   'tag_name': 'GENOTYPE',\n",
       "   'value': 'Evi1 overexpressing'}]}"
      ]
     },
     "execution_count": 93,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "result['data'][2]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 94,
   "id": "1b48424c",
   "metadata": {},
   "outputs": [],
   "source": [
    "with open('data.json', 'w') as f:\n",
    "    json.dump(result, f)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 95,
   "id": "dd10ec00",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'id': '944377', 'project_id': 'PRJNA986528', 'project_name': 'PIKfyve controls dendritic cell function and tumor immunity I', 'project_title': 'PIKfyve controls dendritic cell function and tumor immunity I', 'content': 'To understand the mechanism of Pikfyve loss in dendritic cells, we cultured conventional dendritic cells isolated from the bone marrow of Pikfyvef/f (“WT”) and mice with ItgaxTg/0 Pikfyvef/f (“KO”) mice and performed transcriptome profiling on RNA extracted from cells on Day 9 in culture. Taken together, these data demonstrate the PIKfyve mediates suppression of DC transcriptional maturation programs through the alternate/non-canonical NF-κB regulatory pathway. Overall design: Comparative gene expression profiling of RNA-seq data from primary cultured conventional dendritic cells isolated from wild-type (WT) or Pikfyve-deficient (KO) mice (littermate pair-matched biological triplicates).', 'genotype': [{'id': 'd5ffd8e8-0229-469b-ba10-1462f309718b', 'value': 'WT'}, {'id': '3554f99e-16ce-4899-8c5d-5d4f17234eca', 'value': 'KO'}], 'annotations': [{'start': 151, 'end': 153, 'tag_name': 'GENOTYPE', 'value': 'WT'}, {'start': 612, 'end': 614, 'tag_name': 'GENOTYPE', 'value': 'WT'}, {'start': 193, 'end': 195, 'tag_name': 'GENOTYPE', 'value': 'KO'}, {'start': 638, 'end': 640, 'tag_name': 'GENOTYPE', 'value': 'KO'}]}\n"
     ]
    }
   ],
   "source": [
    "with open('data.json', 'r') as f:\n",
    "    data = json.load(f)\n",
    "    \n",
    "print(data['data'][0])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 96,
   "id": "f3e5cd59",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'text': 'To understand the mechanism of Pikfyve loss in dendritic cells, we cultured conventional dendritic cells isolated from the bone marrow of Pikfyvef/f (“WT”) and mice with ItgaxTg/0 Pikfyvef/f (“KO”) mice and performed transcriptome profiling on RNA extracted from cells on Day 9 in culture. Taken together, these data demonstrate the PIKfyve mediates suppression of DC transcriptional maturation programs through the alternate/non-canonical NF-κB regulatory pathway. Overall design: Comparative gene expression profiling of RNA-seq data from primary cultured conventional dendritic cells isolated from wild-type (WT) or Pikfyve-deficient (KO) mice (littermate pair-matched biological triplicates).', 'entities': [(151, 153, 'GENOTYPE'), (612, 614, 'GENOTYPE'), (193, 195, 'GENOTYPE'), (638, 640, 'GENOTYPE')]}\n"
     ]
    }
   ],
   "source": [
    "\n",
    "training_data = {'classes' : ['GENOTYPE'], 'annotations' : []}\n",
    "for example in data['data']:\n",
    "    temp_dict = {}\n",
    "    temp_dict['text'] = example['content']\n",
    "    temp_dict['entities'] = []\n",
    "    for annotation in example['annotations']:\n",
    "        start = annotation['start']\n",
    "        end = annotation['end']\n",
    "        label = annotation['tag_name'].upper()\n",
    "        temp_dict['entities'].append((start, end, label))\n",
    "    training_data['annotations'].append(temp_dict)\n",
    "  \n",
    "print(training_data['annotations'][0])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 97,
   "id": "6fe609a9",
   "metadata": {},
   "outputs": [],
   "source": [
    "import spacy\n",
    "from spacy.tokens import DocBin\n",
    "from tqdm import tqdm\n",
    "\n",
    "nlp = spacy.blank(\"en\") # load a new spacy model\n",
    "doc_bin = DocBin() # create a DocBin object"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 98,
   "id": "a5f9c348",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "  0%|▏                                         | 6/1804 [00:00<00:37, 48.25it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "WT 151 153 GENOTYPE\n",
      "WT 612 614 GENOTYPE\n",
      "KO 193 195 GENOTYPE\n",
      "KO 638 640 GENOTYPE\n",
      "Control 1481 1488 GENOTYPE\n",
      "Hdac1/Hdac2 double mutant 1493 1518 GENOTYPE\n",
      "WT 603 605 GENOTYPE\n",
      "WT 1001 1003 GENOTYPE\n",
      "Evi1 overexpressing 610 629 GENOTYPE\n",
      "Evi1 overexpressing 913 932 GENOTYPE\n",
      "Evi1 overexpressing 1008 1027 GENOTYPE\n",
      "wild-type 1327 1336 GENOTYPE\n",
      "Control 1999 2006 GENOTYPE\n",
      "PGAM5 KD 2010 2018 GENOTYPE\n",
      "ROSA26-eYFP+/+ Cdh5-CreERT2+ 2144 2172 GENOTYPE\n",
      "ROSA26-eYFP+/+ Cdh5-CreERT2+ 2144 2172 GENOTYPE\n",
      "Prkci flox/flox 1648 1663 GENOTYPE\n",
      "Prkci flox/flox 1728 1743 GENOTYPE\n",
      "MC4R-KO 881 888 GENOTYPE\n",
      "Control 1833 1840 GENOTYPE\n",
      "Control 1881 1888 GENOTYPE\n",
      "PFC_D2R-KO 1636 1646 GENOTYPE\n",
      "PFC_D2R-KO 1959 1969 GENOTYPE\n",
      "PFC_D2R-KO 2009 2019 GENOTYPE\n",
      "Zfp423GFP/+ 103 114 GENOTYPE\n",
      "WT 57 59 GENOTYPE\n",
      "iKO 66 69 GENOTYPE\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "  1%|▍                                        | 20/1804 [00:00<00:28, 62.02it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "GR deletion 1984 1995 GENOTYPE\n",
      "GR deletion 2302 2313 GENOTYPE\n",
      "GR deletion 2533 2544 GENOTYPE\n",
      "GR deletion 2620 2631 GENOTYPE\n",
      "Loxp/Loxp 145 154 GENOTYPE\n",
      "Loxp/Loxp 145 154 GENOTYPE\n",
      "Loxp/Loxp 145 154 GENOTYPE\n",
      "wild-type 1154 1163 GENOTYPE\n",
      "nmrHAS2 355 362 GENOTYPE\n",
      "nmrHAS2 365 372 GENOTYPE\n",
      "nmrHAS2 564 571 GENOTYPE\n",
      "nmrHAS2 660 667 GENOTYPE\n",
      "nmrHAS2 1262 1269 GENOTYPE\n",
      "nmrHAS2 1365 1372 GENOTYPE\n",
      "Lgr5CreER/ KrasLSL-G12D/p53fl/fl/Rosa26-YFP+/+ 1757 1803 GENOTYPE\n",
      "Lgr5CreER/ KrasLSL-G12D/p53fl/fl/Rosa26-YFP+/+ 1763 1809 GENOTYPE\n",
      "WT 1659 1661 GENOTYPE\n",
      "FOXO3 KO 1619 1627 GENOTYPE\n",
      "wildtype 2340 2348 GENOTYPE\n",
      "tumor 119 124 GENOTYPE\n",
      "None 1872 1877 GENOTYPE\n",
      "Skipping entity\n",
      "None 1985 1990 GENOTYPE\n",
      "Skipping entity\n",
      "tumor 2259 2264 GENOTYPE\n",
      "None 2763 2768 GENOTYPE\n",
      "Skipping entity\n",
      "Retnla overexpression 1957 1978 GENOTYPE\n",
      "bigenic 1652 1659 GENOTYPE\n",
      "bigenic 1827 1834 GENOTYPE\n",
      "bigenic 1895 1902 GENOTYPE\n",
      "monogenic 1669 1678 GENOTYPE\n",
      "monogenic 2001 2010 GENOTYPE\n",
      "None 229 233 GENOTYPE\n",
      "Skipping entity\n",
      "None 433 437 GENOTYPE\n",
      "Skipping entity\n",
      "control 1027 1034 GENOTYPE\n",
      "Nkd 395 398 GENOTYPE\n",
      "Nkd 528 531 GENOTYPE\n",
      "Chd4 knockdown 193 207 GENOTYPE\n",
      "wildtype 444 452 GENOTYPE\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "  4%|█▋                                      | 78/1804 [00:00<00:10, 172.37it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "wildtype 1465 1473 GENOTYPE\n",
      "wildtype 1931 1939 GENOTYPE\n",
      "Mtrrgt/gt 692 701 GENOTYPE\n",
      "Mtrrgt/gt 1016 1025 GENOTYPE\n",
      "Mtrrgt/gt 1128 1137 GENOTYPE\n",
      "Mtrrgt/gt 1505 1514 GENOTYPE\n",
      "Mtrrgt/gt 1951 1960 GENOTYPE\n",
      "F2 Mtrr+/+ 1982 1992 GENOTYPE\n",
      "C57BL/6-Ins2Akita/J 2227 2246 GENOTYPE\n",
      "Plaat3-/- 532 541 GENOTYPE\n",
      "Plaat3+/- 592 601 GENOTYPE\n",
      "Nkx6-2 overexpression 384 405 GENOTYPE\n",
      "WT 138 140 GENOTYPE\n",
      "WT 483 485 GENOTYPE\n",
      "Tet2 cKO 94 102 GENOTYPE\n",
      "Tet2 cKO 465 473 GENOTYPE\n",
      "Col2.3kb-Cre; Slc26a2 fl/+ 2001 2027 GENOTYPE\n",
      "Col2.3kb-Cre; Slc26a2 fl/fl 1963 1990 GENOTYPE\n",
      "W/Wv 142 146 GENOTYPE\n",
      "W/Wv 376 380 GENOTYPE\n",
      "W/Wv 627 631 GENOTYPE\n",
      "W/Wv 746 750 GENOTYPE\n",
      "Adgrg2-/Y 104 113 GENOTYPE\n",
      "WT 227 229 GENOTYPE\n",
      "WT 1095 1097 GENOTYPE\n",
      "Akita 1102 1107 GENOTYPE\n",
      "WT 1034 1036 GENOTYPE\n",
      "Akita 1041 1046 GENOTYPE\n",
      "5xFAD 486 491 GENOTYPE\n",
      "5xFAD 1007 1012 GENOTYPE\n",
      "5xFAD 1126 1131 GENOTYPE\n",
      "5xFAD 1137 1142 GENOTYPE\n",
      "Mfn1-/- 460 467 GENOTYPE\n",
      "Mfn1-/- 601 608 GENOTYPE\n",
      "Mfn1-/- 1074 1081 GENOTYPE\n",
      "Mfn1-/- 1578 1585 GENOTYPE\n",
      "Mfn1-/- 1606 1613 GENOTYPE\n",
      "Mfn1-/- 1695 1702 GENOTYPE\n",
      "Mfn2-/- 472 479 GENOTYPE\n",
      "Mfn2-/- 995 1002 GENOTYPE\n",
      "Mfn2-/- 1587 1594 GENOTYPE\n",
      "Mfn2-/- 1625 1632 GENOTYPE\n",
      "WT 207 209 GENOTYPE\n",
      "TNFdARE 1827 1834 GENOTYPE\n",
      "TNFdARE 2068 2075 GENOTYPE\n",
      "WT 1858 1860 GENOTYPE\n",
      "WT 2080 2082 GENOTYPE\n",
      "TNFdARE 2322 2329 GENOTYPE\n",
      "WT 2101 2103 GENOTYPE\n",
      "WT 2334 2336 GENOTYPE\n",
      "WT 211 213 GENOTYPE\n",
      "WT 501 503 GENOTYPE\n",
      "WT 1250 1252 GENOTYPE\n",
      "control 143 150 GENOTYPE\n",
      "MED1 knock 78 88 GENOTYPE\n",
      "UBE3A KO 178 186 GENOTYPE\n",
      "cKO 610 613 GENOTYPE\n",
      "cKO 633 636 GENOTYPE\n",
      "cKO 1458 1461 GENOTYPE\n",
      "cKO 610 613 GENOTYPE\n",
      "cKO 633 636 GENOTYPE\n",
      "cKO 1473 1476 GENOTYPE\n",
      "WT 539 541 GENOTYPE\n",
      "WT 640 642 GENOTYPE\n",
      "Zfp661 KO 647 656 GENOTYPE\n",
      "Cre-Lox-Stop-Lox;(LSL)-c-MycT58A;p53fl/+ 218 258 GENOTYPE\n",
      "Bap1flox 427 435 GENOTYPE\n",
      "Kras 145 149 GENOTYPE\n",
      "Kras 160 164 GENOTYPE\n",
      "Kras/Stk11 160 170 GENOTYPE\n",
      "7 weeks old 548 559 GENOTYPE\n",
      "Type D 1497 1503 GENOTYPE\n",
      "Type D 1468 1474 GENOTYPE\n",
      "Type V 1479 1485 GENOTYPE\n",
      "p53-null 1264 1272 GENOTYPE\n",
      "Sorcs2+/+ 1648 1657 GENOTYPE\n",
      "Sorcs2-/- 1662 1671 GENOTYPE\n",
      "WT 166 168 GENOTYPE\n",
      "Axl KO 123 129 GENOTYPE\n",
      "Axl KO 192 198 GENOTYPE\n",
      "WT 490 492 GENOTYPE\n",
      "WT 849 851 GENOTYPE\n",
      "IRF8KO 161 167 GENOTYPE\n",
      "IRF8KO 204 210 GENOTYPE\n",
      "IRF8KO 699 705 GENOTYPE\n",
      "IRF8KO 853 859 GENOTYPE\n",
      "IRF8KO 874 880 GENOTYPE\n",
      "5xFAD/IRF8KO 198 210 GENOTYPE\n",
      "5xFAD/IRF8KO 868 880 GENOTYPE\n",
      "IRF8flox 1027 1035 GENOTYPE\n",
      "WT 247 249 GENOTYPE\n",
      "Sftpc-CreER; LSL-KrasG12D, LSL-Rosa26-Tdtm 651 693 GENOTYPE\n",
      "SMC1beta knockout 148 165 GENOTYPE\n",
      "SMC1beta knockout 402 419 GENOTYPE\n",
      "WT 1538 1540 GENOTYPE\n",
      "WT 190 192 GENOTYPE\n",
      "Control 209 216 GENOTYPE\n",
      "control-PyMT 66 78 GENOTYPE\n",
      "MYC-PyMT 83 91 GENOTYPE\n",
      "MYC-PyMT 218 226 GENOTYPE\n",
      "PyMT 56 60 GENOTYPE\n",
      "PyMT 122 126 GENOTYPE\n",
      "None 155 159 GENOTYPE\n",
      "Skipping entity\n",
      "PyMT 165 169 GENOTYPE\n",
      "S100a8-creRosa26-LSL-MYCPyMT 131 159 GENOTYPE\n",
      "hepatic TSC1 KO 224 239 GENOTYPE\n",
      "hepatic TSC1 KO 404 419 GENOTYPE\n",
      "E2f1-wt 445 452 GENOTYPE\n",
      "E2f1-ko 457 464 GENOTYPE\n",
      "ATP10A KO 91 100 GENOTYPE\n",
      "ATP10A WT 77 86 GENOTYPE\n",
      "C57/BL6J 324 332 GENOTYPE\n",
      "WT 982 984 GENOTYPE\n",
      "None 240 243 GENOTYPE\n",
      "Skipping entity\n",
      "None 573 576 GENOTYPE\n",
      "Skipping entity\n",
      "wild-type 84 93 GENOTYPE\n",
      "wild- 402 411 GENOTYPE\n",
      "wild- 427 436 GENOTYPE\n",
      "Dp16 106 110 GENOTYPE\n",
      "None 456 460 GENOTYPE\n",
      "Skipping entity\n",
      "None 476 480 GENOTYPE\n",
      "Skipping entity\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "  7%|██▊                                    | 130/1804 [00:00<00:07, 216.00it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "wild-type 84 93 GENOTYPE\n",
      "wild- 402 411 GENOTYPE\n",
      "wild- 427 436 GENOTYPE\n",
      "Dp16 106 110 GENOTYPE\n",
      "None 456 460 GENOTYPE\n",
      "Skipping entity\n",
      "None 476 480 GENOTYPE\n",
      "Skipping entity\n",
      "WT_WT 1553 1558 GENOTYPE\n",
      "WT_d 1560 1564 GENOTYPE\n",
      "195M_195M 1566 1575 GENOTYPE\n",
      "195M_d 1577 1583 GENOTYPE\n",
      "wild-type 83 92 GENOTYPE\n",
      "wild- 401 410 GENOTYPE\n",
      "wild- 426 435 GENOTYPE\n",
      "Dp16 105 109 GENOTYPE\n",
      "None 455 459 GENOTYPE\n",
      "Skipping entity\n",
      "None 475 479 GENOTYPE\n",
      "Skipping entity\n",
      "Dp16 106 110 GENOTYPE\n",
      "None 456 460 GENOTYPE\n",
      "Skipping entity\n",
      "None 476 480 GENOTYPE\n",
      "Skipping entity\n",
      "WT 734 736 GENOTYPE\n",
      "WT 750 752 GENOTYPE\n",
      "WT 60 62 GENOTYPE\n",
      "WT 269 271 GENOTYPE\n",
      "None 356 358 GENOTYPE\n",
      "Skipping entity\n",
      "None 363 365 GENOTYPE\n",
      "Skipping entity\n",
      "None 499 501 GENOTYPE\n",
      "Skipping entity\n",
      "None 506 508 GENOTYPE\n",
      "Skipping entity\n",
      "lncMGC knockout 276 291 GENOTYPE\n",
      "control 140 147 GENOTYPE\n",
      "control 937 944 GENOTYPE\n",
      "cKO 131 134 GENOTYPE\n",
      "cKO 905 908 GENOTYPE\n",
      "SCAP CKO 765 773 GENOTYPE\n",
      "+/+ 1843 1846 GENOTYPE\n",
      "None 839 853 GENOTYPE\n",
      "Skipping entity\n",
      "None 984 998 GENOTYPE\n",
      "Skipping entity\n",
      "None 1214 1228 GENOTYPE\n",
      "Skipping entity\n",
      "Ts65Dn;Df(17)2 1806 1820 GENOTYPE\n",
      "Dp(16)1/Df(16)8 1822 1837 GENOTYPE\n",
      "WT 1236 1238 GENOTYPE\n",
      "OASIS wild type 99 114 GENOTYPE\n",
      "OASIS wild type 408 423 GENOTYPE\n",
      "5XFAD 1472 1477 GENOTYPE\n",
      "NTG 1498 1501 GENOTYPE\n",
      "Fggg390-396A 1458 1470 GENOTYPE\n",
      "Fggg390-396A 1479 1491 GENOTYPE\n",
      "5XFAD 1452 1457 GENOTYPE\n",
      "5XFAD 1472 1477 GENOTYPE\n",
      "NTG 1513 1516 GENOTYPE\n",
      "WT 331 333 GENOTYPE\n",
      "RBPJind 348 355 GENOTYPE\n",
      "Adrb2 knockout 1290 1304 GENOTYPE\n",
      "Adrb3 knockout 1315 1329 GENOTYPE\n",
      "SIRT2 knockout 120 134 GENOTYPE\n",
      "DAAO-TGCdh5 431 442 GENOTYPE\n",
      "DAAO-TGCdh5 821 832 GENOTYPE\n",
      "DAAO-TGCdh5 1557 1568 GENOTYPE\n",
      "empty vector control 472 492 GENOTYPE\n",
      "miR-16 knockdown 452 468 GENOTYPE\n",
      "knockout 106 114 GENOTYPE\n",
      "knockout 278 286 GENOTYPE\n",
      "control 291 298 GENOTYPE\n",
      "Usp28 KO 101 109 GENOTYPE\n",
      "Usp28 KO 226 234 GENOTYPE\n",
      "Usp28 KO 332 340 GENOTYPE\n",
      "WT 95 97 GENOTYPE\n",
      "WT 211 213 GENOTYPE\n",
      "WT 579 581 GENOTYPE\n",
      "WT 793 795 GENOTYPE\n",
      "WT 102 104 GENOTYPE\n",
      "WT 183 185 GENOTYPE\n",
      "WT 490 492 GENOTYPE\n",
      "Mcu-/- 1195 1201 GENOTYPE\n",
      "Mcu-/- 2355 2361 GENOTYPE\n",
      "Krt19-edited 1345 1357 GENOTYPE\n",
      "Tgm2-edited 1359 1370 GENOTYPE\n",
      "NZB/W F1 160 168 GENOTYPE\n",
      "NZB/W F1 328 336 GENOTYPE\n",
      "None 903 906 GENOTYPE\n",
      "Skipping entity\n",
      "LERKO 400 405 GENOTYPE\n",
      "LERKO 481 486 GENOTYPE\n",
      "LERKO 578 583 GENOTYPE\n",
      "LERKO 769 774 GENOTYPE\n",
      "LERKO 1054 1059 GENOTYPE\n",
      "LERKO 1142 1147 GENOTYPE\n",
      "LERKO 1171 1176 GENOTYPE\n",
      "LERKO 1363 1368 GENOTYPE\n",
      "LERKO 1735 1740 GENOTYPE\n",
      "Camk2d knockout 650 665 GENOTYPE\n",
      "db/+ 178 182 GENOTYPE\n",
      "db/+ 238 242 GENOTYPE\n",
      "db/db 72 77 GENOTYPE\n",
      "db/db 119 124 GENOTYPE\n",
      "db/db 264 269 GENOTYPE\n",
      "db/db 276 281 GENOTYPE\n",
      "db/db 331 336 GENOTYPE\n",
      "db/db 343 348 GENOTYPE\n",
      "m/m 187 190 GENOTYPE\n",
      "m/m 526 529 GENOTYPE\n",
      "db/db 114 119 GENOTYPE\n",
      "db/db 448 453 GENOTYPE\n",
      "db/db 488 493 GENOTYPE\n",
      "WT 177 179 GENOTYPE\n",
      "WT 274 276 GENOTYPE\n",
      "Mdr-2 KO 1563 1571 GENOTYPE\n",
      "Xbp1fl 1501 1507 GENOTYPE\n",
      "None 1512 1518 GENOTYPE\n",
      "Skipping entity\n",
      "Xbp1flIre1fl 1512 1524 GENOTYPE\n",
      "WT 81 83 GENOTYPE\n",
      "Crym KO 97 104 GENOTYPE\n",
      "WT 1164 1166 GENOTYPE\n",
      "WT 1175 1177 GENOTYPE\n",
      "WT 1313 1315 GENOTYPE\n",
      "WT 1751 1753 GENOTYPE\n",
      "miR-29a-/- 388 398 GENOTYPE\n",
      "miR-29a-/- 698 708 GENOTYPE\n",
      "miR-29a-/- 829 839 GENOTYPE\n",
      "miR-29a-/- 1085 1095 GENOTYPE\n",
      "miR-29a-/- 1422 1432 GENOTYPE\n",
      "miR-29a-/- 1839 1849 GENOTYPE\n",
      "WT 715 717 GENOTYPE\n",
      "WT 816 818 GENOTYPE\n",
      "APP-KI 286 292 GENOTYPE\n",
      "APP-KI 407 413 GENOTYPE\n",
      "APP-KI 512 518 GENOTYPE\n",
      "APP- 524 530 GENOTYPE\n",
      "APP-KI 722 728 GENOTYPE\n",
      "APP-KI 828 834 GENOTYPE\n",
      "APP- 839 845 GENOTYPE\n",
      "C3aRKO 820 826 GENOTYPE\n",
      "None 846 852 GENOTYPE\n",
      "Skipping entity\n",
      "APP-KI;C3aRKO 839 852 GENOTYPE\n",
      "ATRX KO 864 871 GENOTYPE\n",
      "MCK-Musclin 637 648 GENOTYPE\n",
      "MCK-Musclin 813 824 GENOTYPE\n",
      "None 91 93 GENOTYPE\n",
      "Skipping entity\n",
      "None 221 223 GENOTYPE\n",
      "Skipping entity\n",
      "None 348 350 GENOTYPE\n",
      "Skipping entity\n",
      "GiprEYFP 1944 1952 GENOTYPE\n",
      "Glp1rEYFP 1956 1965 GENOTYPE\n",
      "STINGECKO 638 647 GENOTYPE\n",
      "None 305 307 GENOTYPE\n",
      "Skipping entity\n",
      "WT 630 632 GENOTYPE\n",
      "PexRAP-AKO 453 463 GENOTYPE\n",
      "None 396 399 GENOTYPE\n",
      "Skipping entity\n",
      "mgR 400 403 GENOTYPE\n",
      "None 501 504 GENOTYPE\n",
      "Skipping entity\n",
      "mgR 505 508 GENOTYPE\n",
      "None 649 652 GENOTYPE\n",
      "Skipping entity\n",
      "mgR 653 656 GENOTYPE\n",
      "None 933 936 GENOTYPE\n",
      "Skipping entity\n",
      "mgR 937 940 GENOTYPE\n",
      "WT 1035 1037 GENOTYPE\n",
      "Trp53ko 658 665 GENOTYPE\n",
      "Trp53ko 688 695 GENOTYPE\n",
      "None 239 247 GENOTYPE\n",
      "Skipping entity\n",
      "p53R175H 499 507 GENOTYPE\n",
      "p53R175H 672 680 GENOTYPE\n",
      "p53R175H 703 711 GENOTYPE\n",
      "TCL1 629 633 GENOTYPE\n",
      "TCL1 635 639 GENOTYPE\n",
      "TCL1 733 737 GENOTYPE\n",
      "TCL1 761 765 GENOTYPE\n",
      "TCL1 x Siglecg-/- 635 652 GENOTYPE\n",
      "Siglecg-/- 642 652 GENOTYPE\n",
      "TCL1 x Siglecg-R26ki/ki 733 756 GENOTYPE\n",
      "TCL1 x Siglecg-R26ki/ki 761 784 GENOTYPE\n",
      "Siglecg-R26ki/ki 740 756 GENOTYPE\n",
      "Siglecg-R26ki/ki 768 784 GENOTYPE\n",
      "WT 238 240 GENOTYPE\n",
      "WT 380 382 GENOTYPE\n",
      "Arid1a KO 400 409 GENOTYPE\n",
      "Arid1a Het 158 168 GENOTYPE\n",
      "Arid1a Het 384 394 GENOTYPE\n",
      "STAT1 KO 1671 1679 GENOTYPE\n",
      "MLL-AF9 144 151 GENOTYPE\n",
      "MLL-AF9 174 181 GENOTYPE\n",
      "MLL-AF9/GFP 571 582 GENOTYPE\n",
      "WT 540 542 GENOTYPE\n",
      "WT 814 816 GENOTYPE\n",
      "WT 1477 1479 GENOTYPE\n",
      "Tmprss6-/- 327 337 GENOTYPE\n",
      "Tmprss6-/- 633 643 GENOTYPE\n",
      "Tmprss6-/- 821 831 GENOTYPE\n",
      "Tmprss6-/- 1484 1494 GENOTYPE\n",
      "WT 241 243 GENOTYPE\n",
      "WT 41 43 GENOTYPE\n",
      "WT 224 226 GENOTYPE\n",
      "LKB1 knockout 48 61 GENOTYPE\n",
      "LKB1 knockout 231 244 GENOTYPE\n",
      "Cd45.1/Cd45.2 199 212 GENOTYPE\n",
      "Cd45.1/Cd45.2 934 947 GENOTYPE\n",
      "Cd45.1/Cd45.2 1053 1066 GENOTYPE\n",
      "Cd45.2/Cd45.2 Cre-ERT2 Gata3 Fl/Fl 313 347 GENOTYPE\n",
      "Cd45.2/Cd45.2 Cre-ERT2 Gata3 Fl/Fl 982 1016 GENOTYPE\n",
      "Cd45.2/Cd45.2 Cre-ERT2 Gata3 Fl/Fl 1107 1141 GENOTYPE\n",
      "Ant1A114P,A123D/+ 1443 1460 GENOTYPE\n",
      "None 475 488 GENOTYPE\n",
      "Skipping entity\n",
      "Celf1cKO 501 509 GENOTYPE\n",
      "Tet1 KO 1961 1968 GENOTYPE\n",
      "WT 1944 1946 GENOTYPE\n",
      "wild-type 390 399 GENOTYPE\n",
      "Vsx2<orJ/+>; Mitf<+/+> 999 1021 GENOTYPE\n",
      "STAT4-/- 1400 1408 GENOTYPE\n",
      "WT 68 70 GENOTYPE\n",
      "WT 196 198 GENOTYPE\n",
      "WT 1393 1395 GENOTYPE\n",
      "Slc6a2-eGFP/Rpl10a 542 560 GENOTYPE\n",
      "-eGFP/ 572 590 GENOTYPE\n",
      "WT 171 173 GENOTYPE\n",
      "WT 401 403 GENOTYPE\n",
      "KO 184 186 GENOTYPE\n",
      "KO 408 410 GENOTYPE\n",
      "WT 1985 1987 GENOTYPE\n",
      "C57BL/6 375 382 GENOTYPE\n",
      "PSGL-1-/- 305 314 GENOTYPE\n",
      "Atg5fl/fl 157 166 GENOTYPE\n",
      "Atg5fl/fl 448 457 GENOTYPE\n",
      "Atg5fl/fl 484 493 GENOTYPE\n",
      "Atg5fl/fl 553 562 GENOTYPE\n",
      "Atg5fl/fl LysMcre+/- 484 504 GENOTYPE\n",
      "Atg5fl/fl LysMcre+/+ 157 177 GENOTYPE\n",
      "Atg5fl/fl LysMcre+/+ 553 573 GENOTYPE\n",
      "Atg16L1fl/fl 179 191 GENOTYPE\n",
      "Atg16L1fl/fl 459 471 GENOTYPE\n",
      "Atg16L1fl/fl 506 518 GENOTYPE\n",
      "Atg16L1fl/fl 575 587 GENOTYPE\n",
      "Atg16L1fl/fl LysMcre+/- 506 529 GENOTYPE\n",
      "Atg16L1fl/fl LysMcre+/+ 179 202 GENOTYPE\n",
      "Atg16L1fl/fl LysMcre+/+ 575 598 GENOTYPE\n",
      "Atg7fl/fl LysMcre+/- 531 551 GENOTYPE\n",
      "Atg7fl/fl LysMcre+/+ 207 227 GENOTYPE\n",
      "Atg7fl/fl LysMcre+/+ 603 623 GENOTYPE\n",
      "CTNS KO 1341 1348 GENOTYPE\n",
      "Ribotag 217 224 GENOTYPE\n",
      "Ribotag 413 420 GENOTYPE\n",
      "Ribotag 664 671 GENOTYPE\n",
      "Aldh1l1-CreERT2/Ribotag 397 420 GENOTYPE\n",
      "Aldh1l1-CreERT2/Ribotag 648 671 GENOTYPE\n",
      "WT 82 84 GENOTYPE\n",
      "WT 327 329 GENOTYPE\n",
      "Znrf1 knockout 88 102 GENOTYPE\n",
      "Znrf1 knockout 334 348 GENOTYPE\n",
      "SOD1G93A 469 477 GENOTYPE\n",
      "SOD1G93A 578 586 GENOTYPE\n",
      "SOD1G93A 722 730 GENOTYPE\n",
      "Mbnl2 WT 1241 1249 GENOTYPE\n",
      "wild type 406 415 GENOTYPE\n",
      "DRAG 1473 1477 GENOTYPE\n",
      "Sting-/- 651 659 GENOTYPE\n",
      "M5-DREADD-GFP 1205 1218 GENOTYPE\n",
      "M5-DREADD-GFP 1473 1486 GENOTYPE\n",
      "M5-GFP 1194 1200 GENOTYPE\n",
      "M5-GFP 1355 1361 GENOTYPE\n",
      "M5-GFP 1493 1499 GENOTYPE\n",
      "HOCL 54 58 GENOTYPE\n",
      "None 466 468 GENOTYPE\n",
      "Skipping entity\n",
      "None 477 479 GENOTYPE\n",
      "Skipping entity\n",
      "None 663 665 GENOTYPE\n",
      "Skipping entity\n",
      "WT 1277 1279 GENOTYPE\n",
      "CNS9KO 1284 1290 GENOTYPE\n",
      "CD4-Cre Aclyfl/ 345 361 GENOTYPE\n",
      "NOS1 KO 1251 1258 GENOTYPE\n",
      "Sp3-/- 418 424 GENOTYPE\n",
      "Sp3-/- 544 550 GENOTYPE\n",
      "Sp3-/- 736 742 GENOTYPE\n",
      "Sp3-/- 934 940 GENOTYPE\n",
      "Sp3-/- 1397 1403 GENOTYPE\n",
      "Sp3-/- 1595 1601 GENOTYPE\n",
      "Wild-type 781 790 GENOTYPE\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 12%|████▋                                  | 218/1804 [00:01<00:06, 240.05it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Wild-type 781 790 GENOTYPE\n",
      "WT 1680 1682 GENOTYPE\n",
      "C57BL/6J 107 115 GENOTYPE\n",
      "C57BL/6J 1076 1084 GENOTYPE\n",
      "C57BL/6NCrl 217 228 GENOTYPE\n",
      "C57BL/6NCrl 1176 1187 GENOTYPE\n",
      "Arc/c 485 490 GENOTYPE\n",
      "C57BL6J 256 263 GENOTYPE\n",
      "PPARa KO 118 126 GENOTYPE\n",
      "sFlt1 193 198 GENOTYPE\n",
      "WT 15 17 GENOTYPE\n",
      "WT 50 52 GENOTYPE\n",
      "Cpa3Cre/+ 381 390 GENOTYPE\n",
      "Pax7 363 367 GENOTYPE\n",
      "Pax7 482 486 GENOTYPE\n",
      "Pax7 590 594 GENOTYPE\n",
      "Pax7 799 803 GENOTYPE\n",
      "Pax7 1022 1026 GENOTYPE\n",
      "Pax7 1256 1260 GENOTYPE\n",
      "Pax7 363 367 GENOTYPE\n",
      "Pax7 482 486 GENOTYPE\n",
      "Pax7 590 594 GENOTYPE\n",
      "Pax7 799 803 GENOTYPE\n",
      "Pax7 1022 1026 GENOTYPE\n",
      "Pax7 1256 1260 GENOTYPE\n",
      "WT 591 593 GENOTYPE\n",
      "WT 1921 1923 GENOTYPE\n",
      "Abcb10 knockout 1938 1953 GENOTYPE\n",
      "Pb-Cre; PTENfl/fl Trp53fl/fl 2078 2106 GENOTYPE\n",
      "WT 371 373 GENOTYPE\n",
      "WT 582 584 GENOTYPE\n",
      "Cx3cr1-CreER, Manffl/fl 274 297 GENOTYPE\n",
      "Cpa3Cre/+ 381 390 GENOTYPE\n",
      "WT 390 392 GENOTYPE\n",
      "WT 547 549 GENOTYPE\n",
      "ST3GAL3 knockdown 465 482 GENOTYPE\n",
      "ST3GAL3 knockdown 569 586 GENOTYPE\n",
      "ST3GAL3 knockdown 833 850 GENOTYPE\n",
      "IRF3-knockout 243 256 GENOTYPE\n",
      "IFNAR-knockout 260 274 GENOTYPE\n",
      "Scd1-deficient C57BL/6 298 320 GENOTYPE\n",
      "COX10 KO 191 199 GENOTYPE\n",
      "None 513 520 GENOTYPE\n",
      "Skipping entity\n",
      "C57BL/6 486 493 GENOTYPE\n",
      "TN61R 110 115 GENOTYPE\n",
      "WT 260 262 GENOTYPE\n",
      "RelBko 264 270 GENOTYPE\n",
      "None 293 299 GENOTYPE\n",
      "Skipping entity\n",
      "None 272 277 GENOTYPE\n",
      "Skipping entity\n",
      "None 285 290 GENOTYPE\n",
      "Skipping entity\n",
      "WT 1196 1198 GENOTYPE\n",
      "DTR 1209 1212 GENOTYPE\n",
      "PCMT1+/+ 283 291 GENOTYPE\n",
      "PCMT1+/+ 435 443 GENOTYPE\n",
      "PCMT1+/- 60 68 GENOTYPE\n",
      "PCMT1+/- 293 301 GENOTYPE\n",
      "PCMT1+/- 445 453 GENOTYPE\n",
      "PCMT1-/- 29 37 GENOTYPE\n",
      "PCMT1-/- 302 310 GENOTYPE\n",
      "PCMT1-/- 315 323 GENOTYPE\n",
      "PCMT1-/- 455 463 GENOTYPE\n",
      "PCMT1-/- 468 476 GENOTYPE\n",
      "None 545 547 GENOTYPE\n",
      "Skipping entity\n",
      "None 912 914 GENOTYPE\n",
      "Skipping entity\n",
      "None 1107 1109 GENOTYPE\n",
      "Skipping entity\n",
      "APP 77 80 GENOTYPE\n",
      "APP 317 320 GENOTYPE\n",
      "WT 70 72 GENOTYPE\n",
      "WT 310 312 GENOTYPE\n",
      "WT 1198 1200 GENOTYPE\n",
      "WT 230 232 GENOTYPE\n",
      "WT 260 262 GENOTYPE\n",
      "WT 295 297 GENOTYPE\n",
      "K-Ctrl 128 134 GENOTYPE\n",
      "K-Ctrl 285 291 GENOTYPE\n",
      "K-Setd2KO 139 148 GENOTYPE\n",
      "K-Setd2KO 298 307 GENOTYPE\n",
      "KPY-Ctrl 130 138 GENOTYPE\n",
      "KPY-Ctrl 286 294 GENOTYPE\n",
      "WT 1229 1231 GENOTYPE\n",
      "PP2Cm KO 1194 1202 GENOTYPE\n",
      "Ngn3-Cre;tdTom 179 193 GENOTYPE\n",
      "NeuroD1-Cre;tdTom 198 215 GENOTYPE\n",
      "KO 501 503 GENOTYPE\n",
      "KO 629 631 GENOTYPE\n",
      "KO 717 719 GENOTYPE\n",
      "WT 710 712 GENOTYPE\n",
      "WT 977 979 GENOTYPE\n",
      "Ripk3 knock-out 2013 2028 GENOTYPE\n",
      "Neurog3f/f 2075 2085 GENOTYPE\n",
      "Neurog3f/f 2120 2130 GENOTYPE\n",
      "Neurog3f/f 2160 2170 GENOTYPE\n",
      "Neurog3f/f 2205 2215 GENOTYPE\n",
      "WT 2046 2048 GENOTYPE\n",
      "WT 1176 1178 GENOTYPE\n",
      "DTR 1189 1192 GENOTYPE\n",
      "SKH1 321 325 GENOTYPE\n",
      "WT 2408 2410 GENOTYPE\n",
      "WT 2424 2426 GENOTYPE\n",
      "Aire-/- 515 522 GENOTYPE\n",
      "Aire-/- 963 970 GENOTYPE\n",
      "Aire-/- 1777 1784 GENOTYPE\n",
      "Pglyrp1 KO 263 273 GENOTYPE\n",
      "WT 243 245 GENOTYPE\n",
      "WT 1084 1086 GENOTYPE\n",
      "WT 1391 1393 GENOTYPE\n",
      "CD43 KO 256 263 GENOTYPE\n",
      "Fbn1C1041G/+ 44 56 GENOTYPE\n",
      "Fbn1C1041G/+ 120 132 GENOTYPE\n",
      "p16-INK-ATTAC 1097 1110 GENOTYPE\n",
      "WT 626 628 GENOTYPE\n",
      "WT 820 822 GENOTYPE\n",
      "MMP2-/- 630 637 GENOTYPE\n",
      "MMP2-/- 824 831 GENOTYPE\n",
      "MMP9-/- 639 646 GENOTYPE\n",
      "MMP9-/- 833 840 GENOTYPE\n",
      "control 378 385 GENOTYPE\n",
      "Egr4-/- 1767 1774 GENOTYPE\n",
      "APOE4 672 677 GENOTYPE\n",
      "MX1gfp 155 161 GENOTYPE\n",
      "wild type 1081 1090 GENOTYPE\n",
      "tRNA-Phe-GAA-1-1-/- 1092 1111 GENOTYPE\n",
      "tRNA-Phe-GAA-2-1-/- 1113 1132 GENOTYPE\n",
      "tRNA-Phe-GAA-3-1-/- 1137 1156 GENOTYPE\n",
      "Ndp WT 1248 1254 GENOTYPE\n",
      "Tg-AD+mAb 1663 1672 GENOTYPE\n",
      "TET_WT 2052 2058 GENOTYPE\n",
      "DNMT_WT 2165 2172 GENOTYPE\n",
      "WT 147 149 GENOTYPE\n",
      "WT 175 177 GENOTYPE\n",
      "None 1443 1446 GENOTYPE\n",
      "Skipping entity\n",
      "None 1456 1459 GENOTYPE\n",
      "Skipping entity\n",
      "WT 70 72 GENOTYPE\n",
      "Tfam+/- 74 81 GENOTYPE\n",
      "None 95 102 GENOTYPE\n",
      "Skipping entity\n",
      "Zbp1-/- 83 90 GENOTYPE\n",
      "None 102 109 GENOTYPE\n",
      "Skipping entity\n",
      "Tfam+/-Zbp1-/- 95 109 GENOTYPE\n",
      "Cdh5(PAC)-CreERT2; Rbpjflox/WT (control) 2308 2348 GENOTYPE\n",
      "Cdh5(PAC)-CreERT2; Rbpjflox/flox (mutant) 2353 2394 GENOTYPE\n",
      "Dcir-/- 236 243 GENOTYPE\n",
      "Dcir-/- 492 499 GENOTYPE\n",
      "Dcir-/- 761 768 GENOTYPE\n",
      "Tg2576 3 9 GENOTYPE\n",
      "Tg2576 683 689 GENOTYPE\n",
      "wildtype 858 866 GENOTYPE\n",
      "wildtype 1211 1219 GENOTYPE\n",
      "wildtype 1250 1258 GENOTYPE\n",
      "BrafV600E/Pten-/-/Cxcr2WT 186 211 GENOTYPE\n",
      "BrafV600E/Pten-/-/Cxcr2WT 379 404 GENOTYPE\n",
      "BRAFV600E/PTEN-/-/CXCR2-/- 222 248 GENOTYPE\n",
      "BRAFV600E/PTEN-/-/CXCR2-/- 348 374 GENOTYPE\n",
      "HPK1/B 1225 1231 GENOTYPE\n",
      "HPK1-M46 transgenic 81 100 GENOTYPE\n",
      "C57BL/6J 366 374 GENOTYPE\n",
      "AhR-knockout 937 949 GENOTYPE\n",
      "AhR-knockout 2366 2378 GENOTYPE\n",
      "WT 1148 1150 GENOTYPE\n",
      "WT 1498 1500 GENOTYPE\n",
      "NIK KO 1505 1511 GENOTYPE\n",
      "wild-type 681 690 GENOTYPE\n",
      "Glut4m 669 675 GENOTYPE\n",
      "Calhm6-/- 413 422 GENOTYPE\n",
      "C57BL/6J 1765 1773 GENOTYPE\n",
      "C57BL/6J 2030 2038 GENOTYPE\n",
      "APP/PS1 605 612 GENOTYPE\n",
      "APP/PS1 622 629 GENOTYPE\n",
      "APP/PS1 752 759 GENOTYPE\n",
      "APP/PS1 893 900 GENOTYPE\n",
      "APP/PS1 1137 1144 GENOTYPE\n",
      "APP/PS1 1690 1697 GENOTYPE\n",
      "APP/PS1 1810 1817 GENOTYPE\n",
      "APP/PS1 1928 1935 GENOTYPE\n",
      "APP/PS1 1984 1991 GENOTYPE\n",
      "APP/ 2016 2023 GENOTYPE\n",
      "albumin-knockout (AKO) 18 40 GENOTYPE\n",
      "albumin-knockout (AKO) 300 322 GENOTYPE\n",
      "Wildtype (WT) 0 13 GENOTYPE\n",
      "Wildtype (WT) 282 295 GENOTYPE\n",
      "Wildtype (WT) 612 625 GENOTYPE\n",
      "WT 1165 1167 GENOTYPE\n",
      "PDH knockout 1198 1210 GENOTYPE\n",
      "PDH knockout 1198 1210 GENOTYPE\n",
      "3BKO 822 826 GENOTYPE\n",
      "3BKO 1017 1021 GENOTYPE\n",
      "3BKO 1551 1555 GENOTYPE\n",
      "WT 1497 1499 GENOTYPE\n",
      "WT 1512 1514 GENOTYPE\n",
      "3AKO 1529 1533 GENOTYPE\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 15%|█████▊                                 | 269/1804 [00:01<00:06, 239.99it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/fl 995 1005 GENOTYPE\n",
      "/fl 1023 1033 GENOTYPE\n",
      "Otud4fl/fl 1118 1128 GENOTYPE\n",
      "Otud4fl/fl 1719 1729 GENOTYPE\n",
      "/fl 1748 1758 GENOTYPE\n",
      "Vil-Cre;Otud4fl/fl 987 1005 GENOTYPE\n",
      "Vil-Cre;Otud4fl/fl 1740 1758 GENOTYPE\n",
      "Nr1d1+/+;PyMT 134 147 GENOTYPE\n",
      "Nr1d1+/+;PyMT 327 340 GENOTYPE\n",
      "Nr1d1-/-;PyMT 152 165 GENOTYPE\n",
      "Nr1d1-/-;PyMT 345 358 GENOTYPE\n",
      "RiboTagF/F;Zp3-Cre 1219 1237 GENOTYPE\n",
      "GRTH KO 415 422 GENOTYPE\n",
      "GRTH KO 415 422 GENOTYPE\n",
      "WT 89 91 GENOTYPE\n",
      "WT 213 215 GENOTYPE\n",
      "A/J 268 271 GENOTYPE\n",
      "A/J 571 574 GENOTYPE\n",
      "A/J 1030 1033 GENOTYPE\n",
      "A/J 1588 1591 GENOTYPE\n",
      "C57BL/6J 429 437 GENOTYPE\n",
      "C57BL/6J 1596 1604 GENOTYPE\n",
      "control 44 51 GENOTYPE\n",
      "control 203 210 GENOTYPE\n",
      "Flnc global knockout 56 76 GENOTYPE\n",
      "Mael-/- 306 313 GENOTYPE\n",
      "Mael-/- 468 475 GENOTYPE\n",
      "control 1315 1322 GENOTYPE\n",
      "control 1559 1566 GENOTYPE\n",
      "C151S 234 239 GENOTYPE\n",
      "C151S 421 426 GENOTYPE\n",
      "wild-type 157 166 GENOTYPE\n",
      "wild-type 389 398 GENOTYPE\n",
      "MMP14 F/+; COL10A1cre; R26-tdT; Col1a1-2.3-GFP 1528 1574 GENOTYPE\n",
      "WT 649 651 GENOTYPE\n",
      "hMBP 657 661 GENOTYPE\n",
      "hMBP 720 724 GENOTYPE\n",
      "Seh1cKO 1237 1244 GENOTYPE\n",
      "WT 1110 1112 GENOTYPE\n",
      "wild-type 260 269 GENOTYPE\n",
      "Mdivi1 treatment 19 35 GENOTYPE\n",
      "control 567 574 GENOTYPE\n",
      "WT 885 887 GENOTYPE\n",
      "Prnp-/- 752 759 GENOTYPE\n",
      "Prnp-/- 847 854 GENOTYPE\n",
      "DKI 669 672 GENOTYPE\n",
      "DKI 747 750 GENOTYPE\n",
      "DKI 781 784 GENOTYPE\n",
      "DKI; Prnp-/- 747 759 GENOTYPE\n",
      "None 1416 1418 GENOTYPE\n",
      "Skipping entity\n",
      "None 1442 1444 GENOTYPE\n",
      "Skipping entity\n",
      "Npc1-/- 554 561 GENOTYPE\n",
      "Npc1-/- 899 906 GENOTYPE\n",
      "Idh2LSL-R172K/+;CD4-Cre 1117 1140 GENOTYPE\n",
      "Idh2LSL-R172K/+;CD4-Cre 1094 1117 GENOTYPE\n",
      "Tet2-/-;CD4-Cre 1119 1134 GENOTYPE\n",
      "Tet2-/-;CD4-Cre 1153 1168 GENOTYPE\n",
      "wildtype 1642 1650 GENOTYPE\n",
      "fat-specific Nampt knockout 1655 1682 GENOTYPE\n",
      "WT C57Bl6J/129X1 hybrid 627 650 GENOTYPE\n",
      "AR100Q 222 228 GENOTYPE\n",
      "AR100Q 287 293 GENOTYPE\n",
      "Neurog3f/f 2071 2081 GENOTYPE\n",
      "Neurog3f/f 2116 2126 GENOTYPE\n",
      "Neurog3f/f 2156 2166 GENOTYPE\n",
      "Neurog3f/f 2201 2211 GENOTYPE\n",
      "Il4raF709 1360 1369 GENOTYPE\n",
      "None 1382 1391 GENOTYPE\n",
      "Skipping entity\n",
      "WT 1356 1358 GENOTYPE\n",
      "CCN2fl/fl 840 849 GENOTYPE\n",
      "CCN2fl/fl 1024 1033 GENOTYPE\n",
      "CCN2fl/fl 1305 1314 GENOTYPE\n",
      "WT 963 965 GENOTYPE\n",
      "WT 1265 1267 GENOTYPE\n",
      "WT 1575 1577 GENOTYPE\n",
      "WT 1795 1797 GENOTYPE\n",
      "WT 1882 1884 GENOTYPE\n",
      "WT 1995 1997 GENOTYPE\n",
      "AS 19 21 GENOTYPE\n",
      "AS 410 412 GENOTYPE\n",
      "AS 575 577 GENOTYPE\n",
      "AS 837 839 GENOTYPE\n",
      "AS 1125 1127 GENOTYPE\n",
      "AS 1258 1260 GENOTYPE\n",
      "AS 1490 1492 GENOTYPE\n",
      "AS 1639 1641 GENOTYPE\n",
      "AS 1822 1824 GENOTYPE\n",
      "AS 2014 2016 GENOTYPE\n",
      "Mecp2-null 849 859 GENOTYPE\n",
      "Mecp2-null 1029 1039 GENOTYPE\n",
      "Mecp2-null 1739 1749 GENOTYPE\n",
      "PBS 636 639 GENOTYPE\n",
      "PBS 969 972 GENOTYPE\n",
      "None 19 24 GENOTYPE\n",
      "Skipping entity\n",
      "None 112 117 GENOTYPE\n",
      "Skipping entity\n",
      "Npc1-/- 554 561 GENOTYPE\n",
      "Npc1-/- 899 906 GENOTYPE\n",
      "Npc1-/- 1000 1007 GENOTYPE\n",
      "None 578 580 GENOTYPE\n",
      "Skipping entity\n",
      "PS19-fE3 1337 1345 GENOTYPE\n",
      "PS19-fE4 1294 1302 GENOTYPE\n",
      "WT 173 175 GENOTYPE\n",
      "HBZ 103 106 GENOTYPE\n",
      "HBZ 200 203 GENOTYPE\n",
      "HBZ 235 238 GENOTYPE\n",
      "None 85 89 GENOTYPE\n",
      "Skipping entity\n",
      "None 177 181 GENOTYPE\n",
      "Skipping entity\n",
      "None 212 216 GENOTYPE\n",
      "Skipping entity\n",
      "WT 305 307 GENOTYPE\n",
      "WT 331 333 GENOTYPE\n",
      "WT 618 620 GENOTYPE\n",
      "None 354 356 GENOTYPE\n",
      "Skipping entity\n",
      "WT 243 245 GENOTYPE\n",
      "AhR-KO 112 118 GENOTYPE\n",
      "AhR-KO 355 361 GENOTYPE\n",
      "WT 1699 1701 GENOTYPE\n",
      "Wnt1Cre;PRDM6f/f 187 203 GENOTYPE\n",
      "p53-null 1131 1139 GENOTYPE\n",
      "Ahrf/ 1247 1253 GENOTYPE\n",
      "Ahrf/f 1281 1287 GENOTYPE\n",
      "Ahrf/fCd8cre 1247 1259 GENOTYPE\n",
      "Ahr+/+ 1336 1342 GENOTYPE\n",
      "Ahr+/+ 1537 1543 GENOTYPE\n",
      "Ahr+/+ 1699 1705 GENOTYPE\n",
      "WT 561 563 GENOTYPE\n",
      "Bmal1-flox 319 329 GENOTYPE\n",
      "Bmal1-flox 530 540 GENOTYPE\n",
      "Bmal1-iKO 167 176 GENOTYPE\n",
      "Bmal1-iKO 296 305 GENOTYPE\n",
      "Bmal1-iKO 516 525 GENOTYPE\n",
      "wild-type 156 165 GENOTYPE\n",
      "wild-type 330 339 GENOTYPE\n",
      "ZFP750-/- 875 884 GENOTYPE\n",
      "ZFP750-/- 958 967 GENOTYPE\n",
      "ZFP750-/- 1714 1723 GENOTYPE\n",
      "WT 210 212 GENOTYPE\n",
      "Col1a1Jrt/+ 231 242 GENOTYPE\n",
      "Col1a1Jrt/+ 447 458 GENOTYPE\n",
      "Col1a1Jrt/+ 477 488 GENOTYPE\n",
      "Col1a1Jrt/+ 545 556 GENOTYPE\n",
      "Col1a1Jrt/+ 595 606 GENOTYPE\n",
      "Col1a1Jrt/+ 648 659 GENOTYPE\n",
      "Col1a1Jrt/+ 714 725 GENOTYPE\n",
      "Col1a1Jrt/+ 1208 1219 GENOTYPE\n",
      "wild type 1157 1166 GENOTYPE\n",
      "KAT7 mutant 1176 1187 GENOTYPE\n",
      "CreER 593 598 GENOTYPE\n",
      "None 1909 1914 GENOTYPE\n",
      "Skipping entity\n",
      "None 1989 1994 GENOTYPE\n",
      "Skipping entity\n",
      "WT 113 115 GENOTYPE\n",
      "WT 169 171 GENOTYPE\n",
      "WT 430 432 GENOTYPE\n",
      "Abin1Q478H/Q478H 38 54 GENOTYPE\n",
      "Abin1Q478H/Q478H 143 159 GENOTYPE\n",
      "Abin1Q478H/Q478H 404 420 GENOTYPE\n",
      "WT 160 162 GENOTYPE\n",
      "WT 393 395 GENOTYPE\n",
      "Gmnc-/- 1518 1525 GENOTYPE\n",
      "WT 273 275 GENOTYPE\n",
      "WT 290 292 GENOTYPE\n",
      "C57BL/6 1331 1338 GENOTYPE\n",
      "C57BL/6 1250 1257 GENOTYPE\n",
      "Krt8-YFP mice 694 707 GENOTYPE\n",
      "Krt8-YFP 358 366 GENOTYPE\n",
      "WT 1572 1574 GENOTYPE\n",
      "WT 1627 1629 GENOTYPE\n",
      "WT 1725 1727 GENOTYPE\n",
      "HET 1565 1568 GENOTYPE\n",
      "HET 1620 1623 GENOTYPE\n",
      "HET 1718 1721 GENOTYPE\n",
      "HOM 1711 1714 GENOTYPE\n",
      "Gprc5a-/- 230 239 GENOTYPE\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 20%|███████▉                               | 365/1804 [00:01<00:05, 282.28it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "WT 1183 1185 GENOTYPE\n",
      "KO 1165 1167 GENOTYPE\n",
      "Nr4a3-Tocky Great Smart-17A 167 194 GENOTYPE\n",
      "KO 128 130 GENOTYPE\n",
      "KO 374 376 GENOTYPE\n",
      "Fah-/- 488 494 GENOTYPE\n",
      "RBM47 K329R;K332R 371 388 GENOTYPE\n",
      "WT 1384 1386 GENOTYPE\n",
      "Dp16 113 117 GENOTYPE\n",
      "Dp16 161 165 GENOTYPE\n",
      "Dp16 432 436 GENOTYPE\n",
      "Dp16 448 452 GENOTYPE\n",
      "Dp16^2xIFNRs 161 173 GENOTYPE\n",
      "Dp16^2xIFNRs 448 460 GENOTYPE\n",
      "Dp16 102 106 GENOTYPE\n",
      "Dp16 150 154 GENOTYPE\n",
      "Dp16 421 425 GENOTYPE\n",
      "Dp16 437 441 GENOTYPE\n",
      "Dp16^2xIFNRs 150 162 GENOTYPE\n",
      "Dp16^2xIFNRs 437 449 GENOTYPE\n",
      "Dp16 102 106 GENOTYPE\n",
      "Dp16 150 154 GENOTYPE\n",
      "Dp16 421 425 GENOTYPE\n",
      "Dp16 437 441 GENOTYPE\n",
      "Dp16^2xIFNRs 150 162 GENOTYPE\n",
      "Dp16^2xIFNRs 437 449 GENOTYPE\n",
      "Dp16 114 118 GENOTYPE\n",
      "Dp16 162 166 GENOTYPE\n",
      "Dp16 433 437 GENOTYPE\n",
      "Dp16 449 453 GENOTYPE\n",
      "Dp16^2xIFNRs 162 174 GENOTYPE\n",
      "Dp16^2xIFNRs 449 461 GENOTYPE\n",
      "Dp16 104 108 GENOTYPE\n",
      "Dp16 152 156 GENOTYPE\n",
      "Dp16 423 427 GENOTYPE\n",
      "Dp16 439 443 GENOTYPE\n",
      "Dp16^2xIFNRs 152 164 GENOTYPE\n",
      "Dp16^2xIFNRs 439 451 GENOTYPE\n",
      "Dp16 103 107 GENOTYPE\n",
      "Dp16 151 155 GENOTYPE\n",
      "Dp16 422 426 GENOTYPE\n",
      "Dp16 438 442 GENOTYPE\n",
      "Dp16^2xIFNRs 151 163 GENOTYPE\n",
      "Dp16^2xIFNRs 438 450 GENOTYPE\n",
      "C57Bl6/J 232 240 GENOTYPE\n",
      "WT 633 635 GENOTYPE\n",
      "PS19 260 264 GENOTYPE\n",
      "PS19 513 517 GENOTYPE\n",
      "PS19 586 590 GENOTYPE\n",
      "PS19 640 644 GENOTYPE\n",
      "NOD 414 417 GENOTYPE\n",
      "NOD 471 474 GENOTYPE\n",
      "NOD 966 969 GENOTYPE\n",
      "Neurog3eYFP/+ 185 198 GENOTYPE\n",
      "Neurog3eYFP/+ 288 301 GENOTYPE\n",
      "Prf1-/- 2122 2129 GENOTYPE\n",
      "C57Bl/6 932 939 GENOTYPE\n",
      "WT 99 101 GENOTYPE\n",
      "WT 415 417 GENOTYPE\n",
      "WT 70 72 GENOTYPE\n",
      "WT 340 342 GENOTYPE\n",
      "STING KO 347 355 GENOTYPE\n",
      "KiC 1298 1301 GENOTYPE\n",
      "iC 1279 1281 GENOTYPE\n",
      "None 1299 1301 GENOTYPE\n",
      "Skipping entity\n",
      "WT 1273 1275 GENOTYPE\n",
      "KC 1285 1287 GENOTYPE\n",
      "KPC 1291 1294 GENOTYPE\n",
      "Wild Type 243 252 GENOTYPE\n",
      "Slc20a2 homozygous (HO) knockout 369 401 GENOTYPE\n",
      "wild type 1870 1879 GENOTYPE\n",
      "3xTg-AD 96 103 GENOTYPE\n",
      "3xTg- 567 574 GENOTYPE\n",
      "3xTg-AD 948 955 GENOTYPE\n",
      "3xTg-AD 1128 1135 GENOTYPE\n",
      "3xTg-AD 1890 1897 GENOTYPE\n",
      "WT 918 920 GENOTYPE\n",
      "WT 1173 1175 GENOTYPE\n",
      "None 936 945 GENOTYPE\n",
      "Skipping entity\n",
      "None 1182 1191 GENOTYPE\n",
      "Skipping entity\n",
      "None 936 940 GENOTYPE\n",
      "Skipping entity\n",
      "None 1182 1186 GENOTYPE\n",
      "Skipping entity\n",
      "None 1205 1209 GENOTYPE\n",
      "Skipping entity\n",
      "WT 96 98 GENOTYPE\n",
      "WT 207 209 GENOTYPE\n",
      "WT 321 323 GENOTYPE\n",
      "Nrf2 knockout 103 116 GENOTYPE\n",
      "Atxn1_154Q/2Q 49 62 GENOTYPE\n",
      "Atxn1_154Q/2Q 289 302 GENOTYPE\n",
      "WT 972 974 GENOTYPE\n",
      "mdx5cv 141 147 GENOTYPE\n",
      "mdx5cv 367 373 GENOTYPE\n",
      "mdx5cv 961 967 GENOTYPE\n",
      "WT 1103 1105 GENOTYPE\n",
      "Pdcd1-/- 359 367 GENOTYPE\n",
      "Pdcd1-/- 1115 1123 GENOTYPE\n",
      "WT 1170 1172 GENOTYPE\n",
      "BTBR +/+ 196 204 GENOTYPE\n",
      "BTBR +/+ 466 474 GENOTYPE\n",
      "BTBR ob/ob 76 86 GENOTYPE\n",
      "BTBR ob/ob 181 191 GENOTYPE\n",
      "BTBR ob/ob 451 461 GENOTYPE\n",
      "db/db 1334 1339 GENOTYPE\n",
      "/db 1360 1365 GENOTYPE\n",
      "db/+ 1341 1345 GENOTYPE\n",
      "Trpm7R/R 226 234 GENOTYPE\n",
      "Trpm7R/R 344 352 GENOTYPE\n",
      "Trpm7R/R 537 545 GENOTYPE\n",
      "HGFAC KO 767 775 GENOTYPE\n",
      "HGFAC KO 1268 1276 GENOTYPE\n",
      "empty 458 463 GENOTYPE\n",
      "FUS-ERG 32 39 GENOTYPE\n",
      "FUS-ERG 423 430 GENOTYPE\n",
      "knockout 2176 2184 GENOTYPE\n",
      "knockout 2220 2228 GENOTYPE\n",
      "Pdcd1-/- 359 367 GENOTYPE\n",
      "Pdcd1-/- 1115 1123 GENOTYPE\n",
      "KLF WT 221 227 GENOTYPE\n",
      "KLF K74R 19 27 GENOTYPE\n",
      "KLF K74R 193 201 GENOTYPE\n",
      "WT 75 77 GENOTYPE\n",
      "WT 262 264 GENOTYPE\n",
      "conditional KO 229 243 GENOTYPE\n",
      "Lin28b 336 342 GENOTYPE\n",
      "Lin28b 546 552 GENOTYPE\n",
      "Pink1+/+ 144 152 GENOTYPE\n",
      "Pink1+/+ 317 325 GENOTYPE\n",
      "Pink1-/- 131 139 GENOTYPE\n",
      "Pink1-/- 330 338 GENOTYPE\n",
      "Tfap2b-/- 110 119 GENOTYPE\n",
      "WT 696 698 GENOTYPE\n",
      "WT 1742 1744 GENOTYPE\n",
      "WT 1918 1920 GENOTYPE\n",
      "WT 2017 2019 GENOTYPE\n",
      "Dp16 343 347 GENOTYPE\n",
      "Dp16 1167 1171 GENOTYPE\n",
      "Dp16 1746 1750 GENOTYPE\n",
      "HET 355 358 GENOTYPE\n",
      "WT 335 337 GENOTYPE\n",
      "Leprdb/db 813 822 GENOTYPE\n",
      "Leprdb/db 2321 2330 GENOTYPE\n",
      "WT 703 705 GENOTYPE\n",
      "WT 853 855 GENOTYPE\n",
      "WT 1062 1064 GENOTYPE\n",
      "p63CreER; Piezo1fl/fl ; R26-mTmG (Piezo1 cKO) 111 156 GENOTYPE\n",
      "None 97 99 GENOTYPE\n",
      "Skipping entity\n",
      "None 425 427 GENOTYPE\n",
      "Skipping entity\n",
      "None 523 525 GENOTYPE\n",
      "Skipping entity\n",
      "None 643 645 GENOTYPE\n",
      "Skipping entity\n",
      "None 1003 1005 GENOTYPE\n",
      "Skipping entity\n",
      "None 1174 1176 GENOTYPE\n",
      "Skipping entity\n",
      "None 1537 1539 GENOTYPE\n",
      "Skipping entity\n",
      "None 1814 1816 GENOTYPE\n",
      "Skipping entity\n",
      "None 2096 2098 GENOTYPE\n",
      "Skipping entity\n",
      "None 2103 2105 GENOTYPE\n",
      "Skipping entity\n",
      "None 2114 2116 GENOTYPE\n",
      "Skipping entity\n",
      "Ch25hfl/fl x VE-Cadherin-Cre- 1400 1429 GENOTYPE\n",
      "Ch25hfl/fl x VE-Cadherin-Cre+ 1481 1510 GENOTYPE\n",
      "Cxcr4+/+ 192 200 GENOTYPE\n",
      "Cxcr4+/+ 379 387 GENOTYPE\n",
      "Cxcr4+/1013 218 229 GENOTYPE\n",
      "Cxcr4+/1013 389 400 GENOTYPE\n",
      "Cxcr41013/1013 247 261 GENOTYPE\n",
      "Cxcr41013/1013 405 419 GENOTYPE\n",
      "WT 905 907 GENOTYPE\n",
      "WT 520 522 GENOTYPE\n",
      "WT 746 748 GENOTYPE\n",
      "PTRF knockout 390 403 GENOTYPE\n",
      "PTRF knockout 1148 1161 GENOTYPE\n",
      "WT 1210 1212 GENOTYPE\n",
      "Stat3-/- 1217 1225 GENOTYPE\n",
      "None 1444 1446 GENOTYPE\n",
      "Skipping entity\n",
      "None 1475 1477 GENOTYPE\n",
      "Skipping entity\n",
      "WT 348 350 GENOTYPE\n",
      "Bmal1+/+ 32 40 GENOTYPE\n",
      "Bmal1+/+ 286 294 GENOTYPE\n",
      "Bmal1-/- 45 53 GENOTYPE\n",
      "Bmal1-/- 299 307 GENOTYPE\n",
      "wildtype 1340 1348 GENOTYPE\n",
      "/f 197 200 GENOTYPE\n",
      "/f 334 337 GENOTYPE\n",
      "/d 211 214 GENOTYPE\n",
      "/d 348 351 GENOTYPE\n",
      "WT 275 277 GENOTYPE\n",
      "WT 294 296 GENOTYPE\n",
      "None 535 537 GENOTYPE\n",
      "Skipping entity\n",
      "None 547 549 GENOTYPE\n",
      "Skipping entity\n",
      "H-2k 535 539 GENOTYPE\n",
      "WT 285 287 GENOTYPE\n",
      "EML4-ALK 25 33 GENOTYPE\n",
      "EML4-ALK 317 325 GENOTYPE\n",
      "EML4-ALK 512 520 GENOTYPE\n",
      "EML4-ALK 703 711 GENOTYPE\n",
      "EML4-ALK 1170 1178 GENOTYPE\n",
      "C57BL/6 827 834 GENOTYPE\n",
      "Swiss Webster wild type 1540 1563 GENOTYPE\n",
      "Nlrp3+/+ 61 69 GENOTYPE\n",
      "Nlrp3+/+ 330 338 GENOTYPE\n",
      "Nlrp3+/+ 384 392 GENOTYPE\n",
      "Nlrp3-/- 74 82 GENOTYPE\n",
      "Nlrp3-/- 436 444 GENOTYPE\n",
      "Nlrp3-/- 494 502 GENOTYPE\n",
      "WT 435 437 GENOTYPE\n",
      "Tle134-deficient 442 458 GENOTYPE\n",
      "without miR150 expression 192 217 GENOTYPE\n",
      "without miR150 expression 251 276 GENOTYPE\n",
      "wild type 1063 1072 GENOTYPE\n",
      "ENT3-/- 1077 1084 GENOTYPE\n",
      "STING-N153S 51 62 GENOTYPE\n",
      "STING-N153S 216 227 GENOTYPE\n",
      "Tbx3-KO 72 79 GENOTYPE\n",
      "Tbx3-KO 245 252 GENOTYPE\n",
      "WT 97 99 GENOTYPE\n",
      "WT 216 218 GENOTYPE\n",
      "CD38 KO 104 111 GENOTYPE\n",
      "CD38 KO 223 230 GENOTYPE\n",
      "WT 59 61 GENOTYPE\n",
      "WT 133 135 GENOTYPE\n",
      "SIRT7 KO 66 74 GENOTYPE\n",
      "SIRT7 KO 140 148 GENOTYPE\n",
      "miR-142 KO 199 209 GENOTYPE\n",
      "WT 484 486 GENOTYPE\n",
      "WT 706 708 GENOTYPE\n",
      "Six1kd 1583 1589 GENOTYPE\n",
      "WT 1354 1356 GENOTYPE\n",
      "Ikzf4-/- 1360 1368 GENOTYPE\n",
      "None 92 94 GENOTYPE\n",
      "Skipping entity\n",
      "WT 459 461 GENOTYPE\n",
      "WT 657 659 GENOTYPE\n",
      "None 156 159 GENOTYPE\n",
      "Skipping entity\n",
      "Malat1scr/scr 2718 2731 GENOTYPE\n",
      "WT 2739 2741 GENOTYPE\n",
      "WT 1245 1247 GENOTYPE\n",
      "Mrs2 KO 1252 1259 GENOTYPE\n",
      "PHD2-deltaT 812 823 GENOTYPE\n",
      "WT 828 830 GENOTYPE\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 24%|█████████▎                             | 429/1804 [00:01<00:04, 302.51it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "WT 1490 1492 GENOTYPE\n",
      "WT 963 965 GENOTYPE\n",
      "WT 1647 1649 GENOTYPE\n",
      "None 226 232 GENOTYPE\n",
      "Skipping entity\n",
      "female 400 406 GENOTYPE\n",
      "None 216 220 GENOTYPE\n",
      "Skipping entity\n",
      "None 228 232 GENOTYPE\n",
      "Skipping entity\n",
      "male 391 395 GENOTYPE\n",
      "None 402 406 GENOTYPE\n",
      "Skipping entity\n",
      "WT 805 807 GENOTYPE\n",
      "Lin41 knockdown 292 307 GENOTYPE\n",
      "Wild-type 1643 1652 GENOTYPE\n",
      "FXR KO 544 550 GENOTYPE\n",
      "FXR KO 1105 1111 GENOTYPE\n",
      "TDP43-WT 1509 1517 GENOTYPE\n",
      "TDP43-KQ 1497 1505 GENOTYPE\n",
      "WT 1789 1791 GENOTYPE\n",
      "Shisa9-/- 1306 1315 GENOTYPE\n",
      "Shisa9-/- 1796 1805 GENOTYPE\n",
      "Myh6-CreTg/0 855 867 GENOTYPE\n",
      "Myh6-CreTg/0 926 938 GENOTYPE\n",
      "Myh6-CreTg/0 1014 1026 GENOTYPE\n",
      "Myh6-CreTg/0; Zmynd8fl/fl 926 951 GENOTYPE\n",
      "Myh6-CreTg/0; Zmynd8fl/fl 1014 1039 GENOTYPE\n",
      "Rds 306 309 GENOTYPE\n",
      "Rds 463 466 GENOTYPE\n",
      "Rds 550 553 GENOTYPE\n",
      "Rds 705 708 GENOTYPE\n",
      "Rds 905 908 GENOTYPE\n",
      "WT 1127 1129 GENOTYPE\n",
      "WT 361 368 GENOTYPE\n",
      "WT 397 404 GENOTYPE\n",
      "KO 300 307 GENOTYPE\n",
      "KO 410 417 GENOTYPE\n",
      "Pten(i)pe-/- 124 136 GENOTYPE\n",
      "wildtype 168 176 GENOTYPE\n",
      "wildtype 246 254 GENOTYPE\n",
      "Rbm20 knockout 62 76 GENOTYPE\n",
      "Rbm20 knockout 181 195 GENOTYPE\n",
      "Rbm20 knockout 259 273 GENOTYPE\n",
      "None 516 523 GENOTYPE\n",
      "Skipping entity\n",
      "miR-15/16fl/fl 2677 2691 GENOTYPE\n",
      "Malat1scr/scr 2620 2633 GENOTYPE\n",
      "WT 2641 2643 GENOTYPE\n",
      "HDAC KO 766 773 GENOTYPE\n",
      "WT 610 612 GENOTYPE\n",
      "NOD 232 235 GENOTYPE\n",
      "NOD 598 601 GENOTYPE\n",
      "NOD 1263 1266 GENOTYPE\n",
      "WT 789 791 GENOTYPE\n",
      "WT 1703 1705 GENOTYPE\n",
      "WT 1762 1764 GENOTYPE\n",
      "KO 1712 1714 GENOTYPE\n",
      "KO 1771 1773 GENOTYPE\n",
      "K18-hACE2 510 519 GENOTYPE\n",
      "WT 1314 1316 GENOTYPE\n",
      "WT 103 105 GENOTYPE\n",
      "WT 204 206 GENOTYPE\n",
      "WT 359 361 GENOTYPE\n",
      "FVB/N 62 67 GENOTYPE\n",
      "FVB/N 397 402 GENOTYPE\n",
      "FVB/N 62 67 GENOTYPE\n",
      "FVB/N 397 402 GENOTYPE\n",
      "CD11c+ F480+ SiglecF+ WT 1355 1379 GENOTYPE\n",
      "cSiah1a/2f/f::Lyz2cre 1381 1402 GENOTYPE\n",
      "cSiah1a/2f/f::CD11ccre 1408 1430 GENOTYPE\n",
      "129X1/SvJ 1344 1353 GENOTYPE\n",
      "WT 112 114 GENOTYPE\n",
      "None 184 186 GENOTYPE\n",
      "Skipping entity\n",
      "ApoE4 581 586 GENOTYPE\n",
      "ApoE4 688 693 GENOTYPE\n",
      "ApoE4 1218 1223 GENOTYPE\n",
      "ApoE4 581 586 GENOTYPE\n",
      "ApoE4 688 693 GENOTYPE\n",
      "ApoE4 1218 1223 GENOTYPE\n",
      "ApoE4 581 586 GENOTYPE\n",
      "ApoE4 688 693 GENOTYPE\n",
      "ApoE4 1218 1223 GENOTYPE\n",
      "adipocyte-specific H1.2 knockout 91 123 GENOTYPE\n",
      "PD-1+ 38 43 GENOTYPE\n",
      "PD-1+ 71 76 GENOTYPE\n",
      "PD-1+ 293 298 GENOTYPE\n",
      "PD-1+ 430 435 GENOTYPE\n",
      "c-Myc null 64 74 GENOTYPE\n",
      "C57 738 741 GENOTYPE\n",
      "Nex-Cre; Foxg1fl/fl 821 840 GENOTYPE\n",
      "Foxg1fl/fl 806 816 GENOTYPE\n",
      "Foxg1fl/fl 830 840 GENOTYPE\n",
      "WT 1303 1305 GENOTYPE\n",
      "Kdm8 knockout 1310 1323 GENOTYPE\n",
      "None 229 233 GENOTYPE\n",
      "Skipping entity\n",
      "None 425 429 GENOTYPE\n",
      "Skipping entity\n",
      "C57BL/6J 515 523 GENOTYPE\n",
      "WT 355 357 GENOTYPE\n",
      "Fbxl11-CKO 329 339 GENOTYPE\n",
      "Wtapflox/flox 130 143 GENOTYPE\n",
      "Wtapflox/flox 740 753 GENOTYPE\n",
      "Wtapflox/flox 998 1011 GENOTYPE\n",
      "caMKL1 224 230 GENOTYPE\n",
      "WT 304 306 GENOTYPE\n",
      "WT 461 463 GENOTYPE\n",
      "KPCF 137 141 GENOTYPE\n",
      "KPCF 149 153 GENOTYPE\n",
      "Gli1Cre-ERT2; tdTomatoflox 2093 2119 GENOTYPE\n",
      "WT 1211 1213 GENOTYPE\n",
      "WT 1374 1376 GENOTYPE\n",
      "WT 1458 1460 GENOTYPE\n",
      "WT 0 2 GENOTYPE\n",
      "WT 34 36 GENOTYPE\n",
      "NKR-P1B KO 7 17 GENOTYPE\n",
      "NKR-P1B KO 41 51 GENOTYPE\n",
      "WT 1944 1946 GENOTYPE\n",
      "Tet1 KO 1961 1968 GENOTYPE\n",
      "Tet1 MUT 1948 1956 GENOTYPE\n",
      "Tet1 KO 1961 1968 GENOTYPE\n",
      "WT 1944 1946 GENOTYPE\n",
      "Tet1 MUT 1948 1956 GENOTYPE\n",
      "WT 41 43 GENOTYPE\n",
      "WT 176 178 GENOTYPE\n",
      "P815-M 988 994 GENOTYPE\n",
      "P815-M 1084 1090 GENOTYPE\n",
      "WT 499 501 GENOTYPE\n",
      "WT 737 739 GENOTYPE\n",
      "WT 1974 1976 GENOTYPE\n",
      "WT 2091 2093 GENOTYPE\n",
      "m.5024C>T 1241 1250 GENOTYPE\n",
      "NMDAR knockdown 239 254 GENOTYPE\n",
      "NMDAR knockdown 439 454 GENOTYPE\n",
      "MYC 169 175 GENOTYPE\n",
      "Cx3cr1-CreER:RPL22-HA 1187 1208 GENOTYPE\n",
      "Cx3cr1-CreER:RPL22-HA 1389 1410 GENOTYPE\n",
      "Cx3cr1-CreER:RPL22-HA:Ifngr ko/fl 1389 1422 GENOTYPE\n",
      "WT 493 495 GENOTYPE\n",
      "WT 748 750 GENOTYPE\n",
      "WT 987 989 GENOTYPE\n",
      "GPR174-KO 1119 1128 GENOTYPE\n",
      "WT 865 867 GENOTYPE\n",
      "WT 2030 2032 GENOTYPE\n",
      "/ex 721 726 GENOTYPE\n",
      "/ex 1231 1236 GENOTYPE\n",
      "/ex 2043 2048 GENOTYPE\n",
      "shNT 1134 1138 GENOTYPE\n",
      "shXBP1 1143 1149 GENOTYPE\n",
      "WT 1203 1205 GENOTYPE\n",
      "Thy1-ChR2-YFP heterozygous 1163 1189 GENOTYPE\n",
      "WT 37 39 GENOTYPE\n",
      "WT 162 164 GENOTYPE\n",
      "Nudt15+/+ 142 151 GENOTYPE\n",
      "Nudt15+/+ 314 323 GENOTYPE\n",
      "Nudt15R138C/R138C 171 188 GENOTYPE\n",
      "Nudt15R138C/R138C 328 345 GENOTYPE\n",
      "Semliki Forest virus 442 462 GENOTYPE\n",
      "EcsitPrx1;Rosa26mTmG 199 219 GENOTYPE\n",
      "Ecsitfl/fl;Rosa26mTmG 224 245 GENOTYPE\n",
      "ApoE-/-, Myh11-Cre, Rosa-TdTomato 225 258 GENOTYPE\n",
      "WT 847 849 GENOTYPE\n",
      "WT 844 846 GENOTYPE\n",
      "SERTfl/fl 432 441 GENOTYPE\n",
      "SERTPyramid 416 427 GENOTYPE\n",
      "Mettl3 flox/flox 76 92 GENOTYPE\n",
      "Mettl3 flox/flox 1178 1194 GENOTYPE\n",
      "wild type 150 159 GENOTYPE\n",
      "Tim-4 KO 164 172 GENOTYPE\n",
      "WT 1209 1211 GENOTYPE\n",
      "MacBlue 665 672 GENOTYPE\n",
      "WT 147 149 GENOTYPE\n",
      "Ino80 knockout 175 189 GENOTYPE\n",
      "WT 1485 1487 GENOTYPE\n",
      "cWT (Ncr1Cre) 1187 1200 GENOTYPE\n",
      "cKO (Bach2flox/flox Ncr1Cre) 1204 1232 GENOTYPE\n",
      "WT 1102 1104 GENOTYPE\n",
      "None 1188 1190 GENOTYPE\n",
      "Skipping entity\n",
      "WT 1292 1294 GENOTYPE\n",
      "WT 1413 1415 GENOTYPE\n",
      "WT 1456 1458 GENOTYPE\n",
      "WT 1508 1510 GENOTYPE\n",
      "Bach2-KO 1299 1307 GENOTYPE\n",
      "Bach2-KO 1420 1428 GENOTYPE\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 29%|███████████▌                           | 532/1804 [00:02<00:04, 314.56it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "None 120 122 GENOTYPE\n",
      "Skipping entity\n",
      "None 213 215 GENOTYPE\n",
      "Skipping entity\n",
      "None 306 308 GENOTYPE\n",
      "Skipping entity\n",
      "None 566 568 GENOTYPE\n",
      "Skipping entity\n",
      "KO-HBP1 1292 1299 GENOTYPE\n",
      "IMD 1623 1635 GENOTYPE\n",
      "Tyro3-/-V1 939 949 GENOTYPE\n",
      "Mertk-/-V2Tyro3-/-V2 954 974 GENOTYPE\n",
      "WT129 982 987 GENOTYPE\n",
      "Itgb5-/- 992 1000 GENOTYPE\n",
      "WT 722 724 GENOTYPE\n",
      "KO 457 459 GENOTYPE\n",
      "KO 734 736 GENOTYPE\n",
      "Runx1f/ 138 146 GENOTYPE\n",
      "Runx1f/ 183 191 GENOTYPE\n",
      "Runx1f/ 2337 2345 GENOTYPE\n",
      "Runx1f/ 2376 2384 GENOTYPE\n",
      "Runx1f/ 2654 2662 GENOTYPE\n",
      "Runx1f/ 2713 2721 GENOTYPE\n",
      "WT 2650 2652 GENOTYPE\n",
      "WT 2737 2739 GENOTYPE\n",
      "VHL-KO 80 86 GENOTYPE\n",
      "VHL-KO 91 97 GENOTYPE\n",
      "VHL-KO 323 329 GENOTYPE\n",
      "None 425 427 GENOTYPE\n",
      "Skipping entity\n",
      "None 674 676 GENOTYPE\n",
      "Skipping entity\n",
      "None 775 777 GENOTYPE\n",
      "Skipping entity\n",
      "None 1517 1519 GENOTYPE\n",
      "Skipping entity\n",
      "WT 821 823 GENOTYPE\n",
      "Nf1 flox/flox PostnCre+ 1839 1862 GENOTYPE\n",
      "ko 287 289 GENOTYPE\n",
      "ko 331 333 GENOTYPE\n",
      "Prox1-GFP 1041 1050 GENOTYPE\n",
      "CD74-tdTomato 541 554 GENOTYPE\n",
      "B2m-tdTomato 516 528 GENOTYPE\n",
      "WT 1913 1915 GENOTYPE\n",
      "TG 138 140 GENOTYPE\n",
      "TG 322 324 GENOTYPE\n",
      "Rag1-/-Ox40-Cre 120 135 GENOTYPE\n",
      "Rag1-/-Ox40-Cre 393 408 GENOTYPE\n",
      "Rag1-/-Brd4f/fOx40-Cre 140 162 GENOTYPE\n",
      "Rag1-/-Brd4f/fOx40-Cre 354 376 GENOTYPE\n",
      "MYC-CTNNB1 462 472 GENOTYPE\n",
      "MYC-Tp53 498 506 GENOTYPE\n",
      "None 1116 1119 GENOTYPE\n",
      "Skipping entity\n",
      "miR652-KD 1144 1153 GENOTYPE\n",
      "Vector 1120 1126 GENOTYPE\n",
      "PUS10-OE 1131 1139 GENOTYPE\n",
      "Vector 1116 1122 GENOTYPE\n",
      "PUS10-OE 1127 1135 GENOTYPE\n",
      "None 1208 1211 GENOTYPE\n",
      "Skipping entity\n",
      "miR139-KD 1236 1245 GENOTYPE\n",
      "Pus10-WT 1318 1326 GENOTYPE\n",
      "Pus10-KO 1331 1339 GENOTYPE\n",
      "WT 781 783 GENOTYPE\n",
      "WT 1158 1160 GENOTYPE\n",
      "WT 1412 1414 GENOTYPE\n",
      "WT 1666 1668 GENOTYPE\n",
      "WT 1811 1813 GENOTYPE\n",
      "WT 1888 1890 GENOTYPE\n",
      "WT 184 186 GENOTYPE\n",
      "WT 292 294 GENOTYPE\n",
      "WT 316 318 GENOTYPE\n",
      "CXCL13-TdTomato+ 191 207 GENOTYPE\n",
      "CXCL13-TdTomato+ 623 639 GENOTYPE\n",
      "Ccm1fl/fl 287 296 GENOTYPE\n",
      "Ccm1fl/fl 1192 1201 GENOTYPE\n",
      "/fl 1225 1234 GENOTYPE\n",
      "Cdh5Cre/+ERT2;Ccm1fl/fl 1211 1234 GENOTYPE\n",
      "wildtype 1098 1106 GENOTYPE\n",
      "ST3GAL3 knockdown 465 482 GENOTYPE\n",
      "ST3GAL3 knockdown 569 586 GENOTYPE\n",
      "ST3GAL3 knockdown 833 850 GENOTYPE\n",
      "ST3GAL3 knockdown 1035 1052 GENOTYPE\n",
      "TASTPM 228 234 GENOTYPE\n",
      "TASTPM 567 573 GENOTYPE\n",
      "TASTPM 962 968 GENOTYPE\n",
      "TASTPM 1138 1144 GENOTYPE\n",
      "WT 371 373 GENOTYPE\n",
      "WT 763 765 GENOTYPE\n",
      "WT 1131 1133 GENOTYPE\n",
      "C57L/6JRj 250 259 GENOTYPE\n",
      "ciRS-7 KO 218 227 GENOTYPE\n",
      "ciRS-7 KO 325 334 GENOTYPE\n",
      "Flcn-KO 91 98 GENOTYPE\n",
      "WT 416 418 GENOTYPE\n",
      "Y551A 1512 1517 GENOTYPE\n",
      "Y551A 1519 1524 GENOTYPE\n",
      "Y551A-Dim 1519 1528 GENOTYPE\n",
      "D647V 1530 1535 GENOTYPE\n",
      "D647V 1537 1542 GENOTYPE\n",
      "WT 2389 2391 GENOTYPE\n",
      "IFNgR1-/- 362 371 GENOTYPE\n",
      "IFNgR1-/- 503 512 GENOTYPE\n",
      "STAT1-/- 376 384 GENOTYPE\n",
      "STAT1-/- 489 497 GENOTYPE\n",
      "Wild-type 478 487 GENOTYPE\n",
      "Klf4 cKO 427 435 GENOTYPE\n",
      "Klf4 cKO 641 649 GENOTYPE\n",
      "control 285 292 GENOTYPE\n",
      "control 382 389 GENOTYPE\n",
      "WT 236 238 GENOTYPE\n",
      "WT 403 405 GENOTYPE\n",
      "WT 602 604 GENOTYPE\n",
      "Rcrin knockout 366 380 GENOTYPE\n",
      "Rcrin knockout 565 579 GENOTYPE\n",
      "Klf13- 512 520 GENOTYPE\n",
      "Serpine1 overexpression 61 84 GENOTYPE\n",
      "Smarcb11148del/1148del 817 839 GENOTYPE\n",
      "PyMT 1168 1172 GENOTYPE\n",
      "PyMT 1297 1301 GENOTYPE\n",
      "None 1326 1330 GENOTYPE\n",
      "Skipping entity\n",
      "None 1348 1352 GENOTYPE\n",
      "Skipping entity\n",
      "Cas9 304 308 GENOTYPE\n",
      "Cas9 487 491 GENOTYPE\n",
      "Cas9 1174 1178 GENOTYPE\n",
      "Cas9 1446 1450 GENOTYPE\n",
      "Cas9 2119 2123 GENOTYPE\n",
      "Cas9 2196 2200 GENOTYPE\n",
      "Lipe-AKO; Cas9 2186 2200 GENOTYPE\n",
      "WT 266 268 GENOTYPE\n",
      "WT 1129 1131 GENOTYPE\n",
      "Hom 258 261 GENOTYPE\n",
      "Hom 830 833 GENOTYPE\n",
      "Hom 1149 1152 GENOTYPE\n",
      "WT 1862 1864 GENOTYPE\n",
      "WT 2166 2168 GENOTYPE\n",
      "WT 429 431 GENOTYPE\n",
      "WT 536 538 GENOTYPE\n",
      "WT 2040 2042 GENOTYPE\n",
      "MIF-KO 2052 2058 GENOTYPE\n",
      "human 1269 1274 GENOTYPE\n",
      "human 1287 1292 GENOTYPE\n",
      "BABL/c mouse 485 497 GENOTYPE\n",
      "Zeb1-deleted 301 313 GENOTYPE\n",
      "wild-type 166 175 GENOTYPE\n",
      "littermate 109 127 GENOTYPE\n",
      "littermate 357 375 GENOTYPE\n",
      "littermate control 1202 1220 GENOTYPE\n",
      "littermate control 1241 1259 GENOTYPE\n",
      "5xFAD 971 976 GENOTYPE\n",
      "5xFAD 1915 1920 GENOTYPE\n",
      "5xFAD 2047 2052 GENOTYPE\n",
      "5xFAD 2058 2063 GENOTYPE\n",
      "p16-tdTomato mice 156 173 GENOTYPE\n",
      "Ifnar1-/- 1624 1633 GENOTYPE\n",
      "VPR;Pax8-rtTa 335 348 GENOTYPE\n",
      "VPR;Pax8-rtTa 475 488 GENOTYPE\n",
      "Flt3ITD/ITD 1205 1216 GENOTYPE\n",
      "Scl-CreERT:Flt3ITD/ITD 1194 1216 GENOTYPE\n",
      "Scl-CreERT:Flt3ITD/ITD 1234 1256 GENOTYPE\n",
      "Flt3ITD/ITD 1202 1213 GENOTYPE\n",
      "Flt3ITD/ITD 1215 1226 GENOTYPE\n",
      "Mx1-Cre:Flt3ITD/ITD 1194 1213 GENOTYPE\n",
      "Flt3ITD/ITD 1205 1216 GENOTYPE\n",
      "Scl-CreERT 1194 1204 GENOTYPE\n",
      "Scl-CreERT:Flt3ITD/ITD 1194 1216 GENOTYPE\n",
      "CD21-Cre Atr+/C 1088 1103 GENOTYPE\n",
      "CD21-Cre AtrC/- 1123 1138 GENOTYPE\n",
      "WT 169 171 GENOTYPE\n",
      "WT 343 345 GENOTYPE\n",
      "WT 448 450 GENOTYPE\n",
      "WT 500 502 GENOTYPE\n",
      "Susd2-/- 176 184 GENOTYPE\n",
      "Susd2-/- 349 357 GENOTYPE\n",
      "Susd2-/- 468 476 GENOTYPE\n",
      "Susd2-/- 520 528 GENOTYPE\n",
      "wild type 179 188 GENOTYPE\n",
      "wild type 281 290 GENOTYPE\n",
      "WT 1111 1113 GENOTYPE\n",
      "Rik overexpression 677 695 GENOTYPE\n",
      "Rik overexpression 745 763 GENOTYPE\n",
      "Rik overexpression 1503 1521 GENOTYPE\n",
      "WT 192 194 GENOTYPE\n",
      "WT 296 298 GENOTYPE\n",
      "wild-type 170 179 GENOTYPE\n",
      "wild-type 633 642 GENOTYPE\n",
      "Ccm3 knock-out 267 281 GENOTYPE\n",
      "/flox 1142 1154 GENOTYPE\n",
      "Id2flox/flox 1181 1193 GENOTYPE\n",
      "WT 1995 1997 GENOTYPE\n",
      "B6 123 125 GENOTYPE\n",
      "B6 404 406 GENOTYPE\n",
      "B6 675 677 GENOTYPE\n",
      "B6 1239 1241 GENOTYPE\n",
      "B6 1516 1518 GENOTYPE\n",
      "B6 1727 1729 GENOTYPE\n",
      "B6 2787 2789 GENOTYPE\n",
      "NZO 144 147 GENOTYPE\n",
      "NZO 2835 2838 GENOTYPE\n",
      "NZO 2846 2849 GENOTYPE\n",
      "WSB 215 218 GENOTYPE\n",
      "WSB 2887 2890 GENOTYPE\n",
      "WSB 2896 2899 GENOTYPE\n",
      "CAST 187 191 GENOTYPE\n",
      "CAST 2852 2856 GENOTYPE\n",
      "CAST 2862 2866 GENOTYPE\n",
      "NOD 128 131 GENOTYPE\n",
      "NOD 2820 2823 GENOTYPE\n",
      "NOD 2829 2832 GENOTYPE\n",
      "PWK 197 200 GENOTYPE\n",
      "PWK 206 209 GENOTYPE\n",
      "PWK 542 545 GENOTYPE\n",
      "PWK 682 685 GENOTYPE\n",
      "PWK 1523 1526 GENOTYPE\n",
      "PWK 1731 1734 GENOTYPE\n",
      "PWK 2869 2872 GENOTYPE\n",
      "PWK 2878 2881 GENOTYPE\n",
      "WT 685 687 GENOTYPE\n",
      "WT 865 867 GENOTYPE\n",
      "WT 874 876 GENOTYPE\n",
      "WT 1130 1132 GENOTYPE\n",
      "WT 344 346 GENOTYPE\n",
      "WT 638 640 GENOTYPE\n",
      "ApoA4 knockdown 619 634 GENOTYPE\n",
      "CD200R-/- 454 463 GENOTYPE\n",
      "CD200R-/- 610 619 GENOTYPE\n",
      "CD200R-/- 896 905 GENOTYPE\n",
      "CD200R-/- 1569 1578 GENOTYPE\n",
      "Jmjd4 knockdown 103 118 GENOTYPE\n",
      "control 369 376 GENOTYPE\n",
      "control 466 473 GENOTYPE\n",
      "control 267 274 GENOTYPE\n",
      "control 444 451 GENOTYPE\n",
      "Ovol1/2 iDKO 35 47 GENOTYPE\n",
      "2afl/fl 534 541 GENOTYPE\n",
      "2afl/fl 940 947 GENOTYPE\n",
      "2afl/fl 1064 1071 GENOTYPE\n",
      "2aOKO 387 392 GENOTYPE\n",
      "2aOKO 952 957 GENOTYPE\n",
      "2aOKO 1099 1104 GENOTYPE\n",
      "RUNX1 WT 357 365 GENOTYPE\n",
      "RUNX1 KO 493 501 GENOTYPE\n",
      "Fltp iCre mTmG 271 285 GENOTYPE\n",
      "wild type 82 91 GENOTYPE\n",
      "P2Y4 KO 101 108 GENOTYPE\n",
      "P2Y4 KO 345 352 GENOTYPE\n",
      "P2Y4 KO 684 691 GENOTYPE\n",
      "Pax7-YFP 407 415 GENOTYPE\n",
      "MyoD-KI 13 20 GENOTYPE\n",
      "INTS11Halo 1204 1214 GENOTYPE\n",
      "Arr1-/- 591 598 GENOTYPE\n",
      "Arr1-/- 2663 2670 GENOTYPE\n",
      "SBE1/5 1260 1266 GENOTYPE\n",
      "Selenow knock-out 55 72 GENOTYPE\n",
      "Selenow knock-out 288 305 GENOTYPE\n",
      "/f 198 201 GENOTYPE\n",
      "/f 335 338 GENOTYPE\n",
      "/d 212 215 GENOTYPE\n",
      "/d 349 352 GENOTYPE\n",
      "Rosa-Stopfl/fl-Flag-Ago2 330 354 GENOTYPE\n",
      "WT 341 343 GENOTYPE\n",
      "PD-1-/- 348 355 GENOTYPE\n",
      "WT 42 44 GENOTYPE\n",
      "WT 173 175 GENOTYPE\n",
      "WT 352 354 GENOTYPE\n",
      "WT/IL-17A-EGFP 1116 1130 GENOTYPE\n",
      "wild type 1788 1797 GENOTYPE\n",
      "wild type 390 399 GENOTYPE\n",
      "wild type 951 960 GENOTYPE\n",
      "wild type 998 1007 GENOTYPE\n",
      "Mettl3 cko ctrl 1045 1060 GENOTYPE\n",
      "Mettl3 cko 1045 1055 GENOTYPE\n",
      "Mettl3 cko 1094 1104 GENOTYPE\n",
      "wild type 99 108 GENOTYPE\n",
      "MMTV-Neu 349 357 GENOTYPE\n",
      "MMTV-Neu 524 532 GENOTYPE\n",
      "sgKdm6a RPP 958 969 GENOTYPE\n",
      "2afl/fl 725 732 GENOTYPE\n",
      "2aOKO 333 338 GENOTYPE\n",
      "2aOKO 737 742 GENOTYPE\n",
      "SIRPA knockdown 228 243 GENOTYPE\n",
      "SIRPA knockdown 337 352 GENOTYPE\n",
      "WT 256 258 GENOTYPE\n",
      "ARE-/- 265 271 GENOTYPE\n",
      "None 197 199 GENOTYPE\n",
      "Skipping entity\n",
      "ko 281 283 GENOTYPE\n",
      "miR-142-null 1093 1105 GENOTYPE\n",
      "miR-142-null 1555 1567 GENOTYPE\n",
      "miR-142 1007 1014 GENOTYPE\n",
      "miR-142 1053 1060 GENOTYPE\n",
      "miR-142 1093 1100 GENOTYPE\n",
      "miR-142 1125 1132 GENOTYPE\n",
      "miR-142 1555 1562 GENOTYPE\n",
      "miR-142 1662 1669 GENOTYPE\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 33%|████████████▉                          | 596/1804 [00:02<00:04, 295.78it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "hTNFtg 205 211 GENOTYPE\n",
      "hTNFtg 257 263 GENOTYPE\n",
      "hTNFtg 209 215 GENOTYPE\n",
      "hTNFtg 261 267 GENOTYPE\n",
      "STAT3 47 52 GENOTYPE\n",
      "STAT3 181 186 GENOTYPE\n",
      "STAT3 333 338 GENOTYPE\n",
      "STAT3 415 420 GENOTYPE\n",
      "STAT3 456 461 GENOTYPE\n",
      "STAT3 665 670 GENOTYPE\n",
      "None 674 679 GENOTYPE\n",
      "Skipping entity\n",
      "STAT3 953 958 GENOTYPE\n",
      "None 960 965 GENOTYPE\n",
      "Skipping entity\n",
      "STAT3Y640F 674 684 GENOTYPE\n",
      "STAT3Y640F 960 970 GENOTYPE\n",
      "APLAID mutant 780 793 GENOTYPE\n",
      "APLAID mutant 924 937 GENOTYPE\n",
      "db/db 492 497 GENOTYPE\n",
      "db/db 699 704 GENOTYPE\n",
      "db/m 709 713 GENOTYPE\n",
      "db/db 492 497 GENOTYPE\n",
      "db/db 699 704 GENOTYPE\n",
      "db/m 709 713 GENOTYPE\n",
      "None 788 791 GENOTYPE\n",
      "Skipping entity\n",
      "None 2050 2053 GENOTYPE\n",
      "Skipping entity\n",
      "None 2187 2190 GENOTYPE\n",
      "Skipping entity\n",
      "None 407 410 GENOTYPE\n",
      "Skipping entity\n",
      "None 592 595 GENOTYPE\n",
      "Skipping entity\n",
      "None 775 778 GENOTYPE\n",
      "Skipping entity\n",
      "None 2037 2040 GENOTYPE\n",
      "Skipping entity\n",
      "None 2089 2092 GENOTYPE\n",
      "Skipping entity\n",
      "None 2174 2177 GENOTYPE\n",
      "Skipping entity\n",
      "WT 1363 1365 GENOTYPE\n",
      "WT 1486 1488 GENOTYPE\n",
      "WT 1629 1631 GENOTYPE\n",
      "None 1633 1635 GENOTYPE\n",
      "Skipping entity\n",
      "Sod1f/f 183 190 GENOTYPE\n",
      "Sod1f/f 300 307 GENOTYPE\n",
      "Sod1f/ 312 319 GENOTYPE\n",
      "Sod1f/f;Vil-creERT2 312 331 GENOTYPE\n",
      "KI 597 599 GENOTYPE\n",
      "KI 1546 1548 GENOTYPE\n",
      "TTPKO 397 402 GENOTYPE\n",
      "TTPKO 486 491 GENOTYPE\n",
      "TTPKO 512 517 GENOTYPE\n",
      "None 520 525 GENOTYPE\n",
      "Skipping entity\n",
      "None 1083 1088 GENOTYPE\n",
      "Skipping entity\n",
      "TTPKO 1093 1098 GENOTYPE\n",
      "TTPKO 1176 1181 GENOTYPE\n",
      "None 1427 1432 GENOTYPE\n",
      "Skipping entity\n",
      "TTPKO 1437 1442 GENOTYPE\n",
      "None 1564 1569 GENOTYPE\n",
      "Skipping entity\n",
      "TTPKO 1574 1579 GENOTYPE\n",
      "TTPKO 2128 2133 GENOTYPE\n",
      "TTPKO 2156 2161 GENOTYPE\n",
      "None 2164 2169 GENOTYPE\n",
      "Skipping entity\n",
      "cTTPKO 519 525 GENOTYPE\n",
      "cTTPKO 1082 1088 GENOTYPE\n",
      "cTTPKO 1426 1432 GENOTYPE\n",
      "cTTPKO 1563 1569 GENOTYPE\n",
      "cTTPKO 2163 2169 GENOTYPE\n",
      "C57BL6/J (B6) 106 119 GENOTYPE\n",
      "PGC-1beta muscle-specific knockout 398 432 GENOTYPE\n",
      "wt 1798 1800 GENOTYPE\n",
      "wt 1884 1886 GENOTYPE\n",
      "huTNFtg 624 631 GENOTYPE\n",
      "huTNFtg 758 765 GENOTYPE\n",
      "huTNFtg 1802 1809 GENOTYPE\n",
      "None 1832 1839 GENOTYPE\n",
      "Skipping entity\n",
      "TgColVI-miR-221/222 1811 1830 GENOTYPE\n",
      "-miR-221/222 1840 1859 GENOTYPE\n",
      "huTNFtg;TgColVI-miR-221/222 1832 1859 GENOTYPE\n",
      "Irg1+/+ 1229 1236 GENOTYPE\n",
      "Irg1-/- 1241 1248 GENOTYPE\n",
      "C57Bl6 2141 2147 GENOTYPE\n",
      "TCF/LEF1::H2B::eGFP 1171 1190 GENOTYPE\n",
      "None 1226 1231 GENOTYPE\n",
      "Skipping entity\n",
      "None 1551 1556 GENOTYPE\n",
      "Skipping entity\n",
      "Wild-type 1853 1862 GENOTYPE\n",
      "db/+ 1617 1621 GENOTYPE\n",
      "db/db 1541 1546 GENOTYPE\n",
      "wt 998 1000 GENOTYPE\n",
      "BCL7A KO 982 990 GENOTYPE\n",
      "WT 1135 1137 GENOTYPE\n",
      "B6C3F1 399 405 GENOTYPE\n",
      "KrasG12D; sgP53; dS 1494 1513 GENOTYPE\n",
      "KrasG12D; sgP53; dL 1517 1536 GENOTYPE\n",
      "Pantr2 knockout 214 229 GENOTYPE\n",
      "None 572 574 GENOTYPE\n",
      "Skipping entity\n",
      "WT 1121 1123 GENOTYPE\n",
      "ST2ko 1124 1129 GENOTYPE\n",
      "WT 1097 1099 GENOTYPE\n",
      "ST2ko 1108 1113 GENOTYPE\n",
      "intergenic 218 228 GENOTYPE\n",
      "Snx9 KO 233 240 GENOTYPE\n",
      "C57/Bl6 1775 1782 GENOTYPE\n",
      "Efemp1ki/ki 51 62 GENOTYPE\n",
      "Efemp1ki/ki 389 400 GENOTYPE\n",
      "Efemp1ki/ki 593 604 GENOTYPE\n",
      "None 1217 1219 GENOTYPE\n",
      "Skipping entity\n",
      "FF 1361 1363 GENOTYPE\n",
      "LKO 1410 1413 GENOTYPE\n",
      "PD-L2-KO 1551 1559 GENOTYPE\n",
      "WT 946 948 GENOTYPE\n",
      "mdm 130 133 GENOTYPE\n",
      "mdm 373 376 GENOTYPE\n",
      "mdm 615 618 GENOTYPE\n",
      "mdm 978 981 GENOTYPE\n",
      "UBA6 knockout 1112 1125 GENOTYPE\n",
      "Foxo1TKO 134 142 GENOTYPE\n",
      "Foxo1Ctrl 167 176 GENOTYPE\n",
      "WT 329 331 GENOTYPE\n",
      "WT 508 510 GENOTYPE\n",
      "WT 494 496 GENOTYPE\n",
      "KO 509 511 GENOTYPE\n",
      "C57BL/6 544 551 GENOTYPE\n",
      "euploid 1217 1224 GENOTYPE\n",
      "euploid 1460 1467 GENOTYPE\n",
      "Ts65Dn 1126 1132 GENOTYPE\n",
      "Ts65Dn 1206 1212 GENOTYPE\n",
      "Ts65Dn 1449 1455 GENOTYPE\n",
      "None 1472 1478 GENOTYPE\n",
      "Skipping entity\n",
      "C57BL/6 671 678 GENOTYPE\n",
      "Rosa-Cas9/Cas9 215 229 GENOTYPE\n",
      "Mrfta-/-; Mrtfb-F/F; Rosa-Cas9/Cas9 194 229 GENOTYPE\n",
      "C57BL/6J 22 30 GENOTYPE\n",
      "B6D2F1N 35 42 GENOTYPE\n",
      "wild type 90 99 GENOTYPE\n",
      "wild type 215 224 GENOTYPE\n",
      "(mSmn+/-, hSMN22tg/0) 364 385 GENOTYPE\n",
      "(mSmn-/-, hSMN22tg/0) 323 344 GENOTYPE\n",
      "WT 76 78 GENOTYPE\n",
      "None 84 92 GENOTYPE\n",
      "Skipping entity\n",
      "NUP98-NSD1 207 217 GENOTYPE\n",
      "NUP98-NSD1 249 259 GENOTYPE\n",
      "NUP98-NSD1 617 627 GENOTYPE\n",
      "WT 167 169 GENOTYPE\n",
      "WT 339 341 GENOTYPE\n",
      "Vegfr2Y949F 174 185 GENOTYPE\n",
      "Vegfr2Y949F 347 358 GENOTYPE\n",
      "Ythdc1flox/flox 163 178 GENOTYPE\n",
      "Ythdc1flox/flox 427 442 GENOTYPE\n",
      "Ythdc1flox/flox 665 680 GENOTYPE\n",
      "homozygous 1817 1827 GENOTYPE\n",
      "wt 1427 1429 GENOTYPE\n",
      "ICR 320 323 GENOTYPE\n",
      "WT 1465 1467 GENOTYPE\n",
      "Aldh2 & Fancd2 double knock-outs 1522 1554 GENOTYPE\n",
      "p53 knock-outs 1506 1520 GENOTYPE\n",
      "Aldh2 knock-outs 1469 1485 GENOTYPE\n",
      "Fancd2 knock-outs 1487 1504 GENOTYPE\n",
      "Aldh2 & Fancd2 & p53 triple knock-outs 1556 1594 GENOTYPE\n",
      "Alb-Cre; Rosa-tdTomato 1387 1409 GENOTYPE\n",
      "wild-type 679 688 GENOTYPE\n",
      "wild-type 1176 1185 GENOTYPE\n",
      "wild-type 1456 1465 GENOTYPE\n",
      "Notch3-deficient 1475 1491 GENOTYPE\n",
      "Notch3-deficient 1506 1522 GENOTYPE\n",
      "wild-type 287 296 GENOTYPE\n",
      "WT 212 214 GENOTYPE\n",
      "NFAT5 KO 39 47 GENOTYPE\n",
      "NFAT5 KO 246 254 GENOTYPE\n",
      "Ldlr-/- 391 398 GENOTYPE\n",
      "Ldlr-/- 1052 1059 GENOTYPE\n",
      "WT 1057 1059 GENOTYPE\n",
      "EID 1075 1078 GENOTYPE\n",
      "EID 1079 1082 GENOTYPE\n",
      "EID 1084 1087 GENOTYPE\n",
      "WT 123 125 GENOTYPE\n",
      "WT 219 221 GENOTYPE\n",
      "WT 311 313 GENOTYPE\n",
      "WT 590 592 GENOTYPE\n",
      "WT 123 125 GENOTYPE\n",
      "WT 277 279 GENOTYPE\n",
      "WT 373 375 GENOTYPE\n",
      "WT 642 644 GENOTYPE\n",
      "Tgfbr2 KO 130 139 GENOTYPE\n",
      "Tgfbr2 KO 313 322 GENOTYPE\n",
      "Tgfbr2 KO 380 389 GENOTYPE\n",
      "Tgfbr2 KO 84 93 GENOTYPE\n",
      "Tgfbr2 KO 270 279 GENOTYPE\n",
      "Tgfbr2 KO 339 348 GENOTYPE\n",
      "WT 77 79 GENOTYPE\n",
      "WT 234 236 GENOTYPE\n",
      "WT 332 334 GENOTYPE\n",
      "WT 532 534 GENOTYPE\n",
      "flox/flox 1264 1273 GENOTYPE\n",
      "flox/flox 1308 1317 GENOTYPE\n",
      "WT 1261 1263 GENOTYPE\n",
      "APOE2 160 165 GENOTYPE\n",
      "APOE2 306 311 GENOTYPE\n",
      "APOE2 484 489 GENOTYPE\n",
      "APOE2 626 631 GENOTYPE\n",
      "APOE3 170 175 GENOTYPE\n",
      "APOE3 316 321 GENOTYPE\n",
      "APOE3 647 652 GENOTYPE\n",
      "Emx1-NuTRAP 441 452 GENOTYPE\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 39%|███████████████                        | 696/1804 [00:02<00:03, 314.62it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Esrrb WT 69 77 GENOTYPE\n",
      "WT 1716 1718 GENOTYPE\n",
      "Cnp-/- 1180 1186 GENOTYPE\n",
      "Cnp-/- 1720 1726 GENOTYPE\n",
      "None 1735 1741 GENOTYPE\n",
      "Skipping entity\n",
      "5xFAD 1728 1733 GENOTYPE\n",
      "None 1742 1747 GENOTYPE\n",
      "Skipping entity\n",
      "Ddx5f/f 180 187 GENOTYPE\n",
      "/f 195 202 GENOTYPE\n",
      "Ddx5f/f 327 334 GENOTYPE\n",
      "/f 342 349 GENOTYPE\n",
      "K14Ddx5f/f 192 202 GENOTYPE\n",
      "K14Ddx5f/f 339 349 GENOTYPE\n",
      "Ddx5f/f 171 178 GENOTYPE\n",
      "/f 186 193 GENOTYPE\n",
      "K14Ddx5f/f 183 193 GENOTYPE\n",
      "(mSmn-/-, hSMN22tg/0) 641 662 GENOTYPE\n",
      "(mSmn+/-, hSMN22tg/0) 682 703 GENOTYPE\n",
      "Foxp3YFP-Cre 289 301 GENOTYPE\n",
      "-Cre 320 332 GENOTYPE\n",
      "Il23rflflFoxp3YFP-Cre 311 332 GENOTYPE\n",
      "Osr2-Cre;b-cateninfl/fl 123 146 GENOTYPE\n",
      "Osr2-Cre;b-cateninfl/fl 206 229 GENOTYPE\n",
      "None 471 474 GENOTYPE\n",
      "Skipping entity\n",
      "None 633 636 GENOTYPE\n",
      "Skipping entity\n",
      "None 666 669 GENOTYPE\n",
      "Skipping entity\n",
      "None 876 879 GENOTYPE\n",
      "Skipping entity\n",
      "NSG 109 112 GENOTYPE\n",
      "NSG 450 453 GENOTYPE\n",
      "WT 265 267 GENOTYPE\n",
      "WT 461 463 GENOTYPE\n",
      "WT 476 478 GENOTYPE\n",
      "LysM 1836 1840 GENOTYPE\n",
      "K2KO 1891 1895 GENOTYPE\n",
      "Usp2 sufficient 1252 1267 GENOTYPE\n",
      "Usp2 defficient 1286 1301 GENOTYPE\n",
      "Aire KO 1143 1150 GENOTYPE\n",
      "Aire KO 1171 1178 GENOTYPE\n",
      "Aire KO 1201 1208 GENOTYPE\n",
      "CaSR knockout 218 231 GENOTYPE\n",
      "miR-375 knockout 1502 1518 GENOTYPE\n",
      "PP2A WT 242 249 GENOTYPE\n",
      "PP2A WT 408 415 GENOTYPE\n",
      "WT 137 139 GENOTYPE\n",
      "WT 102 104 GENOTYPE\n",
      "Mesp1 KO 1478 1486 GENOTYPE\n",
      "Prl2+/+ 187 194 GENOTYPE\n",
      "Prl2+/+ 402 409 GENOTYPE\n",
      "Prl2-/- 199 206 GENOTYPE\n",
      "Prl2-/- 414 421 GENOTYPE\n",
      "Prl2-/- 520 527 GENOTYPE\n",
      "KLB-Cre::Ai14 1150 1163 GENOTYPE\n",
      "Filip1l KD 417 427 GENOTYPE\n",
      "Vipr2-/- 623 631 GENOTYPE\n",
      "Vipr2-/- 853 861 GENOTYPE\n",
      "Vipr2-/- 983 991 GENOTYPE\n",
      "Vipr2-/- 1224 1232 GENOTYPE\n",
      "Vipr2-/- 1341 1349 GENOTYPE\n",
      "Vipr2-/- 1673 1681 GENOTYPE\n",
      "Vipr2-/- 1703 1711 GENOTYPE\n",
      "eNOS+/+ 171 178 GENOTYPE\n",
      "eNOS+/+ 258 265 GENOTYPE\n",
      "eNOS-/- 183 190 GENOTYPE\n",
      "eNOS-/- 270 277 GENOTYPE\n",
      "eNOS-/- 282 289 GENOTYPE\n",
      "eNOS+/+ 164 171 GENOTYPE\n",
      "eNOS+/+ 295 302 GENOTYPE\n",
      "eNOS+/- 176 183 GENOTYPE\n",
      "eNOS+/- 307 314 GENOTYPE\n",
      "Pkd1RC/RC 165 174 GENOTYPE\n",
      "WT 134 136 GENOTYPE\n",
      "WT 229 231 GENOTYPE\n",
      "WT 761 763 GENOTYPE\n",
      "WT 1735 1737 GENOTYPE\n",
      "None 144 145 GENOTYPE\n",
      "Skipping entity\n",
      "A 150 151 GENOTYPE\n",
      "None 202 203 GENOTYPE\n",
      "Skipping entity\n",
      "None 234 235 GENOTYPE\n",
      "Skipping entity\n",
      "None 291 292 GENOTYPE\n",
      "Skipping entity\n",
      "None 338 339 GENOTYPE\n",
      "Skipping entity\n",
      "None 383 384 GENOTYPE\n",
      "Skipping entity\n",
      "None 496 497 GENOTYPE\n",
      "Skipping entity\n",
      "None 523 524 GENOTYPE\n",
      "Skipping entity\n",
      "None 555 556 GENOTYPE\n",
      "Skipping entity\n",
      "None 620 621 GENOTYPE\n",
      "Skipping entity\n",
      "None 143 144 GENOTYPE\n",
      "Skipping entity\n",
      "K 147 148 GENOTYPE\n",
      "None 304 305 GENOTYPE\n",
      "Skipping entity\n",
      "None 323 324 GENOTYPE\n",
      "Skipping entity\n",
      "None 615 616 GENOTYPE\n",
      "Skipping entity\n",
      "KA 143 145 GENOTYPE\n",
      "WT 153 155 GENOTYPE\n",
      "WT 287 289 GENOTYPE\n",
      "control 408 415 GENOTYPE\n",
      "control 514 521 GENOTYPE\n",
      "control 882 889 GENOTYPE\n",
      "DSS 328 331 GENOTYPE\n",
      "control (Mafg+/-:Mafk+/-) 2129 2154 GENOTYPE\n",
      "compound (Mafg-/-:Mafk+/-) 2156 2182 GENOTYPE\n",
      "double KO (Mafg-/-:Mafk-/-) 2187 2214 GENOTYPE\n",
      "Col1a1-GFP/Tie2-cre/Rosa- 49 82 GENOTYPE\n",
      "wild type 779 788 GENOTYPE\n",
      "sgC 1015 1018 GENOTYPE\n",
      "sgPrkci/Prkcz 1023 1036 GENOTYPE\n",
      "Prkcif/fPrkczf/f;Villin-Cre 1015 1042 GENOTYPE\n",
      "sgPrkci/Prkcz 1015 1028 GENOTYPE\n",
      "Prkcif/fPrkczf/f;Villin-Cre 1015 1042 GENOTYPE\n",
      "WT 154 156 GENOTYPE\n",
      "WT 1480 1482 GENOTYPE\n",
      "None 1803 1805 GENOTYPE\n",
      "Skipping entity\n",
      "WT 1901 1903 GENOTYPE\n",
      "SIK1f/f;SIK2f/f,SIK3f/f 1584 1607 GENOTYPE\n",
      "Pd1CKO 251 257 GENOTYPE\n",
      "Pd1CKO 619 625 GENOTYPE\n",
      "Myb-68 GFP+ 147 158 GENOTYPE\n",
      "Myb-68 GFP+ 147 158 GENOTYPE\n",
      "Pparg+/+ Foxp3-Cre 248 266 GENOTYPE\n",
      "Ppargf/f Foxp3-Cre 123 141 GENOTYPE\n",
      "Ppargf/f Foxp3-Cre 271 289 GENOTYPE\n",
      "WT 1759 1761 GENOTYPE\n",
      "DKI 1792 1795 GENOTYPE\n",
      "None 86 87 GENOTYPE\n",
      "Skipping entity\n",
      "Il18+/+ mice 266 278 GENOTYPE\n",
      "Il18+/+ mice 459 471 GENOTYPE\n",
      "Sox2GFP 1145 1152 GENOTYPE\n",
      "Wild-type 1275 1284 GENOTYPE\n",
      "GRA knockin 494 505 GENOTYPE\n",
      "GRA knockin 692 703 GENOTYPE\n",
      "GRA knockin 870 881 GENOTYPE\n",
      "GRA knockin 1294 1305 GENOTYPE\n",
      "GRA knockin 1452 1463 GENOTYPE\n",
      "Wild-type 1286 1295 GENOTYPE\n",
      "GRA knockin 494 505 GENOTYPE\n",
      "GRA knockin 703 714 GENOTYPE\n",
      "GRA knockin 881 892 GENOTYPE\n",
      "GRA knockin 1305 1316 GENOTYPE\n",
      "GRA knockin 1463 1474 GENOTYPE\n",
      "Wild-type 1274 1283 GENOTYPE\n",
      "GRA knockin 494 505 GENOTYPE\n",
      "GRA knockin 691 702 GENOTYPE\n",
      "GRA knockin 869 880 GENOTYPE\n",
      "GRA knockin 1293 1304 GENOTYPE\n",
      "GRA knockin 1451 1462 GENOTYPE\n",
      "FtH KO 702 708 GENOTYPE\n",
      "OPA1 1500 1504 GENOTYPE\n",
      "ARNoC/Y 28 35 GENOTYPE\n",
      "ARNoC/Y 259 266 GENOTYPE\n",
      "ARNoC/Y 609 616 GENOTYPE\n",
      "ARNoC/Y 628 635 GENOTYPE\n",
      "wild type 193 202 GENOTYPE\n",
      "wild type 314 323 GENOTYPE\n",
      "wild type 496 505 GENOTYPE\n",
      "WAP-Cre/PIK3CA-H1047R 677 698 GENOTYPE\n",
      "GFP 242 245 GENOTYPE\n",
      "SRRM4 249 254 GENOTYPE\n",
      "Math1-Cre/Eed fl/fl 380 399 GENOTYPE\n",
      "WT 1464 1466 GENOTYPE\n",
      "KO 379 381 GENOTYPE\n",
      "KO 692 694 GENOTYPE\n",
      "KO 1476 1478 GENOTYPE\n",
      "WT 446 448 GENOTYPE\n",
      "WT 475 477 GENOTYPE\n",
      "MLL-AF9 395 402 GENOTYPE\n",
      "C57BL6/j 104 112 GENOTYPE\n",
      "C57BL6/j 510 518 GENOTYPE\n",
      "Rit1 M90I/+ 371 382 GENOTYPE\n",
      "CD19-cre x iDTR 584 599 GENOTYPE\n",
      "p53-null 90 98 GENOTYPE\n",
      "p53-null 339 347 GENOTYPE\n",
      "WT 187 189 GENOTYPE\n",
      "None 194 196 GENOTYPE\n",
      "Skipping entity\n",
      "WT 1140 1142 GENOTYPE\n",
      "None 1147 1149 GENOTYPE\n",
      "Skipping entity\n",
      "WT 1858 1860 GENOTYPE\n",
      "None 1865 1867 GENOTYPE\n",
      "Skipping entity\n",
      "WT 2426 2428 GENOTYPE\n",
      "None 2433 2435 GENOTYPE\n",
      "Skipping entity\n",
      "wild type 1084 1093 GENOTYPE\n",
      "knockout 391 399 GENOTYPE\n",
      "knockout 1160 1168 GENOTYPE\n",
      "WT 97 99 GENOTYPE\n",
      "WT 216 218 GENOTYPE\n",
      "edited 627 633 GENOTYPE\n",
      "WT 508 510 GENOTYPE\n",
      "GFP 1071 1074 GENOTYPE\n",
      "mouse FOXP2 1084 1095 GENOTYPE\n",
      "mutant 2260 2266 GENOTYPE\n",
      "wildtype 280 288 GENOTYPE\n",
      "Arid5b overexpression 255 276 GENOTYPE\n",
      "None 72 74 GENOTYPE\n",
      "Skipping entity\n",
      "None 221 223 GENOTYPE\n",
      "Skipping entity\n",
      "None 81 83 GENOTYPE\n",
      "Skipping entity\n",
      "wild-type 1443 1452 GENOTYPE\n",
      "wild-type 1963 1972 GENOTYPE\n",
      "Rac1/Rac3 wt 913 925 GENOTYPE\n",
      "Rac1/Rac3 dmut 982 996 GENOTYPE\n",
      "WT 377 379 GENOTYPE\n",
      "Tbx1+/- 399 406 GENOTYPE\n",
      "Tbx1+/- 431 438 GENOTYPE\n",
      "WT 1269 1271 GENOTYPE\n",
      "WT 176 178 GENOTYPE\n",
      "Wild type FVB/N 1401 1416 GENOTYPE\n",
      "C9-BAC 203 209 GENOTYPE\n",
      "C9- 260 266 GENOTYPE\n",
      "C9- 303 309 GENOTYPE\n",
      "C9-BAC;C9orf72+/- 260 277 GENOTYPE\n",
      "C9-BAC;C9orf72-/- 303 320 GENOTYPE\n",
      "WT 196 198 GENOTYPE\n",
      "-BAC 181 198 GENOTYPE\n",
      "C9orf72+/-;C9-BAC 466 483 GENOTYPE\n",
      "WT 1412 1414 GENOTYPE\n",
      "None 1690 1692 GENOTYPE\n",
      "Skipping entity\n",
      "Pten knock out 1113 1127 GENOTYPE\n",
      "Pten knock out 1113 1127 GENOTYPE\n",
      "Pten knock out 1113 1127 GENOTYPE\n",
      "APP/PS1 415 422 GENOTYPE\n",
      "APP/PS1 650 657 GENOTYPE\n",
      "APP/PS1 846 853 GENOTYPE\n",
      "APP/PS1 910 917 GENOTYPE\n",
      "APP/PS1 1016 1023 GENOTYPE\n",
      "Pten knock out 1113 1127 GENOTYPE\n",
      "Pi*M 292 296 GENOTYPE\n",
      "Pi*M 873 877 GENOTYPE\n",
      "Pi*Z 336 340 GENOTYPE\n",
      "Pi*Z 864 868 GENOTYPE\n",
      "C57BL/6J WT 212 223 GENOTYPE\n",
      "DKO 1328 1331 GENOTYPE\n",
      "DKO 1761 1764 GENOTYPE\n",
      "IR 43 45 GENOTYPE\n",
      "IR 267 269 GENOTYPE\n",
      "IR 407 409 GENOTYPE\n",
      "IR 890 892 GENOTYPE\n",
      "IR 1210 1212 GENOTYPE\n",
      "IR 1271 1273 GENOTYPE\n",
      "IR 1408 1410 GENOTYPE\n",
      "IR 1436 1438 GENOTYPE\n",
      "IR 1461 1463 GENOTYPE\n",
      "IR 1528 1530 GENOTYPE\n",
      "IR 1766 1768 GENOTYPE\n",
      "None 1509 1511 GENOTYPE\n",
      "Skipping entity\n",
      "None 1779 1781 GENOTYPE\n",
      "Skipping entity\n",
      "JMO 1720 1723 GENOTYPE\n",
      "JMO 1786 1789 GENOTYPE\n",
      "Cxcr4+ 237 243 GENOTYPE\n",
      "Cxcr4+ 358 364 GENOTYPE\n",
      "Cxcr4+ 384 390 GENOTYPE\n",
      "Cxcr4- 420 426 GENOTYPE\n",
      "WT 634 636 GENOTYPE\n",
      "WT 1318 1320 GENOTYPE\n",
      "WT 336 338 GENOTYPE\n",
      "None 1394 1396 GENOTYPE\n",
      "Skipping entity\n",
      "WT 336 338 GENOTYPE\n",
      "WT 336 338 GENOTYPE\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 44%|█████████████████▏                     | 795/1804 [00:03<00:03, 315.58it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "WT and Ptgs2-cKO 426 442 GENOTYPE\n",
      "WT and Ptgs2-cKO 494 510 GENOTYPE\n",
      "WT 1258 1260 GENOTYPE\n",
      "Snap29 knockout 1222 1237 GENOTYPE\n",
      "Snap29 knockout 1724 1739 GENOTYPE\n",
      "Apoe(-/-), Itgb3(-/-) 239 260 GENOTYPE\n",
      "Apoe(-/-), Itgb3(-/-) 1097 1118 GENOTYPE\n",
      "Apoe(-/-), Itgb3(-/-) 1259 1280 GENOTYPE\n",
      "Apoe(-/-), Itgb3(-/-) 1372 1393 GENOTYPE\n",
      "Apoe(-/-), Itgb3(+/+) 213 234 GENOTYPE\n",
      "Apoe(-/-), Itgb3(+/+) 1071 1092 GENOTYPE\n",
      "Apoe(-/-), Itgb3(+/+) 1233 1254 GENOTYPE\n",
      "Apoe(-/-), Itgb3(+/+) 1346 1367 GENOTYPE\n",
      "WT 181 183 GENOTYPE\n",
      "Mpl-/- Dnm2Plt+/+ 1433 1450 GENOTYPE\n",
      "Mpl-/- Dnm2Plt-/- 811 828 GENOTYPE\n",
      "Mpl-/- Dnm2Plt-/- 890 907 GENOTYPE\n",
      "Mpl-/- Dnm2Plt-/- 983 1000 GENOTYPE\n",
      "Mpl-/- Dnm2Plt-/- 1108 1125 GENOTYPE\n",
      "Mpl-/- Dnm2Plt-/- 1929 1946 GENOTYPE\n",
      "SOD1(G93A) 252 262 GENOTYPE\n",
      "SOD1(G93A) 882 892 GENOTYPE\n",
      "SOD1(G93A) 1109 1119 GENOTYPE\n",
      "SOD1(G93A) 1502 1512 GENOTYPE\n",
      "ReverbhDKO 212 222 GENOTYPE\n",
      "ReverbhDKO 499 509 GENOTYPE\n",
      "ReverbhDKO 1046 1056 GENOTYPE\n",
      "ReverbhDKO 212 222 GENOTYPE\n",
      "ReverbhDKO 499 509 GENOTYPE\n",
      "ReverbhDKO 1053 1063 GENOTYPE\n",
      "Control 1338 1345 GENOTYPE\n",
      "WT 1515 1517 GENOTYPE\n",
      "WT 1496 1498 GENOTYPE\n",
      "Hvemfl/fl mice 1467 1481 GENOTYPE\n",
      "Ros1-/- 133 140 GENOTYPE\n",
      "Pten S380A/S380A 1015 1031 GENOTYPE\n",
      "Pten S380D/S380D 1036 1052 GENOTYPE\n",
      "wild type 447 456 GENOTYPE\n",
      "Wfs1-/- 114 121 GENOTYPE\n",
      "Wfs1-/- 466 473 GENOTYPE\n",
      "CYB5R3wt/wt, Myh6-CreERT2 1598 1623 GENOTYPE\n",
      "CYB5R3fl/fl, Myh6-CreERT2 1551 1576 GENOTYPE\n",
      "knockout 148 156 GENOTYPE\n",
      "knockout 240 248 GENOTYPE\n",
      "control 166 173 GENOTYPE\n",
      "control 262 269 GENOTYPE\n",
      "C57BL/6J 112 120 GENOTYPE\n",
      "C57BL/6J 248 256 GENOTYPE\n",
      "WT 266 268 GENOTYPE\n",
      "WT 382 384 GENOTYPE\n",
      "Alb/MDA-9 93 102 GENOTYPE\n",
      "Alb/MDA-9 273 282 GENOTYPE\n",
      "Alb/MDA-9 389 398 GENOTYPE\n",
      "WT 254 256 GENOTYPE\n",
      "CCR10+/EGFP 216 227 GENOTYPE\n",
      "WT 1617 1619 GENOTYPE\n",
      "miR-155KO 1624 1633 GENOTYPE\n",
      "VGlut2Cre;R26-nGFP 256 274 GENOTYPE\n",
      "Mettl14 fl/fl;Chx10-Egfp/Cre 373 401 GENOTYPE\n",
      "DKO 448 451 GENOTYPE\n",
      "None 462 465 GENOTYPE\n",
      "Skipping entity\n",
      "Pspc1 knockdown 148 163 GENOTYPE\n",
      "Lgr5CreER/KRasG12D/p53cKO/ Rosa-YFP 522 557 GENOTYPE\n",
      "Tet2 knockout 1090 1103 GENOTYPE\n",
      "Tet2 knockout 1207 1220 GENOTYPE\n",
      "Tet2 knockout 1546 1559 GENOTYPE\n",
      "Tet2 knockout 1816 1829 GENOTYPE\n",
      "wild-type 1834 1843 GENOTYPE\n",
      "wild type 922 931 GENOTYPE\n",
      "wild type 922 931 GENOTYPE\n",
      "Foxd1Cre;Ptch1 (flox/-) 293 316 GENOTYPE\n",
      "Ncr1-Cre 315 323 GENOTYPE\n",
      "Itgb1 fl/fl 1509 1520 GENOTYPE\n",
      "Itgb1 iKO 1552 1561 GENOTYPE\n",
      "RAMP1-/- (CD45.2+) 128 146 GENOTYPE\n",
      "RAMP1-/- (CD45.2+) 519 537 GENOTYPE\n",
      "RAMP1wt (CD45.1+) 107 124 GENOTYPE\n",
      "RAMP1wt (CD45.1+) 542 559 GENOTYPE\n",
      "C57BL/6 800 807 GENOTYPE\n",
      "wt 1101 1103 GENOTYPE\n",
      "ANXA2 knockout 228 242 GENOTYPE\n",
      "ANXA5 knockout 246 260 GENOTYPE\n",
      "WT 902 904 GENOTYPE\n",
      "WT 1400 1402 GENOTYPE\n",
      "WT 1249 1251 GENOTYPE\n",
      "Nnat+/-p 1962 1970 GENOTYPE\n",
      "Nnat+/-p 1978 1986 GENOTYPE\n",
      "wild type 1998 2007 GENOTYPE\n",
      "Grn-/-; GRNtg 475 488 GENOTYPE\n",
      "Grn+/- 425 431 GENOTYPE\n",
      "Grn+/+ 383 389 GENOTYPE\n",
      "C57BL/6 1006 1013 GENOTYPE\n",
      "WT 353 355 GENOTYPE\n",
      "Ern2-/- 360 367 GENOTYPE\n",
      "Slc26a4KO 302 311 GENOTYPE\n",
      "Slc26a7KO 288 297 GENOTYPE\n",
      "WT 122 124 GENOTYPE\n",
      "WT 335 337 GENOTYPE\n",
      "Cd1d KO 129 136 GENOTYPE\n",
      "Cd1d KO 342 349 GENOTYPE\n",
      "wild type 66 75 GENOTYPE\n",
      "wild type 265 274 GENOTYPE\n",
      "Ldlr-/- 93 100 GENOTYPE\n",
      "Ldlr-/- 276 283 GENOTYPE\n",
      "Ldlr-/- 480 487 GENOTYPE\n",
      "Apoe-/- 138 145 GENOTYPE\n",
      "Apoe-/- 285 292 GENOTYPE\n",
      "Apoe-/- 492 499 GENOTYPE\n",
      "LysM-cre x Rel+/+ 1679 1696 GENOTYPE\n",
      "LysM-cre x RelloxP/loxP 1741 1764 GENOTYPE\n",
      "IRE1F/F 1634 1641 GENOTYPE\n",
      "K14CreERT2 IRE1 F/F 1736 1755 GENOTYPE\n",
      "TRIM28f/f 451 460 GENOTYPE\n",
      "TRIM28d/d 472 481 GENOTYPE\n",
      "Pgrcre/+ 446 454 GENOTYPE\n",
      "TRIM28f/f 450 459 GENOTYPE\n",
      "TRIM28d/d 471 480 GENOTYPE\n",
      "Sox9-CreERT2;R26RtdTomato 247 272 GENOTYPE\n",
      "WT 1519 1521 GENOTYPE\n",
      "NFIA-ETO2 4 13 GENOTYPE\n",
      "NFIA-ETO2 285 294 GENOTYPE\n",
      "NFIA-ETO2 401 410 GENOTYPE\n",
      "NFIA-ETO2 516 525 GENOTYPE\n",
      "FVB/n 1198 1203 GENOTYPE\n",
      "mTmG 1208 1212 GENOTYPE\n",
      "TGAC8 25 30 GENOTYPE\n",
      "TGAC8 246 251 GENOTYPE\n",
      "WT 18 20 GENOTYPE\n",
      "WT 256 258 GENOTYPE\n",
      "None 298 300 GENOTYPE\n",
      "Skipping entity\n",
      "Cdh1F/F;PtenF/F 1128 1143 GENOTYPE\n",
      "EN1-Cre;mTmG 1240 1252 GENOTYPE\n",
      "ETV6::JAK2;Rag2-/-;Cdkn2a-/- 71 99 GENOTYPE\n",
      "Rag2-/-;Cdkn2a-/- 49 66 GENOTYPE\n",
      "None 82 99 GENOTYPE\n",
      "Skipping entity\n",
      "Ube3aOE 146 153 GENOTYPE\n",
      "Ube3aOE 384 391 GENOTYPE\n",
      "Ube3aOE 618 625 GENOTYPE\n",
      "Ube3aOE 772 779 GENOTYPE\n",
      "None 44 47 GENOTYPE\n",
      "Skipping entity\n",
      "None 822 825 GENOTYPE\n",
      "Skipping entity\n",
      "None 931 934 GENOTYPE\n",
      "Skipping entity\n",
      "None 1101 1104 GENOTYPE\n",
      "Skipping entity\n",
      "None 44 47 GENOTYPE\n",
      "Skipping entity\n",
      "None 822 825 GENOTYPE\n",
      "Skipping entity\n",
      "None 931 934 GENOTYPE\n",
      "Skipping entity\n",
      "None 1101 1104 GENOTYPE\n",
      "Skipping entity\n",
      "None 44 47 GENOTYPE\n",
      "Skipping entity\n",
      "None 822 825 GENOTYPE\n",
      "Skipping entity\n",
      "None 931 934 GENOTYPE\n",
      "Skipping entity\n",
      "None 1101 1104 GENOTYPE\n",
      "Skipping entity\n",
      "Wildtype 1092 1100 GENOTYPE\n",
      "Dhh+/- 1003 1009 GENOTYPE\n",
      "Dhh+/- 1125 1131 GENOTYPE\n",
      "Dhh-/- 995 1001 GENOTYPE\n",
      "Dhh-/- 1151 1157 GENOTYPE\n",
      "PdgfraDreER-tdTomato 193 213 GENOTYPE\n",
      "PCSK9 KO 529 537 GENOTYPE\n",
      "PCSK9 KO 725 733 GENOTYPE\n",
      "PCSK9 KO 884 892 GENOTYPE\n",
      "PCSK9 KO 1707 1715 GENOTYPE\n",
      "WT 560 562 GENOTYPE\n",
      "WT 1701 1703 GENOTYPE\n",
      "Idh1LSL-R132H/+;Vav-Cre 1385 1408 GENOTYPE\n",
      "Idh2LSL-R140Q/+;Vav-Cre 1410 1433 GENOTYPE\n",
      "Idh2LSL-R172K/+;Vav-Cre 1435 1458 GENOTYPE\n",
      "Tet2-/- 1464 1471 GENOTYPE\n",
      "SCAP+/+CD19Cre/+ (WT) 79 100 GENOTYPE\n",
      "SCAPfl/fl CD19Cre/+ (KO) 105 129 GENOTYPE\n",
      "C57BL/6 1121 1128 GENOTYPE\n",
      "AnxA1 knockout 871 885 GENOTYPE\n",
      "None 296 298 GENOTYPE\n",
      "Skipping entity\n",
      "KO 549 551 GENOTYPE\n",
      "wild type 542 551 GENOTYPE\n",
      "wild-type 319 328 GENOTYPE\n",
      "Lemd2 p.L13R knock-in 338 359 GENOTYPE\n",
      "WT 485 487 GENOTYPE\n",
      "WT 618 620 GENOTYPE\n",
      "WT 1216 1218 GENOTYPE\n",
      "WT 1272 1274 GENOTYPE\n",
      "WT 1284 1286 GENOTYPE\n",
      "CTRL 356 360 GENOTYPE\n",
      "CTRL 839 843 GENOTYPE\n",
      "ECKO 178 182 GENOTYPE\n",
      "ECKO 845 849 GENOTYPE\n",
      "SMKO 244 248 GENOTYPE\n",
      "SMKO 851 855 GENOTYPE\n",
      "DKO 285 288 GENOTYPE\n",
      "DKO 379 382 GENOTYPE\n",
      "DKO 667 670 GENOTYPE\n",
      "DKO 860 863 GENOTYPE\n",
      "Kras (+/loxG12Vlox);Trp53(-/-) 485 515 GENOTYPE\n",
      "Kras (+/-);Trp53(-/-) 547 568 GENOTYPE\n",
      "Kras-G12C, Trp53-KO 111 130 GENOTYPE\n",
      "TCF-7 cKO 1379 1388 GENOTYPE\n",
      "TCF-7 cKO 1516 1525 GENOTYPE\n",
      "Control 1178 1185 GENOTYPE\n",
      "None 44 47 GENOTYPE\n",
      "Skipping entity\n",
      "None 822 825 GENOTYPE\n",
      "Skipping entity\n",
      "None 931 934 GENOTYPE\n",
      "Skipping entity\n",
      "None 1101 1104 GENOTYPE\n",
      "Skipping entity\n",
      "LOY 1229 1232 GENOTYPE\n",
      "None 868 876 GENOTYPE\n",
      "Skipping entity\n",
      "PostnMCM 907 915 GENOTYPE\n",
      "Yapfl/fl;Wwtr1fl/+;PostnMCM 849 876 GENOTYPE\n",
      "KO 916 918 GENOTYPE\n",
      "KO 1944 1946 GENOTYPE\n",
      "Exosc3 fl/fl Cre+ 786 803 GENOTYPE\n",
      "WT 177 179 GENOTYPE\n",
      "WT 246 248 GENOTYPE\n",
      "WT 503 505 GENOTYPE\n",
      "TRPM2-KO 184 192 GENOTYPE\n",
      "TRPM2-KO 253 261 GENOTYPE\n",
      "TRPM2-KO 510 518 GENOTYPE\n",
      "WT 925 927 GENOTYPE\n",
      "APP/PS1 212 219 GENOTYPE\n",
      "APP/PS1 744 751 GENOTYPE\n",
      "TrkA KI 856 863 GENOTYPE\n",
      "TrkA KI 1081 1088 GENOTYPE\n",
      "WT 470 472 GENOTYPE\n",
      "WT 732 734 GENOTYPE\n",
      "WT 912 914 GENOTYPE\n",
      "WT 1243 1245 GENOTYPE\n",
      "WT 1280 1282 GENOTYPE\n",
      "Prmt1 knockdown 618 633 GENOTYPE\n",
      "Prmt1 knockdown 659 674 GENOTYPE\n",
      "Flt3 983 987 GENOTYPE\n",
      "Flt3 1316 1320 GENOTYPE\n",
      "Flt3 1810 1814 GENOTYPE\n",
      "Flt3 1854 1858 GENOTYPE\n",
      "Flt3 1998 2002 GENOTYPE\n",
      "Flt3 2084 2088 GENOTYPE\n",
      "CD18fl/fl 1610 1619 GENOTYPE\n",
      "WT 559 561 GENOTYPE\n",
      "Flox 2369 2373 GENOTYPE\n",
      "None 475 479 GENOTYPE\n",
      "Skipping entity\n",
      "M 657 661 GENOTYPE\n",
      "M 785 789 GENOTYPE\n",
      "M 911 915 GENOTYPE\n",
      "K36M 1173 1177 GENOTYPE\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 48%|██████████████████▌                    | 859/1804 [00:03<00:03, 306.57it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "None 475 479 GENOTYPE\n",
      "Skipping entity\n",
      "M 657 661 GENOTYPE\n",
      "M 785 789 GENOTYPE\n",
      "M 911 915 GENOTYPE\n",
      "K36M 1173 1177 GENOTYPE\n",
      "Maf1 knockdown 663 677 GENOTYPE\n",
      "Brf1 knockdown 365 379 GENOTYPE\n",
      "MAF1 overexpression 274 293 GENOTYPE\n",
      "MAF1 overexpression 702 721 GENOTYPE\n",
      "wildtype 1168 1176 GENOTYPE\n",
      "knockout 362 370 GENOTYPE\n",
      "knockout 681 689 GENOTYPE\n",
      "knockout 1181 1189 GENOTYPE\n",
      "Rif1 CKO 495 503 GENOTYPE\n",
      "Dnmt3l KO 1469 1478 GENOTYPE\n",
      "Dnmt3b KO 1432 1441 GENOTYPE\n",
      "Dnmt3b KO 1927 1936 GENOTYPE\n",
      "Dnmt3b KO 2036 2045 GENOTYPE\n",
      "Dnmt3a KO 1421 1430 GENOTYPE\n",
      "Arid1b heterozygote 853 872 GENOTYPE\n",
      "WT 227 229 GENOTYPE\n",
      "C57BL6/J 60 68 GENOTYPE\n",
      "Dnmt3b KO 1432 1441 GENOTYPE\n",
      "Dnmt3b KO 1927 1936 GENOTYPE\n",
      "Dnmt3b KO 2036 2045 GENOTYPE\n",
      "Cpa3Cre/+ 570 579 GENOTYPE\n",
      "F1 mus/cast 818 829 GENOTYPE\n",
      "F1 mus/cast 1450 1461 GENOTYPE\n",
      "TCF-7 cKO 535 544 GENOTYPE\n",
      "TCF-7 cKO 1003 1012 GENOTYPE\n",
      "TCF-7 cKO 1152 1161 GENOTYPE\n",
      "TCF-7 cKO 1289 1298 GENOTYPE\n",
      "-flox/flox 1203 1216 GENOTYPE\n",
      "Srf-flox/flox 1232 1245 GENOTYPE\n",
      "Tbx4-rtTA;tet-O-Cre;Srf-flox/flox 1183 1216 GENOTYPE\n",
      "NEK7 whole body knockout 276 300 GENOTYPE\n",
      "mutant 604 610 GENOTYPE\n",
      "mutant 740 746 GENOTYPE\n",
      "None 923 929 GENOTYPE\n",
      "Skipping entity\n",
      "mutant 937 943 GENOTYPE\n",
      "mutant 1173 1179 GENOTYPE\n",
      "None 1373 1379 GENOTYPE\n",
      "Skipping entity\n",
      "wild type 726 735 GENOTYPE\n",
      "wild type 948 957 GENOTYPE\n",
      "WT 1402 1404 GENOTYPE\n",
      "Ikzf3-/- 1408 1416 GENOTYPE\n",
      "Dct-rtTA; tetO-H2B-GFP 296 318 GENOTYPE\n",
      "Dct-rtTA; tetO-H2B-GFP 527 549 GENOTYPE\n",
      "control 942 949 GENOTYPE\n",
      "Osr2-Cre;Alk5fl/fl 954 972 GENOTYPE\n",
      "Ptf1aFlpO/+;KrasFrt-STOP-Frt-G12D/+ 172 207 GENOTYPE\n",
      "Ptf1aFlpO/+;KrasFrt-STOP-Frt- 241 276 GENOTYPE\n",
      "Ptf1aFlpO/+;KrasFrt-STOP-Frt-G12D/+;LysMcre;Arg1f/f 241 292 GENOTYPE\n",
      "GR-cKO 179 185 GENOTYPE\n",
      "GR-cKO 340 346 GENOTYPE\n",
      "GR-cKO 707 713 GENOTYPE\n",
      "GR-cKO 179 185 GENOTYPE\n",
      "GR-cKO 333 339 GENOTYPE\n",
      "GR-cKO 708 714 GENOTYPE\n",
      "Tyk2wt 303 309 GENOTYPE\n",
      "Tyk2KO 136 142 GENOTYPE\n",
      "Tyk2KO 311 317 GENOTYPE\n",
      "Tyk2DIEC 206 214 GENOTYPE\n",
      "Tyk2DIEC 329 337 GENOTYPE\n",
      "Tyk2fl 318 324 GENOTYPE\n",
      "WT 1499 1501 GENOTYPE\n",
      "None 1210 1212 GENOTYPE\n",
      "Skipping entity\n",
      "None 1456 1458 GENOTYPE\n",
      "Skipping entity\n",
      "wild-type 520 529 GENOTYPE\n",
      "wild type 186 195 GENOTYPE\n",
      "wild type 276 285 GENOTYPE\n",
      "wild type 579 588 GENOTYPE\n",
      "K14Cre;R26mT/mG 955 970 GENOTYPE\n",
      "Mettl3-KO 2047 2056 GENOTYPE\n",
      "CCR2 RFP/+ 85 95 GENOTYPE\n",
      "CCR2 RFP/+ 145 155 GENOTYPE\n",
      "CCR2 RFP/RFP 107 119 GENOTYPE\n",
      "CCR2 RFP/RFP 186 198 GENOTYPE\n",
      "Fbn1 C1039G/+; CCR2 RFP/+ 130 155 GENOTYPE\n",
      "Fbn1 C1039G/+; CCR2 RFP/RFP 171 198 GENOTYPE\n",
      "Idh1LSL-R132H/+;Vav-Cre 1385 1408 GENOTYPE\n",
      "Idh2LSL-R140Q/+;Vav-Cre 1410 1433 GENOTYPE\n",
      "Idh2LSL-R172K/+;Vav-Cre 1439 1462 GENOTYPE\n",
      "wild-type 629 638 GENOTYPE\n",
      "dominant-negative transforming growth factor beta receptor II 648 709 GENOTYPE\n",
      "Aid WT 465 471 GENOTYPE\n",
      "Aid WT 543 549 GENOTYPE\n",
      "Aid KO 626 632 GENOTYPE\n",
      "KrasLSL-G12D; Trp53LSL-R172H; Ptf1a-Cre 98 137 GENOTYPE\n",
      "wild type 75 84 GENOTYPE\n",
      "wild type 1482 1491 GENOTYPE\n",
      "control 1485 1492 GENOTYPE\n",
      "wildtype 456 464 GENOTYPE\n",
      "Dlx5/6-CIE 2220 2230 GENOTYPE\n",
      "Dlx5/6-CIE 2255 2265 GENOTYPE\n",
      "Dlx5/6-CIE; Rosa-Sp9-OE/+ 2220 2245 GENOTYPE\n",
      "WT 1273 1275 GENOTYPE\n",
      "WT 1682 1684 GENOTYPE\n",
      "WT 2451 2453 GENOTYPE\n",
      "K5aSE-KO 619 627 GENOTYPE\n",
      "K5aSE-KO 691 699 GENOTYPE\n",
      "K5aSE-KO 1108 1116 GENOTYPE\n",
      "Klf5-KO 646 653 GENOTYPE\n",
      "Nkx3-1 991 997 GENOTYPE\n",
      "CIH 77 80 GENOTYPE\n",
      "CIH 233 236 GENOTYPE\n",
      "CIH 351 354 GENOTYPE\n",
      "LoxP 439 443 GENOTYPE\n",
      "WT 482 484 GENOTYPE\n",
      "db/db 1157 1162 GENOTYPE\n",
      "C57BL6 210 216 GENOTYPE\n",
      "C57BL6 260 266 GENOTYPE\n",
      "control 1008 1015 GENOTYPE\n",
      "WT 2009 2011 GENOTYPE\n",
      "None 181 183 GENOTYPE\n",
      "Skipping entity\n",
      "WT 156 158 GENOTYPE\n",
      "WT 318 320 GENOTYPE\n",
      "Trpv2fl/fl 38 48 GENOTYPE\n",
      "/fl 62 72 GENOTYPE\n",
      "Trpv2fl/fl 1745 1755 GENOTYPE\n",
      "/fl 1780 1790 GENOTYPE\n",
      "Trpv2fl/fl 1837 1847 GENOTYPE\n",
      "/fl 1872 1882 GENOTYPE\n",
      "Lyz2-Cre;Trpv2fl/fl 53 72 GENOTYPE\n",
      "Lyz2-Cre;Trpv2fl/fl 1771 1790 GENOTYPE\n",
      "Lyz2-Cre;Trpv2fl/fl 1863 1882 GENOTYPE\n",
      "NTG 585 588 GENOTYPE\n",
      "TG 198 200 GENOTYPE\n",
      "TG 353 355 GENOTYPE\n",
      "TG 561 563 GENOTYPE\n",
      "None 586 588 GENOTYPE\n",
      "Skipping entity\n",
      "DKO 150 153 GENOTYPE\n",
      "DKO 217 220 GENOTYPE\n",
      "DKO 255 258 GENOTYPE\n",
      "DKO 474 477 GENOTYPE\n",
      "DKO 488 491 GENOTYPE\n",
      "DKO 526 529 GENOTYPE\n",
      "DKO/hA-I 255 263 GENOTYPE\n",
      "DKO/hA-I 526 534 GENOTYPE\n",
      "Cxcl12DsRed/+ 1646 1659 GENOTYPE\n",
      "Cxcl12DsRed/DsRed 1554 1571 GENOTYPE\n",
      "MNX1 overexpression 573 592 GENOTYPE\n",
      "MNX1 overexpression 1292 1311 GENOTYPE\n",
      "WT 1418 1420 GENOTYPE\n",
      "WT 1372 1374 GENOTYPE\n",
      "KO 1386 1388 GENOTYPE\n",
      "WT 1383 1385 GENOTYPE\n",
      "KO 1397 1399 GENOTYPE\n",
      "WT 1401 1403 GENOTYPE\n",
      "Gne KO 1390 1396 GENOTYPE\n",
      "PLD 554 557 GENOTYPE\n",
      "PLD 606 609 GENOTYPE\n",
      "WT 599 601 GENOTYPE\n",
      "PLD 554 557 GENOTYPE\n",
      "PLD 606 609 GENOTYPE\n",
      "WT 599 601 GENOTYPE\n",
      "Fh1 fl/fl 465 474 GENOTYPE\n",
      "Fh1 fl/fl 547 556 GENOTYPE\n",
      "Fh1 fl/fl g1Hira 547 563 GENOTYPE\n",
      "Fh1-/- g1Hira 568 581 GENOTYPE\n",
      "TMPRSS2-CreERT2;Rosa26-CAG-LSL-ETV4AAA 280 318 GENOTYPE\n",
      "TMPRSS2-CreERT2;Rosa26-CAG-LSL- 323 361 GENOTYPE\n",
      "TMPRSS2-CreERT2;Rosa26-CAG-LSL-ETV4AAA;Trp53L/L 323 370 GENOTYPE\n",
      "TMPRSS2-CreERT2;Rosa26-CAG-LSL-ETV4WT 241 278 GENOTYPE\n",
      "TMPRSS2-CreERT2;Rosa26-CAG-LSL-EYFP 204 239 GENOTYPE\n",
      "Cre- 1551 1555 GENOTYPE\n",
      "CD19-CD45+TER119+CD11b+ 236 259 GENOTYPE\n",
      "CD19-CD45+TER119+CD11b+ 612 635 GENOTYPE\n",
      "CD19-CD45+TER119-CD11b+ 173 196 GENOTYPE\n",
      "CD19-CD45+TER119-CD11b+ 549 572 GENOTYPE\n",
      "SKO 501 504 GENOTYPE\n",
      "SKO 570 573 GENOTYPE\n",
      "DKO 444 447 GENOTYPE\n",
      "DKO 578 581 GENOTYPE\n",
      "Chrm3 KO 805 813 GENOTYPE\n",
      "Chrm3 KO 894 902 GENOTYPE\n",
      "WT 105 107 GENOTYPE\n",
      "NFKO 112 116 GENOTYPE\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 51%|███████████████████▉                   | 924/1804 [00:03<00:02, 301.39it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Mtg16-null 1305 1315 GENOTYPE\n",
      "Mtg16-null 1652 1662 GENOTYPE\n",
      "Mtg16-null 1871 1881 GENOTYPE\n",
      "Mtg16-null 2236 2246 GENOTYPE\n",
      "Mtg16-null 2301 2311 GENOTYPE\n",
      "Mtg16-null 2904 2914 GENOTYPE\n",
      "WT 1505 1507 GENOTYPE\n",
      "WT 189 191 GENOTYPE\n",
      "Tet2 mutant 63 74 GENOTYPE\n",
      "SIRT6f/f 1297 1305 GENOTYPE\n",
      "/f 1318 1326 GENOTYPE\n",
      "Prx1cre;SIRT6f/f 1310 1326 GENOTYPE\n",
      "None 635 647 GENOTYPE\n",
      "Skipping entity\n",
      "heterozygous 1904 1916 GENOTYPE\n",
      "heterozygous 2092 2104 GENOTYPE\n",
      "LiKM (KrasG12D mutation and Mdm2 loss in the liver) 54 105 GENOTYPE\n",
      "WT 1484 1486 GENOTYPE\n",
      "IFNgR KO 1471 1479 GENOTYPE\n",
      "IFNAR KO 1461 1469 GENOTYPE\n",
      "Ebf1-/- 28 35 GENOTYPE\n",
      "Ebf1-/- 156 163 GENOTYPE\n",
      "Tnpo3+/+ mb1Cre 58 73 GENOTYPE\n",
      "Tnpo3+/+ mb1Cre 209 224 GENOTYPE\n",
      "Tnpo3fl/fl mb1Cre 28 45 GENOTYPE\n",
      "Tnpo3fl/fl mb1Cre 170 187 GENOTYPE\n",
      "OTUD3 KO 0 8 GENOTYPE\n",
      "Apc Min/+ 1531 1540 GENOTYPE\n",
      "liver-specific Ftcd 126 154 GENOTYPE\n",
      "WT 279 281 GENOTYPE\n",
      "WT 431 433 GENOTYPE\n",
      "WT 533 535 GENOTYPE\n",
      "WT 694 696 GENOTYPE\n",
      "Apc Min/+ 1527 1536 GENOTYPE\n",
      "CTL 1374 1377 GENOTYPE\n",
      "MLKL-/- 361 368 GENOTYPE\n",
      "MLKL-/- 1380 1387 GENOTYPE\n",
      "RIPK3-/- 329 337 GENOTYPE\n",
      "RIPK3-/- 1399 1407 GENOTYPE\n",
      "Foxp3YFP-Cre mice 505 522 GENOTYPE\n",
      "Foxp3YFP-Cre mice 671 688 GENOTYPE\n",
      "SRC2fl/fl/Foxp3YFP-Cre mice 495 522 GENOTYPE\n",
      "SRC2fl/fl/Foxp3YFP-Cre mice 661 688 GENOTYPE\n",
      "control (RM-1-Vector) 657 678 GENOTYPE\n",
      "Bhlhe22 overexpression 689 711 GENOTYPE\n",
      "MLL-ENL 18 25 GENOTYPE\n",
      "WT 185 187 GENOTYPE\n",
      "MLL-ENL 18 25 GENOTYPE\n",
      "WT 125 127 GENOTYPE\n",
      "WT 290 292 GENOTYPE\n",
      "Mpst knockout 301 314 GENOTYPE\n",
      "/Fl 440 445 GENOTYPE\n",
      "Fl/Fl 447 452 GENOTYPE\n",
      "/Fl 464 469 GENOTYPE\n",
      "Fl/Fl 648 653 GENOTYPE\n",
      "LKO 523 526 GENOTYPE\n",
      "LKO 658 661 GENOTYPE\n",
      "Ets1-flox 375 384 GENOTYPE\n",
      "Ets1-flox 405 414 GENOTYPE\n",
      "ZFP251 KO 273 282 GENOTYPE\n",
      "extra copy of the Dyrk1A gene 189 218 GENOTYPE\n",
      "extra copy of the mouse Chr16 syntenic region (Dp(16)1Yey) 249 307 GENOTYPE\n",
      "WT 55 57 GENOTYPE\n",
      "None 181 183 GENOTYPE\n",
      "Skipping entity\n",
      "wild type 2186 2195 GENOTYPE\n",
      "Sox9+/Y440X 555 566 GENOTYPE\n",
      "Sox9+/Y440X 877 888 GENOTYPE\n",
      "Sox9+/Y440X 2393 2404 GENOTYPE\n",
      "WT 276 278 GENOTYPE\n",
      "WT 516 518 GENOTYPE\n",
      "None 157 164 GENOTYPE\n",
      "Skipping entity\n",
      "miR-15a/b+/+ Vav-Cre 1479 1499 GENOTYPE\n",
      "miR-15a/bfl/fl Vav-Cre 1522 1544 GENOTYPE\n",
      "WT 69 71 GENOTYPE\n",
      "WT 179 181 GENOTYPE\n",
      "Ubc-CreERT2; Csf1fl/fl 1321 1343 GENOTYPE\n",
      "PRMT1Ifl/ 312 322 GENOTYPE\n",
      "PRMT1fl/fl; cx3r1cre-ert2 358 383 GENOTYPE\n",
      "None 0 3 GENOTYPE\n",
      "Skipping entity\n",
      "LiK 107 110 GENOTYPE\n",
      "None 272 275 GENOTYPE\n",
      "Skipping entity\n",
      "LiK 341 344 GENOTYPE\n",
      "None 349 352 GENOTYPE\n",
      "Skipping entity\n",
      "LiKM 0 4 GENOTYPE\n",
      "LiKM 272 276 GENOTYPE\n",
      "LiKM 349 353 GENOTYPE\n",
      "EBF1-dTAG_N 531 542 GENOTYPE\n",
      "CTRL 447 451 GENOTYPE\n",
      "BATKO 140 145 GENOTYPE\n",
      "BATKO 204 209 GENOTYPE\n",
      "BATKO 417 422 GENOTYPE\n",
      "dCas9-KRAB 1240 1250 GENOTYPE\n",
      "dCas9-KRAB 1757 1767 GENOTYPE\n",
      "WT 522 524 GENOTYPE\n",
      "WT 690 692 GENOTYPE\n",
      "WT 1181 1183 GENOTYPE\n",
      "Nlrx1-/- 722 730 GENOTYPE\n",
      "Nox2-/- 732 739 GENOTYPE\n",
      "Prx5-/- 744 751 GENOTYPE\n",
      "Ifnar-/- 694 702 GENOTYPE\n",
      "ACTA2 R179H variant 214 233 GENOTYPE\n",
      "ACTA2 R179H variant 588 607 GENOTYPE\n",
      "WT 84 86 GENOTYPE\n",
      "WT 231 233 GENOTYPE\n",
      "WT 346 348 GENOTYPE\n",
      "OT-I 1128 1132 GENOTYPE\n",
      "WT 46 48 GENOTYPE\n",
      "WT 214 216 GENOTYPE\n",
      "WT 675 677 GENOTYPE\n",
      "Nlrx1-/- 264 272 GENOTYPE\n",
      "Nlrx1-/- 697 705 GENOTYPE\n",
      "Nrf2-/- 725 732 GENOTYPE\n",
      "Prx5-/- 304 311 GENOTYPE\n",
      "Prx5-/- 741 748 GENOTYPE\n",
      "Prx5+/+ 760 767 GENOTYPE\n",
      "Ptpmt1 cardiomyocytes-specific Knockout 123 162 GENOTYPE\n",
      "Mrps5 fl/fl 127 138 GENOTYPE\n",
      "Mrps5 fl/fl 143 154 GENOTYPE\n",
      "Mrps5 fl/fl 324 335 GENOTYPE\n",
      "Mrps5 fl/fl 340 351 GENOTYPE\n",
      "Mrps5 fl/fl; cTNT-Cre 143 164 GENOTYPE\n",
      "Mrps5 fl/fl; cTNT-Cre 340 361 GENOTYPE\n",
      "parental 287 295 GENOTYPE\n",
      "control 123 130 GENOTYPE\n",
      "control 289 296 GENOTYPE\n",
      "control 373 380 GENOTYPE\n",
      "control 442 449 GENOTYPE\n",
      "Arid1a KO 301 310 GENOTYPE\n",
      "None 451 453 GENOTYPE\n",
      "Skipping entity\n",
      "None 635 637 GENOTYPE\n",
      "Skipping entity\n",
      "Mrps5 fl/ 461 472 GENOTYPE\n",
      "Mrps5 fl/fl 494 505 GENOTYPE\n",
      "WT 949 951 GENOTYPE\n",
      "WT 1459 1461 GENOTYPE\n",
      "FHL2 overexpression 29 48 GENOTYPE\n",
      "WT 69 71 GENOTYPE\n",
      "WT 220 222 GENOTYPE\n",
      "WT 320 322 GENOTYPE\n",
      "WT 576 578 GENOTYPE\n",
      "PPO1 268 272 GENOTYPE\n",
      "PPO1 399 403 GENOTYPE\n",
      "PPO2 218 222 GENOTYPE\n",
      "PPO2 349 353 GENOTYPE\n",
      "None 123 125 GENOTYPE\n",
      "Skipping entity\n",
      "None 186 188 GENOTYPE\n",
      "Skipping entity\n",
      "WT 310 312 GENOTYPE\n",
      "None 136 138 GENOTYPE\n",
      "Skipping entity\n",
      "None 199 201 GENOTYPE\n",
      "Skipping entity\n",
      "Apoe-/-Ccl17wt/e 427 443 GENOTYPE\n",
      "Apoe-/-Ccl17wt/e 669 685 GENOTYPE\n",
      "Apoe-/-Ccl17wt/e 838 854 GENOTYPE\n",
      "Apoe-/-Ccl17e/e 447 462 GENOTYPE\n",
      "Apoe-/-Ccl17e/e 689 704 GENOTYPE\n",
      "Apoe-/-Ccl17e/e 864 879 GENOTYPE\n",
      "P195A/p150- 1315 1326 GENOTYPE\n",
      "wild type 748 757 GENOTYPE\n",
      "Trem2 KO 199 207 GENOTYPE\n",
      "Trem2 KO 386 394 GENOTYPE\n",
      "AR-flox/y 276 285 GENOTYPE\n",
      "AR-flox/y 304 313 GENOTYPE\n",
      "S100A14 CKO 701 712 GENOTYPE\n",
      "S100A14 CKO 997 1008 GENOTYPE\n",
      "S100A14 CKO 1143 1154 GENOTYPE\n",
      "WT 920 922 GENOTYPE\n",
      "Perk KO 927 934 GENOTYPE\n",
      "C57BL/6J 228 236 GENOTYPE\n",
      "WT 1269 1271 GENOTYPE\n",
      "wild type 804 813 GENOTYPE\n",
      "Osx-Cre, Ai9 239 251 GENOTYPE\n",
      "Hmga2 KO 0 8 GENOTYPE\n",
      "Hmga2 KO 302 310 GENOTYPE\n",
      "Hmga2 KO 385 393 GENOTYPE\n",
      "Hmga2 KO 628 636 GENOTYPE\n",
      "Hmga2 KO 938 946 GENOTYPE\n",
      "Hmga2 KO 1006 1014 GENOTYPE\n",
      "Wild Type 1658 1667 GENOTYPE\n",
      "LSL-KrasG12D mice 0 17 GENOTYPE\n",
      "Fnip2C/C 339 347 GENOTYPE\n",
      "Fnip2T/T 326 334 GENOTYPE\n",
      "C57BL/6J WT 1839 1850 GENOTYPE\n",
      "Flt3-ITD 51 59 GENOTYPE\n",
      "wild type 20 29 GENOTYPE\n",
      "Lyz2Cre+/+YFP+/- 1637 1653 GENOTYPE\n",
      "Lyz2Cre+/+Ctnnb1exon3del/wtYFP+/- 1667 1700 GENOTYPE\n",
      "C57Bl/6 139 146 GENOTYPE\n",
      "B6.SLE123 169 178 GENOTYPE\n",
      "B6.SLE123 426 435 GENOTYPE\n",
      "heterozygous 573 585 GENOTYPE\n",
      "heterozygous 1627 1639 GENOTYPE\n",
      "heterozygous 1804 1816 GENOTYPE\n",
      "shCTL 418 423 GENOTYPE\n",
      "shCTL 474 479 GENOTYPE\n",
      "shCTL 881 886 GENOTYPE\n",
      "YFP- 1172 1176 GENOTYPE\n",
      "YFP+ 1158 1162 GENOTYPE\n",
      "YFP- 1172 1176 GENOTYPE\n",
      "YFP+ 1158 1162 GENOTYPE\n",
      "wild type 1289 1298 GENOTYPE\n",
      "MafA-/-;MafB flox/flox (A0B2) 1305 1334 GENOTYPE\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 60%|██████████████████████▉               | 1087/1804 [00:03<00:01, 367.43it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "WT 251 253 GENOTYPE\n",
      "WT 367 369 GENOTYPE\n",
      "KC 1913 1915 GENOTYPE\n",
      "KC 2958 2960 GENOTYPE\n",
      "KC 3051 3053 GENOTYPE\n",
      "None 3057 3059 GENOTYPE\n",
      "Skipping entity\n",
      "KPC 1920 1923 GENOTYPE\n",
      "KPC 2164 2167 GENOTYPE\n",
      "None 805 807 GENOTYPE\n",
      "Skipping entity\n",
      "None 1271 1273 GENOTYPE\n",
      "Skipping entity\n",
      "Nestin-Cre; P53 f/f; Pten f/+; R26 tdTomato/+ 1109 1154 GENOTYPE\n",
      "Sox2eGFP 977 985 GENOTYPE\n",
      "iTat 72 76 GENOTYPE\n",
      "iTat 375 379 GENOTYPE\n",
      "iTat 411 415 GENOTYPE\n",
      "SETD4 overexpression 855 875 GENOTYPE\n",
      "Adig 2042 2046 GENOTYPE\n",
      "APOE2 93 98 GENOTYPE\n",
      "APOE2 318 323 GENOTYPE\n",
      "APOE3 100 105 GENOTYPE\n",
      "APOE4 111 116 GENOTYPE\n",
      "APOE4 328 333 GENOTYPE\n",
      "None 827 829 GENOTYPE\n",
      "Skipping entity\n",
      "WT 1768 1770 GENOTYPE\n",
      "WT 522 524 GENOTYPE\n",
      "WT 1549 1551 GENOTYPE\n",
      "5KO 1556 1559 GENOTYPE\n",
      "Ts65Dn 87 93 GENOTYPE\n",
      "Ts65Dn 451 457 GENOTYPE\n",
      "BMAd-DTA 71 79 GENOTYPE\n",
      "BMAd-DTA 322 330 GENOTYPE\n",
      "BMAd-DTA 431 439 GENOTYPE\n",
      "BMAd-DTA 674 682 GENOTYPE\n",
      "BMAd-DTA 871 879 GENOTYPE\n",
      "ORP4L knock-in 107 121 GENOTYPE\n",
      "ORP4L knock-in 212 226 GENOTYPE\n",
      "ORP4L knock-in 1140 1154 GENOTYPE\n",
      "Ahrr KO 1134 1141 GENOTYPE\n",
      "WT 56 58 GENOTYPE\n",
      "WT 143 145 GENOTYPE\n",
      "Maf flox/flox 417 430 GENOTYPE\n",
      "Maf flox/flox 485 498 GENOTYPE\n",
      "Maf flox/flox 1435 1448 GENOTYPE\n",
      "None 1538 1540 GENOTYPE\n",
      "Skipping entity\n",
      "WT 1699 1701 GENOTYPE\n",
      "WT 1817 1819 GENOTYPE\n",
      "WT 1844 1846 GENOTYPE\n",
      "Acss2 KO 1568 1576 GENOTYPE\n",
      "Acss2 KO 1703 1711 GENOTYPE\n",
      "Control 1235 1242 GENOTYPE\n",
      "HD 168 170 GENOTYPE\n",
      "HD 618 620 GENOTYPE\n",
      "HD 1228 1230 GENOTYPE\n",
      "wild type 395 404 GENOTYPE\n",
      "wild type 639 648 GENOTYPE\n",
      "WT 50 52 GENOTYPE\n",
      "WT 120 122 GENOTYPE\n",
      "WT 177 179 GENOTYPE\n",
      "None 662 664 GENOTYPE\n",
      "Skipping entity\n",
      "Ptpn1+/+ 145 153 GENOTYPE\n",
      "Ptpn1-/- 158 166 GENOTYPE\n",
      "wild type 199 208 GENOTYPE\n",
      "WT 499 501 GENOTYPE\n",
      "WT 614 616 GENOTYPE\n",
      "WT 805 807 GENOTYPE\n",
      "CD300ld KO 511 521 GENOTYPE\n",
      "CD300ld KO 621 631 GENOTYPE\n",
      "WT 49 51 GENOTYPE\n",
      "WT 92 94 GENOTYPE\n",
      "PRMT1-KO 19 27 GENOTYPE\n",
      "Foxp4fl/fl 326 336 GENOTYPE\n",
      "hACE2 transgenic mice 134 155 GENOTYPE\n",
      "hACE2 transgenic mice 685 706 GENOTYPE\n",
      "hACE2 transgenic mice 728 749 GENOTYPE\n",
      "Sirt2 KO 1621 1629 GENOTYPE\n",
      "Sirt6mKO 335 343 GENOTYPE\n",
      "Sirt6mKO 428 436 GENOTYPE\n",
      "WT 587 589 GENOTYPE\n",
      "WT 1780 1782 GENOTYPE\n",
      "FOXP3 FL 163 171 GENOTYPE\n",
      "FOXP3 FL 1260 1268 GENOTYPE\n",
      "Foxp3 exon 2 deletion 577 598 GENOTYPE\n",
      "Cd4-CreERT2;Tcf7+l+ 543 562 GENOTYPE\n",
      "Cd4-CreERT2;Tcf7fl/fl 518 539 GENOTYPE\n",
      "Hnf4a fl/fl 1057 1068 GENOTYPE\n",
      "Hnf4a fl/fl 1097 1108 GENOTYPE\n",
      "Hnf4a fl/fl Villin-Cre 1097 1119 GENOTYPE\n",
      "Hnf4a fl/fl 1393 1404 GENOTYPE\n",
      "Hnf4a fl/fl 1433 1444 GENOTYPE\n",
      "Hnf4a fl/fl Villin-Cre 1433 1455 GENOTYPE\n",
      "Angptl2+/+ 128 138 GENOTYPE\n",
      "Angptl2+/+ 295 305 GENOTYPE\n",
      "Angptl2-/- 142 152 GENOTYPE\n",
      "Angptl2-/- 309 319 GENOTYPE\n",
      "Flox 693 697 GENOTYPE\n",
      "Flox 842 846 GENOTYPE\n",
      "wild type 550 559 GENOTYPE\n",
      "wild type 588 597 GENOTYPE\n",
      "WT 1481 1483 GENOTYPE\n",
      "5XFAD 1488 1493 GENOTYPE\n",
      "control 151 158 GENOTYPE\n",
      "control 252 259 GENOTYPE\n",
      "control 555 562 GENOTYPE\n",
      "control 656 663 GENOTYPE\n",
      "None 297 307 GENOTYPE\n",
      "Skipping entity\n",
      "Tmem173-/- 535 545 GENOTYPE\n",
      "wild type 129 138 GENOTYPE\n",
      "BOD1f/f 195 202 GENOTYPE\n",
      "/f 219 226 GENOTYPE\n",
      "S100a16+/- 571 581 GENOTYPE\n",
      "S100a16+/- 1882 1892 GENOTYPE\n",
      "wt 104 106 GENOTYPE\n",
      "wt 215 217 GENOTYPE\n",
      "None 231 233 GENOTYPE\n",
      "Skipping entity\n",
      "wt 266 268 GENOTYPE\n",
      "Rbx1-deficient 111 125 GENOTYPE\n",
      "Rbx1-deficient 273 287 GENOTYPE\n",
      "PP2A KO 1585 1592 GENOTYPE\n",
      "LysMcrePP2Acfl/fl 1748 1765 GENOTYPE\n",
      "WT 1741 1743 GENOTYPE\n",
      "Piezo1MGKO; FAD 1036 1051 GENOTYPE\n",
      "Piezo1MGKO; FAD 1153 1168 GENOTYPE\n",
      "Piezo1fl/fl; FAD 1086 1102 GENOTYPE\n",
      "Piezo1fl/fl; FAD 1173 1189 GENOTYPE\n",
      "K320E-skm 2824 2833 GENOTYPE\n",
      "wild-type 2838 2847 GENOTYPE\n",
      "Maf.flox 84 92 GENOTYPE\n",
      "Maf.flox 97 105 GENOTYPE\n",
      "Maf.flox 203 211 GENOTYPE\n",
      "Maf.flox 216 224 GENOTYPE\n",
      "Vil.Cre Maf.flox 76 92 GENOTYPE\n",
      "Vil.Cre Maf.flox 195 211 GENOTYPE\n",
      "control 123 130 GENOTYPE\n",
      "control 290 297 GENOTYPE\n",
      "control 374 381 GENOTYPE\n",
      "control 443 450 GENOTYPE\n",
      "Arid1a KO 302 311 GENOTYPE\n",
      "WT 283 285 GENOTYPE\n",
      "WT 1052 1054 GENOTYPE\n",
      "WT 1322 1324 GENOTYPE\n",
      "WT 1610 1612 GENOTYPE\n",
      "wild type 1045 1054 GENOTYPE\n",
      "H1-/- 1022 1027 GENOTYPE\n",
      "wild type 1045 1054 GENOTYPE\n",
      "H1-/- 1022 1027 GENOTYPE\n",
      "TKO 375 378 GENOTYPE\n",
      "TKO 390 393 GENOTYPE\n",
      "None 418 421 GENOTYPE\n",
      "Skipping entity\n",
      "TKO 468 471 GENOTYPE\n",
      "None 476 479 GENOTYPE\n",
      "Skipping entity\n",
      "TKO_FL 418 424 GENOTYPE\n",
      "TKO_FL 476 482 GENOTYPE\n",
      "control 232 239 GENOTYPE\n",
      "control 416 423 GENOTYPE\n",
      "myometrial PGR knockout 204 227 GENOTYPE\n",
      "WT 138 140 GENOTYPE\n",
      "WT 818 820 GENOTYPE\n",
      "delta-RRE 826 835 GENOTYPE\n",
      "WT 600 602 GENOTYPE\n",
      "WT 1081 1083 GENOTYPE\n",
      "WT 1369 1371 GENOTYPE\n",
      "WT 1604 1606 GENOTYPE\n",
      "WT 1969 1971 GENOTYPE\n",
      "IL6R.ko 2006 2013 GENOTYPE\n",
      "TGFalpha 1451 1459 GENOTYPE\n",
      "WT 172 174 GENOTYPE\n",
      "SIRT5OE 743 750 GENOTYPE\n",
      "SIRT5OE 860 867 GENOTYPE\n",
      "WT 48 50 GENOTYPE\n",
      "WT 495 497 GENOTYPE\n",
      "WT 1404 1406 GENOTYPE\n",
      "WT 1918 1920 GENOTYPE\n",
      "wild-type 987 996 GENOTYPE\n",
      "4SA-Rad knock-in 966 982 GENOTYPE\n",
      "Atf5(rep/+); Omp(iresGFP/+) 1322 1349 GENOTYPE\n",
      "Atf5(rep/+); Omp(iresGFP/+) 1439 1466 GENOTYPE\n",
      "Atf5(rep/+); Omp(iresGFP/+); P2(M71iresLacZ/+) 1439 1485 GENOTYPE\n",
      "AR-deficient 533 545 GENOTYPE\n",
      "control 521 528 GENOTYPE\n",
      "Cd34-CreERT2;R26-tdTomato 322 347 GENOTYPE\n",
      "Cd34-CreERT2;R26-tdTomato 714 739 GENOTYPE\n",
      "WT 2142 2144 GENOTYPE\n",
      "WT 2178 2180 GENOTYPE\n",
      "None 57 62 GENOTYPE\n",
      "Skipping entity\n",
      "mGFAPTg170-2 68 80 GENOTYPE\n",
      "KPC;YFP 136 143 GENOTYPE\n",
      "KPC;YFP 369 376 GENOTYPE\n",
      "None 658 665 GENOTYPE\n",
      "Skipping entity\n",
      "KPC;YFP;aSMA-RFP 658 674 GENOTYPE\n",
      "Clr-f-/- 1386 1394 GENOTYPE\n",
      "Tbl1xr1Y446C/Y446C 425 443 GENOTYPE\n",
      "wild type 411 420 GENOTYPE\n",
      "nontransgenic mice 298 316 GENOTYPE\n",
      "nontransgenic mice 458 476 GENOTYPE\n",
      "WT 1252 1254 GENOTYPE\n",
      "WT 1407 1409 GENOTYPE\n",
      "WT 1436 1438 GENOTYPE\n",
      "Gabbr1 control 181 195 GENOTYPE\n",
      "Gabbr1 control 370 384 GENOTYPE\n",
      "Gabbr1 cKO 200 210 GENOTYPE\n",
      "wildtype 1167 1175 GENOTYPE\n",
      "Dlg1 conditional knock out 1106 1132 GENOTYPE\n",
      "WT 73 75 GENOTYPE\n",
      "WT 612 614 GENOTYPE\n",
      "WT 829 831 GENOTYPE\n",
      "Gsdmdfl/fl Cx3cr1-cre 2092 2113 GENOTYPE\n",
      "Gsdmdfl/fl Cx3cr1-cre 2422 2443 GENOTYPE\n",
      "Cx3cr1-cre 2103 2113 GENOTYPE\n",
      "Cx3cr1-cre 2118 2128 GENOTYPE\n",
      "Cx3cr1-cre 2407 2417 GENOTYPE\n",
      "Cx3cr1-cre 2433 2443 GENOTYPE\n",
      "Slc25a43-KO 1118 1129 GENOTYPE\n",
      "GRf/f 240 245 GENOTYPE\n",
      "GRf/f 378 383 GENOTYPE\n",
      "GRmKO 279 284 GENOTYPE\n",
      "GRmKO 387 392 GENOTYPE\n",
      "WT 317 319 GENOTYPE\n",
      "shCTR 1220 1225 GENOTYPE\n",
      "shSF3B1 1230 1237 GENOTYPE\n",
      "WT 212 214 GENOTYPE\n",
      "Gabbr1 control 94 108 GENOTYPE\n",
      "Gabbr1 control 204 218 GENOTYPE\n",
      "Gabbr1 cKO 113 123 GENOTYPE\n",
      "KO 475 477 GENOTYPE\n",
      "HET 466 469 GENOTYPE\n",
      "None 462 464 GENOTYPE\n",
      "Skipping entity\n",
      "heterozygous CX3CR1 GFP knock-in mice (CX3CR1 GFP/+) 266 318 GENOTYPE\n",
      "Wild-type 140 149 GENOTYPE\n",
      "Wild-type 1525 1534 GENOTYPE\n",
      "Alkbh5-/- 125 134 GENOTYPE\n",
      "Alkbh5-/- 1510 1519 GENOTYPE\n",
      "wild type 196 205 GENOTYPE\n",
      "WT 1566 1568 GENOTYPE\n",
      "5xFAD 109 114 GENOTYPE\n",
      "5xFAD 742 747 GENOTYPE\n",
      "5xFAD 881 886 GENOTYPE\n",
      "5xFAD 1182 1187 GENOTYPE\n",
      "5xFAD 1556 1561 GENOTYPE\n",
      "CDK5f/f 364 371 GENOTYPE\n",
      "CDK5f/f 1089 1096 GENOTYPE\n",
      "/f 1119 1126 GENOTYPE\n",
      "Cdh5-CreERT2;CDK5f/f 1106 1126 GENOTYPE\n",
      "Zfp266 KO 55 64 GENOTYPE\n",
      "Zfp266 KO 242 251 GENOTYPE\n",
      "Zfp266 KO 337 346 GENOTYPE\n",
      "WT 1230 1232 GENOTYPE\n",
      "wild-type 301 310 GENOTYPE\n",
      "FFI 240 243 GENOTYPE\n",
      "FFI 637 640 GENOTYPE\n",
      "FFI 734 737 GENOTYPE\n",
      "FFI 1337 1340 GENOTYPE\n",
      "CJD 206 209 GENOTYPE\n",
      "CJD 629 632 GENOTYPE\n",
      "CJD 739 742 GENOTYPE\n",
      "CJD 1356 1359 GENOTYPE\n",
      "Control 1345 1352 GENOTYPE\n",
      "Control 1364 1371 GENOTYPE\n",
      "FOXO3 KO 1626 1634 GENOTYPE\n",
      "Pparafl/fl 402 412 GENOTYPE\n",
      "Aldh1l1-Cre/ERT2 x Ribotag 175 201 GENOTYPE\n",
      "control 160 167 GENOTYPE\n",
      "Nfatc1-fl/fl (WT) 644 661 GENOTYPE\n",
      "None 1540 1542 GENOTYPE\n",
      "Skipping entity\n",
      "None 1562 1568 GENOTYPE\n",
      "Skipping entity\n",
      "None 1582 1588 GENOTYPE\n",
      "Skipping entity\n",
      "WT 1334 1336 GENOTYPE\n",
      "WT 1338 1340 GENOTYPE\n",
      "WWP2Mut/Mut 1349 1360 GENOTYPE\n",
      "WWP2Mut/Mut 1366 1377 GENOTYPE\n",
      "WT 1252 1254 GENOTYPE\n",
      "WT 1252 1254 GENOTYPE\n",
      "WT 1281 1283 GENOTYPE\n",
      "circSry KO 259 269 GENOTYPE\n",
      "ATP7B-/- 524 532 GENOTYPE\n",
      "ATP7B-/- 1017 1025 GENOTYPE\n",
      "ATP7B-/- 1507 1515 GENOTYPE\n",
      "CaMKK2 KO 943 952 GENOTYPE\n",
      "NRAS 61R/R 158 168 GENOTYPE\n",
      "NRAS 61K/K 172 182 GENOTYPE\n",
      "NRAS 61L/L 186 196 GENOTYPE\n",
      "NRAS 61H/H 200 210 GENOTYPE\n",
      "WT 104 106 GENOTYPE\n",
      "WT 214 216 GENOTYPE\n",
      "CD4-CrexR26-LSL-YFPxSt2fl/fl 312 340 GENOTYPE\n",
      "control 1059 1066 GENOTYPE\n",
      "WT 181 183 GENOTYPE\n",
      "KO 185 187 GENOTYPE\n",
      "ANRIL knockout 709 723 GENOTYPE\n",
      "ANRIL knockout 1333 1347 GENOTYPE\n",
      "ANRIL knockout 2010 2024 GENOTYPE\n",
      "control 280 287 GENOTYPE\n",
      "None 417 424 GENOTYPE\n",
      "Skipping entity\n",
      "VAV-Cre WT 1252 1262 GENOTYPE\n",
      "IL-1rn-KO 518 527 GENOTYPE\n",
      "WT 656 658 GENOTYPE\n",
      "None 751 754 GENOTYPE\n",
      "Skipping entity\n",
      "Arc 759 762 GENOTYPE\n",
      "None 889 892 GENOTYPE\n",
      "Skipping entity\n",
      "None 970 973 GENOTYPE\n",
      "Skipping entity\n",
      "None 980 983 GENOTYPE\n",
      "Skipping entity\n",
      "Arc 1050 1053 GENOTYPE\n",
      "None 1462 1465 GENOTYPE\n",
      "Skipping entity\n",
      "Arc 2049 2052 GENOTYPE\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 64%|████████████████████████▍             | 1163/1804 [00:04<00:01, 330.99it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CreERT2Commd3flox/+ 131 150 GENOTYPE\n",
      "CreERT2Commd3flox/+ 1047 1066 GENOTYPE\n",
      "CreERT2Commd3flox/flox 155 177 GENOTYPE\n",
      "CreERT2Commd3flox/flox 1071 1093 GENOTYPE\n",
      "WT 464 466 GENOTYPE\n",
      "WT 1227 1229 GENOTYPE\n",
      "COMMD3C170A/C170A 718 735 GENOTYPE\n",
      "COMMD3C170A/C170A 1233 1250 GENOTYPE\n",
      "wild type 510 519 GENOTYPE\n",
      "wild type 570 579 GENOTYPE\n",
      "lnc57Rik-/- 586 597 GENOTYPE\n",
      "lncGM1082-/- 599 611 GENOTYPE\n",
      "WT 1780 1782 GENOTYPE\n",
      "NOX4 KO 1787 1794 GENOTYPE\n",
      "floxB/floxB 339 356 GENOTYPE\n",
      "Keap1 floxB/floxB 369 386 GENOTYPE\n",
      "control 937 944 GENOTYPE\n",
      "Prox1-GFP 68 77 GENOTYPE\n",
      "Prox1- 142 151 GENOTYPE\n",
      "Prox1- 212 221 GENOTYPE\n",
      "Prox1-GFP 256 265 GENOTYPE\n",
      "miR-21-knockdown 1723 1739 GENOTYPE\n",
      "transgenic 513 523 GENOTYPE\n",
      "transgenic 727 737 GENOTYPE\n",
      "transgenic 1079 1089 GENOTYPE\n",
      "wildtype 714 722 GENOTYPE\n",
      "wildtype 955 963 GENOTYPE\n",
      "wild type 1121 1130 GENOTYPE\n",
      "wild-type 74 83 GENOTYPE\n",
      "wild-type 190 199 GENOTYPE\n",
      "control 279 286 GENOTYPE\n",
      "control 1900 1907 GENOTYPE\n",
      "control 2046 2053 GENOTYPE\n",
      "miR-26a -/- 233 244 GENOTYPE\n",
      "miR-26a -/- 638 649 GENOTYPE\n",
      "Alb-Cre/Foxk1(F/+) 542 560 GENOTYPE\n",
      "Alb-Cre/Foxk1(F/F) 565 583 GENOTYPE\n",
      "VilCreER 106 114 GENOTYPE\n",
      "RIPK1-KO 334 342 GENOTYPE\n",
      "control 591 598 GENOTYPE\n",
      "APP/PS1 1270 1277 GENOTYPE\n",
      "APP/PS1 1282 1289 GENOTYPE\n",
      "KDM6B KO 38 46 GENOTYPE\n",
      "KDM6B KO 79 87 GENOTYPE\n",
      "control 104 111 GENOTYPE\n",
      "control 265 272 GENOTYPE\n",
      "None 177 179 GENOTYPE\n",
      "Skipping entity\n",
      "None 390 392 GENOTYPE\n",
      "Skipping entity\n",
      "siPRMT1 184 191 GENOTYPE\n",
      "siPRMT1 397 404 GENOTYPE\n",
      "None 183 188 GENOTYPE\n",
      "Skipping entity\n",
      "Mcemp1-/- 38 47 GENOTYPE\n",
      "Mcemp1-/- 213 222 GENOTYPE\n",
      "Mcemp1-/- 38 47 GENOTYPE\n",
      "Mcemp1-/- 182 191 GENOTYPE\n",
      "RelA-flfl 1142 1151 GENOTYPE\n",
      "RelA-flfl 1267 1276 GENOTYPE\n",
      "RelA-flfl x Cd19-cre 1267 1287 GENOTYPE\n",
      "RelA-flfl 1262 1271 GENOTYPE\n",
      "WT 1097 1099 GENOTYPE\n",
      "wild type 44 53 GENOTYPE\n",
      "wild type 207 216 GENOTYPE\n",
      "Mll5-/- 57 64 GENOTYPE\n",
      "Mll5-/- 226 233 GENOTYPE\n",
      "BLTcre.mcl1fl/+ 1080 1095 GENOTYPE\n",
      "BLTcre.mcl1fl/+ 1276 1291 GENOTYPE\n",
      "N40K 223 227 GENOTYPE\n",
      "N40K 348 352 GENOTYPE\n",
      "N40K 367 371 GENOTYPE\n",
      "N40K 453 457 GENOTYPE\n",
      "N40K 475 479 GENOTYPE\n",
      "N40K 664 668 GENOTYPE\n",
      "N40K 685 689 GENOTYPE\n",
      "N40K 731 735 GENOTYPE\n",
      "5xFAD 697 702 GENOTYPE\n",
      "5xFAD 743 748 GENOTYPE\n",
      "Tau P301S 816 825 GENOTYPE\n",
      "EP3F/F 1001 1007 GENOTYPE\n",
      "/F 1020 1026 GENOTYPE\n",
      "NKp46CreEP3F/F 1012 1026 GENOTYPE\n",
      "wild type 1384 1393 GENOTYPE\n",
      "knock-in Col3a1+/G182R 234 256 GENOTYPE\n",
      "wild type 1213 1222 GENOTYPE\n",
      "SP-CI73T 708 723 GENOTYPE\n",
      "wild type 75 84 GENOTYPE\n",
      "lnc956 knockout 149 164 GENOTYPE\n",
      "wild type 395 404 GENOTYPE\n",
      "wild type 547 556 GENOTYPE\n",
      "GSDMD-/- 382 390 GENOTYPE\n",
      "GSDMD-/- 534 542 GENOTYPE\n",
      "GSDMD-/- 677 685 GENOTYPE\n",
      "CD4-specific KLF10 knockout mice 203 235 GENOTYPE\n",
      "GFP+Lin-c-Kit+ Usp18f/f 743 766 GENOTYPE\n",
      "wild type 57 66 GENOTYPE\n",
      "wild type 745 754 GENOTYPE\n",
      "Hoil1-/-Il4raflox/flox 1645 1667 GENOTYPE\n",
      "Hoil1-/-Il4raflox/ 1673 1695 GENOTYPE\n",
      "Hoil1-/-Il4raflox/flox+Vil1cre 1673 1703 GENOTYPE\n",
      "Hoil1+/+Il4raflox/flox 1589 1611 GENOTYPE\n",
      "Hoil1+/+Il4raflox/ 1613 1635 GENOTYPE\n",
      "Hoil1+/+Il4raflox/flox+Vil1cre 1613 1643 GENOTYPE\n",
      "WT 124 126 GENOTYPE\n",
      "WT 144 146 GENOTYPE\n",
      "WT 810 812 GENOTYPE\n",
      "WT 1388 1390 GENOTYPE\n",
      "Pou2f3-/- 55 64 GENOTYPE\n",
      "Pou2f3-/- 1347 1356 GENOTYPE\n",
      "Gpr151 WT 742 751 GENOTYPE\n",
      "Gpr151 WT 1139 1148 GENOTYPE\n",
      "Gpr151 KO 1070 1079 GENOTYPE\n",
      "RUS KD 1325 1331 GENOTYPE\n",
      "Arid1a f/f 316 326 GENOTYPE\n",
      "Arid1a f/f 342 352 GENOTYPE\n",
      "Ltf iCre/+ Arid1a f/f 331 352 GENOTYPE\n",
      "Fam111a+/+ 1497 1507 GENOTYPE\n",
      "Fam111a-/- 540 550 GENOTYPE\n",
      "Fam111a-/- 565 575 GENOTYPE\n",
      "Fam111a-/- 1181 1191 GENOTYPE\n",
      "Fam111a-/- 1251 1261 GENOTYPE\n",
      "Fam111a-/- 1531 1541 GENOTYPE\n",
      "wildtype 1769 1777 GENOTYPE\n",
      "WT 419 421 GENOTYPE\n",
      "KO 212 214 GENOTYPE\n",
      "RE 341 343 GENOTYPE\n",
      "Sox9fl/fl;Sox1Cre 352 369 GENOTYPE\n",
      "Sox9fl/fl;Sox1Cre 609 626 GENOTYPE\n",
      "Sox9fl/ 850 867 GENOTYPE\n",
      "Sox9fl/ 1288 1305 GENOTYPE\n",
      "Sox9fl/fl;Sox1Cre 3129 3146 GENOTYPE\n",
      "Sox9fl/fl;NestinCre 374 393 GENOTYPE\n",
      "Sox9fl/fl;NestinCre 631 650 GENOTYPE\n",
      "Sox9fl/fl;NestinCre 3151 3170 GENOTYPE\n",
      "Wild type 1725 1734 GENOTYPE\n",
      "Nkx3.1CreERT2/+; Ptenflox/flox; K-RasLSLG12D/+; R26r-Confetti 1141 1202 GENOTYPE\n",
      "miR-124a-1/2 DKO 639 655 GENOTYPE\n",
      "miR-124a-1/2 DKO 838 854 GENOTYPE\n",
      "miR-124a-1/2 DKO 980 996 GENOTYPE\n",
      "miR-124a-1/2 DKO 1148 1164 GENOTYPE\n",
      "miR-124a-1/2 DKO 1776 1792 GENOTYPE\n",
      "Foxp3-Gfp 1094 1103 GENOTYPE\n",
      "WT 1388 1390 GENOTYPE\n",
      "WT 1418 1420 GENOTYPE\n",
      "Tlr7Y264H 972 981 GENOTYPE\n",
      "WT 8 10 GENOTYPE\n",
      "wild type 58 67 GENOTYPE\n",
      "wild type 245 254 GENOTYPE\n",
      "wild type 320 329 GENOTYPE\n",
      "wild type 1101 1110 GENOTYPE\n",
      "wild type 1834 1843 GENOTYPE\n",
      "Hnrnpul1-/- 91 102 GENOTYPE\n",
      "Hnrnpul1-/- 259 270 GENOTYPE\n",
      "Hnrnpul1-/- 334 345 GENOTYPE\n",
      "Hnrnpul1-/- 1193 1204 GENOTYPE\n",
      "Hnrnpul1-/- 1312 1323 GENOTYPE\n",
      "Hnrnpul1-/- 1848 1859 GENOTYPE\n",
      "WT 31 33 GENOTYPE\n",
      "wild type 969 978 GENOTYPE\n",
      "wild type 1161 1170 GENOTYPE\n",
      "None 372 374 GENOTYPE\n",
      "Skipping entity\n",
      "None 646 648 GENOTYPE\n",
      "Skipping entity\n",
      "None 787 789 GENOTYPE\n",
      "Skipping entity\n",
      "alpha7-nAChR knockout 230 251 GENOTYPE\n",
      "WT 264 266 GENOTYPE\n",
      "WT 298 300 GENOTYPE\n",
      "Cdc14ab KO 1346 1356 GENOTYPE\n",
      "Bax-/-,Bak-/- 1072 1085 GENOTYPE\n",
      "None 1096 1109 GENOTYPE\n",
      "Skipping entity\n",
      "Bax-/-,Bak-/-,Mcl-1-/- 1096 1118 GENOTYPE\n",
      "Lrat-Cre; Prmt1f/f 112 130 GENOTYPE\n",
      "Lrat-Cre; Prmt1f/f 241 259 GENOTYPE\n",
      "Lrat-Cre; Prmt1f/f 284 302 GENOTYPE\n",
      "Lrat-Cre; Prmt1f/f 443 461 GENOTYPE\n",
      "Lrat-Cre; Prmt1f/f 866 884 GENOTYPE\n",
      "Lrat-Cre; Prmt1f/f 959 977 GENOTYPE\n",
      "Lrat-Cre; Prmt1f/f 1170 1188 GENOTYPE\n",
      "Lrat-Cre; Prmt1f/f 1479 1497 GENOTYPE\n",
      "Prmt1f/f 94 102 GENOTYPE\n",
      "Prmt1f/f 122 130 GENOTYPE\n",
      "Prmt1f/f 215 223 GENOTYPE\n",
      "Prmt1f/f 251 259 GENOTYPE\n",
      "Prmt1f/f 266 274 GENOTYPE\n",
      "Prmt1f/f 294 302 GENOTYPE\n",
      "Prmt1f/f 430 438 GENOTYPE\n",
      "Prmt1f/f 453 461 GENOTYPE\n",
      "Prmt1f/f 853 861 GENOTYPE\n",
      "Prmt1f/f 876 884 GENOTYPE\n",
      "Prmt1f/f 946 954 GENOTYPE\n",
      "Prmt1f/f 969 977 GENOTYPE\n",
      "Prmt1f/f 1180 1188 GENOTYPE\n",
      "Prmt1f/f 1489 1497 GENOTYPE\n",
      "Prmt1f/f 1507 1515 GENOTYPE\n",
      "VSX2 SE WT 1525 1535 GENOTYPE\n",
      "VSX2 SE KO 1565 1575 GENOTYPE\n",
      "VSX2 EN1 WT 1605 1616 GENOTYPE\n",
      "VSX2 EN1 KO 1648 1659 GENOTYPE\n",
      "WT 1260 1262 GENOTYPE\n",
      "MCM2-2A 398 405 GENOTYPE\n",
      "MCM2-2A 782 789 GENOTYPE\n",
      "MCM2-2A 1272 1279 GENOTYPE\n",
      "wild type 313 322 GENOTYPE\n",
      "wild type 3761 3770 GENOTYPE\n",
      "Bend2 WT 149 157 GENOTYPE\n",
      "Bend2 WT 358 366 GENOTYPE\n",
      "Maf-wt 1333 1339 GENOTYPE\n",
      "Maf-wt 1749 1755 GENOTYPE\n",
      "WT 1934 1936 GENOTYPE\n",
      "Dnajb6+/- heterozygous mutant 382 411 GENOTYPE\n",
      "E0771GFP cells 223 237 GENOTYPE\n",
      "E0771GFP cells 839 853 GENOTYPE\n",
      "M12GFP cells 250 262 GENOTYPE\n",
      "Hv1 null 95 103 GENOTYPE\n",
      "Mettl7a1 151 159 GENOTYPE\n",
      "Mettl7a1 212 220 GENOTYPE\n",
      "Mettl7a1 294 302 GENOTYPE\n",
      "Mettl7a1 1268 1276 GENOTYPE\n",
      "Mettl7a1 151 159 GENOTYPE\n",
      "Mettl7a1 212 220 GENOTYPE\n",
      "Mettl7a1 294 302 GENOTYPE\n",
      "mock 1219 1223 GENOTYPE\n",
      "OVE26 643 648 GENOTYPE\n",
      "OVE26 1539 1544 GENOTYPE\n",
      "Gen1-KO 148 155 GENOTYPE\n",
      "Gen1-KO 381 388 GENOTYPE\n",
      "Gen1-KO 849 856 GENOTYPE\n",
      "Mus81-knockout 55 69 GENOTYPE\n",
      "Gen1-Mus81-double knockout 75 101 GENOTYPE\n",
      "WT 111 113 GENOTYPE\n",
      "WT 202 204 GENOTYPE\n",
      "WT 1687 1689 GENOTYPE\n",
      "Alb-hKL 146 153 GENOTYPE\n",
      "Alb-hKL 234 241 GENOTYPE\n",
      "WT 158 160 GENOTYPE\n",
      "WT 248 250 GENOTYPE\n",
      "Prdm6f/f; SM22-Cre 41 59 GENOTYPE\n",
      "Prdm6f/f; SM22-Cre 191 209 GENOTYPE\n",
      "Prdm6f/f; SM22-Cre 415 433 GENOTYPE\n",
      "Prdm6f/f; SM22-Cre 523 541 GENOTYPE\n",
      "Prdm6f/f; Wnt1-Cre2; ZsGreen1 45 74 GENOTYPE\n",
      "Prdm6+/+; Wnt1-Cre2; ZsGreen1 115 144 GENOTYPE\n",
      "Pim1-/- 1108 1115 GENOTYPE\n",
      "Pim1-/- 1144 1151 GENOTYPE\n",
      "WT 1054 1056 GENOTYPE\n",
      "WT 1082 1084 GENOTYPE\n",
      "Syk fl/fl 1184 1193 GENOTYPE\n",
      "Syk fl/fl 1210 1219 GENOTYPE\n",
      "Syk fl/fl 1223 1232 GENOTYPE\n",
      "Syk fl/fl 1255 1264 GENOTYPE\n",
      "Cx3cr1-creER Syk fl/fl 1197 1219 GENOTYPE\n",
      "Cx3cr1-creER Syk fl/fl 1242 1264 GENOTYPE\n",
      "Syk fl/fl 5XFAD 1223 1238 GENOTYPE\n",
      "Syk fl/fl 5XFAD 1255 1270 GENOTYPE\n",
      "Cx3cr1-creER Syk fl/fl 5XFAD 1242 1270 GENOTYPE\n",
      "Trem2 WT 5XFAD 1279 1293 GENOTYPE\n",
      "Trem2 KO 5XFAD 1301 1315 GENOTYPE\n",
      "Jmjd3 knock out 214 229 GENOTYPE\n",
      "wild type 116 125 GENOTYPE\n",
      "wild type 322 331 GENOTYPE\n",
      "GTF2ird1-/- 130 141 GENOTYPE\n",
      "GTF2ird1-/- 160 171 GENOTYPE\n",
      "GTF2ird1-/- 340 351 GENOTYPE\n",
      "wild type 1641 1650 GENOTYPE\n",
      "second generation TERC-/- 1660 1685 GENOTYPE\n",
      "Slc25a47hep+/+ 1941 1955 GENOTYPE\n",
      "Slc25a47hep-/- 753 767 GENOTYPE\n",
      "Slc25a47hep-/- 783 797 GENOTYPE\n",
      "Slc25a47hep-/- 1922 1936 GENOTYPE\n",
      "CD18DLy6G 916 925 GENOTYPE\n",
      "CD18DLy6G 1055 1064 GENOTYPE\n",
      "CD18DLy6G 1789 1798 GENOTYPE\n",
      "CD18fl/fl 1074 1083 GENOTYPE\n",
      "CD18fl/fl 1775 1784 GENOTYPE\n",
      "wild type 3001 3010 GENOTYPE\n",
      "None 3020 3030 GENOTYPE\n",
      "Skipping entity\n",
      "wild type 1211 1220 GENOTYPE\n",
      "wild type 1321 1330 GENOTYPE\n",
      "CST-KO 172 178 GENOTYPE\n",
      "CST-KO 336 342 GENOTYPE\n",
      "CST-KO 706 712 GENOTYPE\n",
      "CST-KO 843 849 GENOTYPE\n",
      "miR-301a-/- 107 118 GENOTYPE\n",
      "miR-301a-/- 233 244 GENOTYPE\n",
      "miR-301a-/- 274 285 GENOTYPE\n",
      "A20fl/fl mice with EIIa-Cre mice 1356 1388 GENOTYPE\n",
      "loxp/loxp 469 483 GENOTYPE\n",
      "loxp/loxp; Stra8-Cre 469 494 GENOTYPE\n",
      "WT 201 203 GENOTYPE\n",
      "WT 1702 1704 GENOTYPE\n",
      "WT 11 13 GENOTYPE\n",
      "WT 367 369 GENOTYPE\n",
      "Mfn1-/- 1482 1489 GENOTYPE\n",
      "Mfn2-/- 1491 1498 GENOTYPE\n",
      "wild type 477 486 GENOTYPE\n",
      "None 229 237 GENOTYPE\n",
      "Skipping entity\n",
      "Aanat-/- 496 504 GENOTYPE\n",
      "wild type 1124 1133 GENOTYPE\n",
      "Bnc1tr/tr 1143 1152 GENOTYPE\n",
      "Bnc1tr/tr 1145 1154 GENOTYPE\n",
      "wild type 1126 1135 GENOTYPE\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 70%|██████████████████████████▋           | 1269/1804 [00:04<00:01, 330.59it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CIP-KI-flox 1626 1637 GENOTYPE\n",
      "CIP-KI- 1767 1778 GENOTYPE\n",
      "CIP-KI-flox 2038 2049 GENOTYPE\n",
      "CIP-OE 1525 1531 GENOTYPE\n",
      "CIP-OE 1612 1618 GENOTYPE\n",
      "CIP-OE 1759 1765 GENOTYPE\n",
      "CIP-OE 2007 2013 GENOTYPE\n",
      "Znhit1 flox/flox 113 129 GENOTYPE\n",
      "Znhit1 flox/flox 297 313 GENOTYPE\n",
      "Znhit1 flox/flox 336 352 GENOTYPE\n",
      "BDNF Val66Met 375 388 GENOTYPE\n",
      "BDNF Val66Met 475 488 GENOTYPE\n",
      "BDNF Val66Met 695 708 GENOTYPE\n",
      "WT 37 39 GENOTYPE\n",
      "Scd2f/ 1392 1399 GENOTYPE\n",
      "Scd2f/f 1415 1422 GENOTYPE\n",
      "Scd2f/f;Col1a1-Cre 1392 1410 GENOTYPE\n",
      "wild type 978 987 GENOTYPE\n",
      "Adora1 knockout 94 109 GENOTYPE\n",
      "Adora1 knockout 924 939 GENOTYPE\n",
      "Jmjd1c B KO 383 394 GENOTYPE\n",
      "WT 375 377 GENOTYPE\n",
      "sham 132 136 GENOTYPE\n",
      "EV-circSCMH1 184 196 GENOTYPE\n",
      "KO 422 424 GENOTYPE\n",
      "WT 1215 1217 GENOTYPE\n",
      "WT 632 634 GENOTYPE\n",
      "Trem2(-/-) 617 627 GENOTYPE\n",
      "Fip200 KO 462 471 GENOTYPE\n",
      "Fip200 KO 590 599 GENOTYPE\n",
      "Atg5 KO 473 480 GENOTYPE\n",
      "Atg5 KO 601 608 GENOTYPE\n",
      "Atg7 KO 486 493 GENOTYPE\n",
      "Atg7 KO 610 617 GENOTYPE\n",
      "wt 492 494 GENOTYPE\n",
      "AR100Q 499 505 GENOTYPE\n",
      "KO 1255 1257 GENOTYPE\n",
      "None 1265 1267 GENOTYPE\n",
      "Skipping entity\n",
      "KO 1391 1393 GENOTYPE\n",
      "None 1401 1403 GENOTYPE\n",
      "Skipping entity\n",
      "WT 1202 1204 GENOTYPE\n",
      "None 1212 1214 GENOTYPE\n",
      "Skipping entity\n",
      "WT 1323 1325 GENOTYPE\n",
      "None 1333 1335 GENOTYPE\n",
      "Skipping entity\n",
      "wild type 1118 1127 GENOTYPE\n",
      "wild type 1204 1213 GENOTYPE\n",
      "Senp1-/- 1273 1281 GENOTYPE\n",
      "None 1283 1291 GENOTYPE\n",
      "Skipping entity\n",
      "Senp1+/+ 1263 1271 GENOTYPE\n",
      "Senp1-/-;RIPK1D138N/D138N 1283 1308 GENOTYPE\n",
      "Senp1flox/flox 1330 1344 GENOTYPE\n",
      "Senp1flox/ 1346 1360 GENOTYPE\n",
      "Senp1flox/flox;AlbCre/+ 1346 1369 GENOTYPE\n",
      "Alkbh1 f/f 393 403 GENOTYPE\n",
      "Alkbh1 f/f 643 653 GENOTYPE\n",
      "Alkbh1 f/f, Albumine-cre 393 417 GENOTYPE\n",
      "wildtype 87 95 GENOTYPE\n",
      "wildtype 291 299 GENOTYPE\n",
      "WT 132 134 GENOTYPE\n",
      "WT 349 351 GENOTYPE\n",
      "WT 583 585 GENOTYPE\n",
      "WT 820 822 GENOTYPE\n",
      "WT 360 362 GENOTYPE\n",
      "Colec11-/- 432 442 GENOTYPE\n",
      "Colec11-/- 861 871 GENOTYPE\n",
      "wild type 260 269 GENOTYPE\n",
      "wild type 377 386 GENOTYPE\n",
      "RBPjcKO 277 284 GENOTYPE\n",
      "RBPjcKO 396 403 GENOTYPE\n",
      "WT 888 890 GENOTYPE\n",
      "WT 637 639 GENOTYPE\n",
      "SARAF KO 10 18 GENOTYPE\n",
      "NTG 1386 1389 GENOTYPE\n",
      "Fl-Tcf21iCre 1408 1420 GENOTYPE\n",
      "Tg-Tcf21iCre 1449 1461 GENOTYPE\n",
      "flox control 1870 1882 GENOTYPE\n",
      "Cdh5-Cre; PDGFRb fl/fl 451 473 GENOTYPE\n",
      "PDGFRb fl/fl 427 439 GENOTYPE\n",
      "PDGFRb fl/fl 461 473 GENOTYPE\n",
      "Col1a2- 109 121 GENOTYPE\n",
      "Col1a2- 917 929 GENOTYPE\n",
      "Col1a2-Foxf2-TRAMP 141 159 GENOTYPE\n",
      "Col1a2-Foxf2-TRAMP 344 362 GENOTYPE\n",
      "Col1a2-Foxf2-TRAMP 556 574 GENOTYPE\n",
      "Col1a2-Foxf2-TRAMP 833 851 GENOTYPE\n",
      "Col1a2-Foxf2-TRAMP 943 961 GENOTYPE\n",
      "TaglnCre;Elnf/f 80 95 GENOTYPE\n",
      "TaglnCre;Elnf/f 172 187 GENOTYPE\n",
      "1 Elnf/f and 2 Elnf/+ 268 289 GENOTYPE\n",
      "Sirt5+/+ 1589 1597 GENOTYPE\n",
      "Sirt5-/- 1604 1612 GENOTYPE\n",
      "WT 1626 1628 GENOTYPE\n",
      "wild type 625 634 GENOTYPE\n",
      "Prdm16-het 641 651 GENOTYPE\n",
      "Prdm16-null 503 514 GENOTYPE\n",
      "Prdm16-null 656 667 GENOTYPE\n",
      "wild type 81 90 GENOTYPE\n",
      "wild type 194 203 GENOTYPE\n",
      "WT 1627 1629 GENOTYPE\n",
      "Acly 352 356 GENOTYPE\n",
      "Acly 546 550 GENOTYPE\n",
      "Acly 746 750 GENOTYPE\n",
      "None 870 874 GENOTYPE\n",
      "Skipping entity\n",
      "ATG7FF 1254 1260 GENOTYPE\n",
      "None 1265 1271 GENOTYPE\n",
      "Skipping entity\n",
      "ATG7FF 1410 1416 GENOTYPE\n",
      "None 1436 1442 GENOTYPE\n",
      "Skipping entity\n",
      "None 1491 1497 GENOTYPE\n",
      "Skipping entity\n",
      "None 1509 1515 GENOTYPE\n",
      "Skipping entity\n",
      "ATG7FF;Cx3cr1CreER 1265 1283 GENOTYPE\n",
      "ATG7FF;Cx3cr1CreER 1436 1454 GENOTYPE\n",
      "None 1509 1527 GENOTYPE\n",
      "Skipping entity\n",
      "asrij floxed 464 476 GENOTYPE\n",
      "asrij floxed 862 874 GENOTYPE\n",
      "asrij KO 296 304 GENOTYPE\n",
      "asrij KO 452 460 GENOTYPE\n",
      "asrij KO 878 886 GENOTYPE\n",
      "WT 1535 1537 GENOTYPE\n",
      "Elf4fl/fl-Villin-Cre 12 32 GENOTYPE\n",
      "Elf4fl/fl-Villin-Cre 128 148 GENOTYPE\n",
      "Foxn1 del AR3 1129 1146 GENOTYPE\n",
      "nu/nu 1215 1220 GENOTYPE\n",
      "Dpf2f/f 353 360 GENOTYPE\n",
      "Dpf2f/f 449 456 GENOTYPE\n",
      "Dpf2f/f 538 545 GENOTYPE\n",
      "Dpf2f/f 676 683 GENOTYPE\n",
      "Dpf2f/f 1821 1828 GENOTYPE\n",
      "Dpf2D/D 365 372 GENOTYPE\n",
      "Dpf2D/D 461 468 GENOTYPE\n",
      "Dpf2D/D 550 557 GENOTYPE\n",
      "Dpf2D/D 688 695 GENOTYPE\n",
      "Dpf2D/D 1833 1840 GENOTYPE\n",
      "Dpf2f/f 353 360 GENOTYPE\n",
      "Dpf2f/f 449 456 GENOTYPE\n",
      "Dpf2f/f 538 545 GENOTYPE\n",
      "Dpf2f/f 676 683 GENOTYPE\n",
      "Dpf2f/f 1536 1543 GENOTYPE\n",
      "Dpf2D/D 365 372 GENOTYPE\n",
      "Dpf2D/D 461 468 GENOTYPE\n",
      "Dpf2D/D 550 557 GENOTYPE\n",
      "Dpf2D/D 688 695 GENOTYPE\n",
      "Dpf2D/D 1548 1555 GENOTYPE\n",
      "Dpf2f/f 353 360 GENOTYPE\n",
      "Dpf2f/f 449 456 GENOTYPE\n",
      "Dpf2f/f 538 545 GENOTYPE\n",
      "Dpf2f/f 676 683 GENOTYPE\n",
      "Dpf2f/f 849 856 GENOTYPE\n",
      "Dpf2f/f 1928 1935 GENOTYPE\n",
      "Dpf2D/D 365 372 GENOTYPE\n",
      "Dpf2D/D 461 468 GENOTYPE\n",
      "Dpf2D/D 550 557 GENOTYPE\n",
      "Dpf2D/D 688 695 GENOTYPE\n",
      "Dpf2D/D 861 868 GENOTYPE\n",
      "Dpf2D/D 1940 1947 GENOTYPE\n",
      "NrCAM KO 99 107 GENOTYPE\n",
      "NrCAM KO 176 184 GENOTYPE\n",
      "Slc2a3-fl/fl Cd4-cre 93 113 GENOTYPE\n",
      "Slc2a3-fl/fl Cd4-cre 95 115 GENOTYPE\n",
      "c-Jun-/- 685 693 GENOTYPE\n",
      "wild type 741 750 GENOTYPE\n",
      "Bhlhe40-/- 504 514 GENOTYPE\n",
      "Bhlhe40-/- 683 693 GENOTYPE\n",
      "Bhlhe40-/- 936 946 GENOTYPE\n",
      "Bhlhe40-/- 988 998 GENOTYPE\n",
      "Bhlhe40-/- 1048 1058 GENOTYPE\n",
      "Bhlhe40-/- 1408 1418 GENOTYPE\n",
      "Bhlhe40-/- 1599 1609 GENOTYPE\n",
      "Bhlhe40-/- 1869 1879 GENOTYPE\n",
      "WT 1094 1096 GENOTYPE\n",
      "WT 1173 1175 GENOTYPE\n",
      "Bgn KO 1101 1107 GENOTYPE\n",
      "Bgn KO 1179 1185 GENOTYPE\n",
      "Aldh1l1-L10-GFP 150 165 GENOTYPE\n",
      "braf;pten melanomas with ezh2 genetic inactivation (TBPE) 218 275 GENOTYPE\n",
      "braf;pten melanomas (TBP) 185 210 GENOTYPE\n",
      "impaired GR (GRdim/dim) 63 86 GENOTYPE\n",
      "Parp1-/- 31 39 GENOTYPE\n",
      "arcA overexpression 30 49 GENOTYPE\n",
      "None 1299 1301 GENOTYPE\n",
      "Skipping entity\n",
      "Dph1 KO 464 471 GENOTYPE\n",
      "Eu-TCL1-Rab27DKO 197 213 GENOTYPE\n",
      "Eu-TCL1-Rab27DKO 288 304 GENOTYPE\n",
      "DkkMo 161 166 GENOTYPE\n",
      "Fmo3+/+ 86 93 GENOTYPE\n",
      "Fmo3+/+ 766 773 GENOTYPE\n",
      "wild type 429 438 GENOTYPE\n",
      "wild type 557 566 GENOTYPE\n",
      "wild type 2131 2140 GENOTYPE\n",
      "control 82 89 GENOTYPE\n",
      "Tet TKO 43 50 GENOTYPE\n",
      "Control 1552 1559 GENOTYPE\n",
      "G9a cKO 280 287 GENOTYPE\n",
      "G9a cKO 905 912 GENOTYPE\n",
      "G9a cKO 1561 1568 GENOTYPE\n",
      "G9a-Glp cDKO 298 310 GENOTYPE\n",
      "G9a-Glp cDKO 660 672 GENOTYPE\n",
      "G9a-Glp cDKO 775 787 GENOTYPE\n",
      "G9a-Glp cDKO 887 899 GENOTYPE\n",
      "G9a-Glp cDKO 1573 1585 GENOTYPE\n",
      "Gabrg2tm1Wul/J +/+ 61 79 GENOTYPE\n",
      "Gabrg2tm1Wul/J +/+ 334 352 GENOTYPE\n",
      "Gabrg2tm1Wul/J +/+ 431 449 GENOTYPE\n",
      "Gabrg2tm1Wul/J F77I/F77I 83 107 GENOTYPE\n",
      "Gabrg2tm1Wul/J F77I/F77I 379 403 GENOTYPE\n",
      "Gabrg2tm1Wul/J F77I/F77I 477 501 GENOTYPE\n",
      "WT 183 185 GENOTYPE\n",
      "liver-specific XBP1 deletion 191 219 GENOTYPE\n",
      "None 475 482 GENOTYPE\n",
      "Skipping entity\n",
      "wild type 1532 1541 GENOTYPE\n",
      "Lgr5-DTR-eGFP 1506 1519 GENOTYPE\n",
      "Arid1acKO 296 305 GENOTYPE\n",
      "Arid1acKO 844 853 GENOTYPE\n",
      "Arid1acKO 1717 1726 GENOTYPE\n",
      "Arid1afl/fl 310 321 GENOTYPE\n",
      "Arid1afl/fl 828 839 GENOTYPE\n",
      "Arid1afl/fl 1731 1742 GENOTYPE\n",
      "Pten+/- 416 423 GENOTYPE\n",
      "Pten+/- 655 662 GENOTYPE\n",
      "None 709 716 GENOTYPE\n",
      "Skipping entity\n",
      "Pten+/- 1563 1570 GENOTYPE\n",
      "Pten+/- 2374 2381 GENOTYPE\n",
      "Pten+/- 2431 2438 GENOTYPE\n",
      "Pten+/- 2569 2576 GENOTYPE\n",
      "Pten+/- 2707 2714 GENOTYPE\n",
      "Pten+/- 2845 2852 GENOTYPE\n",
      "Pten+/- 2982 2989 GENOTYPE\n",
      "Pten+/- 3113 3120 GENOTYPE\n",
      "Pten+/- 3258 3265 GENOTYPE\n",
      "Nes-KO 686 692 GENOTYPE\n",
      "Nes-KO 748 754 GENOTYPE\n",
      "Nes-KO 3396 3402 GENOTYPE\n",
      "Nes-KO 3420 3426 GENOTYPE\n",
      "Nes-KO 3534 3540 GENOTYPE\n",
      "Nes-KO 3558 3564 GENOTYPE\n",
      "Nes-KO 3679 3685 GENOTYPE\n",
      "Nes-KO 3703 3709 GENOTYPE\n",
      "WT 1686 1688 GENOTYPE\n",
      "Pggt1bfl/flLys-Cre 1774 1792 GENOTYPE\n",
      "wild type 1608 1617 GENOTYPE\n",
      "Trem2-/- 1471 1479 GENOTYPE\n",
      "ITGA5 fl/fl 907 918 GENOTYPE\n",
      "ITGA5 fl/fl 1019 1030 GENOTYPE\n",
      "ITGA5 fl/fl 1262 1273 GENOTYPE\n",
      "WT 362 364 GENOTYPE\n",
      "wild-type 1363 1372 GENOTYPE\n",
      "Ngn3-Cre; Ctnna1 flox/flox 986 1012 GENOTYPE\n",
      "wild type 967 976 GENOTYPE\n",
      "wild-type 377 386 GENOTYPE\n",
      "Rtcb knock-out 1321 1335 GENOTYPE\n",
      "WT 1314 1316 GENOTYPE\n",
      "Srsf10-cKO 1474 1484 GENOTYPE\n",
      "Srsf10-cKO 1557 1567 GENOTYPE\n",
      "Srsf10-cKO 1668 1678 GENOTYPE\n",
      "Srsf10-cKO 1777 1787 GENOTYPE\n",
      "Srsf10-cKO 1849 1859 GENOTYPE\n",
      "Control 1462 1469 GENOTYPE\n",
      "Control 1545 1552 GENOTYPE\n",
      "Control 1656 1663 GENOTYPE\n",
      "Control 1765 1772 GENOTYPE\n",
      "Control 1837 1844 GENOTYPE\n",
      "Rad21-/- 1163 1171 GENOTYPE\n",
      "non-transgenic 774 788 GENOTYPE\n",
      "3xTg 145 149 GENOTYPE\n",
      "3xTg 189 193 GENOTYPE\n",
      "wild-type 1426 1435 GENOTYPE\n",
      "wild-type 1566 1575 GENOTYPE\n",
      "wild-type 2027 2036 GENOTYPE\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 77%|█████████████████████████████         | 1381/1804 [00:04<00:01, 349.59it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Osterix-Cre:Tgfbr2f/f 86 107 GENOTYPE\n",
      "Terc KO 46 53 GENOTYPE\n",
      "Runx3 R122C;Iqgap3-2A-tdTomato 2014 2044 GENOTYPE\n",
      "Runx3 WT;Iqgap3-2A-tdTomato 1982 2009 GENOTYPE\n",
      "CCR2-DTR 300 308 GENOTYPE\n",
      "CCR2-DTR 1966 1974 GENOTYPE\n",
      "WT 273 275 GENOTYPE\n",
      "WT 1939 1941 GENOTYPE\n",
      "Bap1f/f 1508 1515 GENOTYPE\n",
      "Bap1f/f 1547 1554 GENOTYPE\n",
      "ApoE-/- 265 272 GENOTYPE\n",
      "ApoE-/- 466 473 GENOTYPE\n",
      "Atf7ip+/fl 65 75 GENOTYPE\n",
      "Atf7ip+/fl 411 421 GENOTYPE\n",
      "Atf7ip+/fl 454 464 GENOTYPE\n",
      "Atf7ip+/fl 739 749 GENOTYPE\n",
      "Atf7ip+/fl 823 833 GENOTYPE\n",
      "Atf7ipfl/fl 80 91 GENOTYPE\n",
      "Atf7ipfl/fl 426 437 GENOTYPE\n",
      "Atf7ipfl/fl 469 480 GENOTYPE\n",
      "Atf7ipfl/fl 696 707 GENOTYPE\n",
      "Atf7ipfl/fl 838 849 GENOTYPE\n",
      "WT 318 320 GENOTYPE\n",
      "WT 459 461 GENOTYPE\n",
      "WT 1460 1462 GENOTYPE\n",
      "STAT3-cKO 175 184 GENOTYPE\n",
      "STAT3-cKO 560 569 GENOTYPE\n",
      "STAT3-cKO 695 704 GENOTYPE\n",
      "STAT3-WT 218 226 GENOTYPE\n",
      "STAT3-WT 547 555 GENOTYPE\n",
      "STAT3-WT 682 690 GENOTYPE\n",
      "Ctr 243 246 GENOTYPE\n",
      "WT 159 161 GENOTYPE\n",
      "WT 332 334 GENOTYPE\n",
      "Cdx2Apcf/w 1145 1155 GENOTYPE\n",
      "Mettl3 KO 778 787 GENOTYPE\n",
      "WT 756 758 GENOTYPE\n",
      "wild type 1651 1660 GENOTYPE\n",
      "Irg1-/- 1328 1335 GENOTYPE\n",
      "S-M- 604 608 GENOTYPE\n",
      "S+M- 556 560 GENOTYPE\n",
      "wild type 1208 1217 GENOTYPE\n",
      "wild type 80 89 GENOTYPE\n",
      "wild type 443 452 GENOTYPE\n",
      "wild type 563 572 GENOTYPE\n",
      "GPX4 knockdown 467 481 GENOTYPE\n",
      "GPX4 knockdown 765 779 GENOTYPE\n",
      "Vdrf/f 618 624 GENOTYPE\n",
      "Vdr-cKO 312 319 GENOTYPE\n",
      "Vdr-cKO 606 613 GENOTYPE\n",
      "HDAC6 global knockout 1187 1208 GENOTYPE\n",
      "adiponectin-Cre controls (AdCre) 1495 1527 GENOTYPE\n",
      "Pparafl/fl 1232 1242 GENOTYPE\n",
      "control 1029 1036 GENOTYPE\n",
      "wildtype 211 219 GENOTYPE\n",
      "None 618 625 GENOTYPE\n",
      "Skipping entity\n",
      "control 728 735 GENOTYPE\n",
      "control 1161 1168 GENOTYPE\n",
      "control 1728 1735 GENOTYPE\n",
      "WT 1336 1338 GENOTYPE\n",
      "Tie2-Cre::ROSA-Evi1 63 82 GENOTYPE\n",
      "Tie2-Cre::ROSA-Evi1 175 194 GENOTYPE\n",
      "Tie2-Cre::ROSA-Evi1 239 258 GENOTYPE\n",
      "Tie2-Cre::ROSA-Evi1 357 376 GENOTYPE\n",
      "-Evi1 73 82 GENOTYPE\n",
      "-Evi1 185 194 GENOTYPE\n",
      "-Evi1 249 258 GENOTYPE\n",
      "-Evi1 367 376 GENOTYPE\n",
      "ROSA-Evi1 381 390 GENOTYPE\n",
      "Mast4 KO 265 273 GENOTYPE\n",
      "Mast4 KO 400 408 GENOTYPE\n",
      "Mettl3fl/fl 236 247 GENOTYPE\n",
      "Mettl3fl/fl 257 268 GENOTYPE\n",
      "VillinCre-ERT2; ApcFL/FL; KrasLSL-G12D/+; Tp53KO/KO; R26R-Confetti; Lgr5DTR-eGFP 330 410 GENOTYPE\n",
      "monocyte-depleted (CCR2-DTR) 1865 1893 GENOTYPE\n",
      "wild type 55 64 GENOTYPE\n",
      "wild type 75 84 GENOTYPE\n",
      "wild type 1846 1855 GENOTYPE\n",
      "Cxcr4WHIM/+ 338 349 GENOTYPE\n",
      "Cxcr4+/+ 365 373 GENOTYPE\n",
      "Cxcr4WHIM/+ 305 316 GENOTYPE\n",
      "Cxcr4+/+ 332 340 GENOTYPE\n",
      "None 618 625 GENOTYPE\n",
      "Skipping entity\n",
      "control 728 735 GENOTYPE\n",
      "control 1161 1168 GENOTYPE\n",
      "control 1709 1716 GENOTYPE\n",
      "WT 749 751 GENOTYPE\n",
      "control 240 247 GENOTYPE\n",
      "control 537 544 GENOTYPE\n",
      "miR-322/-503 overexpression 58 85 GENOTYPE\n",
      "WT 279 281 GENOTYPE\n",
      "Rpl3l-/- 286 294 GENOTYPE\n",
      "WT 275 277 GENOTYPE\n",
      "Rpl3l-/- 282 290 GENOTYPE\n",
      "WT 380 382 GENOTYPE\n",
      "AML 43 46 GENOTYPE\n",
      "AML 334 337 GENOTYPE\n",
      "AML 513 516 GENOTYPE\n",
      "AML 801 804 GENOTYPE\n",
      "AML 1160 1163 GENOTYPE\n",
      "AML 1222 1225 GENOTYPE\n",
      "AML 1289 1292 GENOTYPE\n",
      "Trp53fl/fl 208 218 GENOTYPE\n",
      "Trp53fl/fl 285 295 GENOTYPE\n",
      "Trp53fl/fl 397 407 GENOTYPE\n",
      "Trp53fl/fl 448 458 GENOTYPE\n",
      "Trp53fl/fl 525 535 GENOTYPE\n",
      "wild type 1220 1229 GENOTYPE\n",
      "wild type 1335 1344 GENOTYPE\n",
      "Bud31-vKO 1239 1248 GENOTYPE\n",
      "Hi-Myc; Otud6a-/y 563 580 GENOTYPE\n",
      "Hi-Myc; Otud6a-/y 698 715 GENOTYPE\n",
      "Hi-Myc; Otud6a+/y 541 558 GENOTYPE\n",
      "Hi-Myc; Otud6a+/y 676 693 GENOTYPE\n",
      "Otud6a-/y 530 539 GENOTYPE\n",
      "Otud6a-/y 571 580 GENOTYPE\n",
      "Otud6a-/y 633 642 GENOTYPE\n",
      "Otud6a-/y 706 715 GENOTYPE\n",
      "Otud6a+/y 519 528 GENOTYPE\n",
      "Otud6a+/y 549 558 GENOTYPE\n",
      "Otud6a+/y 619 628 GENOTYPE\n",
      "Otud6a+/y 684 693 GENOTYPE\n",
      "PRNP(ZH3/ZH3) 406 419 GENOTYPE\n",
      "Tbet-KO 230 237 GENOTYPE\n",
      "Tbet-KO 615 622 GENOTYPE\n",
      "Tbet-KO 663 670 GENOTYPE\n",
      "WT 218 220 GENOTYPE\n",
      "WT 304 306 GENOTYPE\n",
      "WT 340 342 GENOTYPE\n",
      "WT 379 381 GENOTYPE\n",
      "WT 415 417 GENOTYPE\n",
      "WT 472 474 GENOTYPE\n",
      "WT 482 484 GENOTYPE\n",
      "WT 579 581 GENOTYPE\n",
      "ST2-KO 158 164 GENOTYPE\n",
      "ST2-KO 518 524 GENOTYPE\n",
      "ST2-KO 565 571 GENOTYPE\n",
      "WT-TSCs 1095 1102 GENOTYPE\n",
      "WT-ESCs 1086 1093 GENOTYPE\n",
      "Rif1-KO iTSCs 1121 1134 GENOTYPE\n",
      "Arid1b 1459 1472 GENOTYPE\n",
      "GRko/ko 1259 1266 GENOTYPE\n",
      "GRD/D 240 245 GENOTYPE\n",
      "GRD/D 313 318 GENOTYPE\n",
      "GRD/D 1204 1209 GENOTYPE\n",
      "GRL/L 548 553 GENOTYPE\n",
      "GRL/L 1219 1224 GENOTYPE\n",
      "GRwt/wt 1180 1187 GENOTYPE\n",
      "GRD+L/D+L 407 416 GENOTYPE\n",
      "GRD+L/D+L 1237 1246 GENOTYPE\n",
      "Bend2 WT 146 154 GENOTYPE\n",
      "Bend2 WT 399 407 GENOTYPE\n",
      "KO 83 85 GENOTYPE\n",
      "KO 143 145 GENOTYPE\n",
      "Egfp.l10PepckSlc40a1Pepckdelta/delta 558 594 GENOTYPE\n",
      "Egfp.l10PepckSlc40a1Pepckdelta/delta 754 790 GENOTYPE\n",
      "Egfp.l10PepckSlc40a1Pepckdelta/delta 1225 1261 GENOTYPE\n",
      "Egfp.l10PepckSlc40a1Pepckdelta/delta 2278 2314 GENOTYPE\n",
      "Egfp.l10Pepck 540 553 GENOTYPE\n",
      "None 558 571 GENOTYPE\n",
      "Skipping entity\n",
      "Egfp.l10Pepck 736 749 GENOTYPE\n",
      "None 754 767 GENOTYPE\n",
      "Skipping entity\n",
      "Egfp.l10Pepck 1207 1220 GENOTYPE\n",
      "None 1225 1238 GENOTYPE\n",
      "Skipping entity\n",
      "Egfp.l10Pepck 2260 2273 GENOTYPE\n",
      "None 2278 2291 GENOTYPE\n",
      "Skipping entity\n",
      "Smarce1-MD 168 178 GENOTYPE\n",
      "Smarce1-MD 191 201 GENOTYPE\n",
      "Smarce1-MD 425 435 GENOTYPE\n",
      "SP-C C121Gneo 266 279 GENOTYPE\n",
      "SP-C 231 241 GENOTYPE\n",
      "SP-C 434 444 GENOTYPE\n",
      "SP-C C185Gneo 231 244 GENOTYPE\n",
      "SP-C C185Gneo 434 447 GENOTYPE\n",
      "SP-C WT 409 416 GENOTYPE\n",
      "Smad7 fl/fl 2900 2911 GENOTYPE\n",
      "Trim71-R595H/R595H 813 831 GENOTYPE\n",
      "GRwt/wt 1180 1187 GENOTYPE\n",
      "GRD+L/D+L 407 416 GENOTYPE\n",
      "GRD+L/D+L 1206 1215 GENOTYPE\n",
      "GRL/L 548 553 GENOTYPE\n",
      "GRL/L 1196 1201 GENOTYPE\n",
      "GRD/D 240 245 GENOTYPE\n",
      "GRD/D 313 318 GENOTYPE\n",
      "GRD/D 1189 1194 GENOTYPE\n",
      "Carm1 skm+/+ 183 195 GENOTYPE\n",
      "Carm1 skm-/- 148 160 GENOTYPE\n",
      "Tbx5(Reint);Prrx1(RE) 50 71 GENOTYPE\n",
      "Tbx5(REint) 23 34 GENOTYPE\n",
      "Tbx5(REdown) 36 48 GENOTYPE\n",
      "control 76 83 GENOTYPE\n",
      "Tln2-null 74 83 GENOTYPE\n",
      "Tln2-null 88 97 GENOTYPE\n",
      "Tln2-null 213 222 GENOTYPE\n",
      "Tln2-null 421 430 GENOTYPE\n",
      "Tln2-null 435 444 GENOTYPE\n",
      "Tln2-null; Tln1-specific CF knockout 88 124 GENOTYPE\n",
      "Tln2-null; Tln1-specific CF knockout 213 249 GENOTYPE\n",
      "Tln2-null; Tln1-specific CF knockout 435 471 GENOTYPE\n",
      "Tmem216-/- 52 62 GENOTYPE\n",
      "Tmem216-/- 403 413 GENOTYPE\n",
      "wild type 1315 1324 GENOTYPE\n",
      "PKN2 KO 981 988 GENOTYPE\n",
      "wild-type 1648 1657 GENOTYPE\n",
      "S82A 48 52 GENOTYPE\n",
      "WT 1215 1217 GENOTYPE\n",
      "Traf6 KO 1579 1587 GENOTYPE\n",
      "Tet2 KO 1570 1577 GENOTYPE\n",
      "wildtype 345 353 GENOTYPE\n",
      "Idua+/- 375 382 GENOTYPE\n",
      "ApoE-/- 279 286 GENOTYPE\n",
      "ApoE-/- 537 544 GENOTYPE\n",
      "ApoE-/- 717 724 GENOTYPE\n",
      "Zfp335 deficient 678 694 GENOTYPE\n",
      "MAGL Mye-/- 75 86 GENOTYPE\n",
      "MAGL Mye-/- 199 210 GENOTYPE\n",
      "SPT 243 246 GENOTYPE\n",
      "SPT 479 482 GENOTYPE\n",
      "SPT 658 661 GENOTYPE\n",
      "WT 272 274 GENOTYPE\n",
      "WT 467 469 GENOTYPE\n",
      "WT 651 653 GENOTYPE\n",
      "Zfp335 deficient 670 686 GENOTYPE\n",
      "WT 663 665 GENOTYPE\n",
      "Smad3SmKO 442 451 GENOTYPE\n",
      "Smad3+/+ 285 293 GENOTYPE\n",
      "Smad3+/+ 348 356 GENOTYPE\n",
      "PKN2 KO 981 988 GENOTYPE\n",
      "STAT3fl/fl CD11c-Cre+ 1662 1683 GENOTYPE\n",
      "STAT3fl/fl CD11c-Cre- 1636 1657 GENOTYPE\n",
      "edKO 137 141 GENOTYPE\n",
      "edKO 317 321 GENOTYPE\n",
      "WT/mTmG/ Myh11ERT2 182 200 GENOTYPE\n",
      "WT/mTmG/ Myh11ERT2 481 499 GENOTYPE\n",
      "WT/mTmG/ Myh11ERT2 1711 1729 GENOTYPE\n",
      "miR-21fl/flmTmG/Myh11ERT2 205 230 GENOTYPE\n",
      "miR-21fl/flmTmG/Myh11ERT2 504 529 GENOTYPE\n",
      "miR-21fl/flmTmG/Myh11ERT2 1757 1782 GENOTYPE\n",
      "miR-21fl/flmTmG/Myh11ERT2 2554 2579 GENOTYPE\n",
      "wildtype 1684 1692 GENOTYPE\n",
      "None 326 333 GENOTYPE\n",
      "Skipping entity\n",
      "control 1436 1443 GENOTYPE\n",
      "control 1615 1622 GENOTYPE\n",
      "Kat7 knockout 1211 1224 GENOTYPE\n",
      "Adar-/-; Adarb1-/-; Gria2R/R 1429 1457 GENOTYPE\n",
      "Adgre5-/- and CD11c-cre x Gna13fl/ 82 117 GENOTYPE\n",
      "WT 692 694 GENOTYPE\n",
      "IRF4-deficient 59 73 GENOTYPE\n",
      "IRF4-deficient 166 180 GENOTYPE\n",
      "IRF4-deficient 468 482 GENOTYPE\n",
      "WT 740 742 GENOTYPE\n",
      "Arhgef1-/- 78 88 GENOTYPE\n",
      "Arhgef1-/- 177 187 GENOTYPE\n",
      "Arhgef1-/- 551 561 GENOTYPE\n",
      "WT 544 546 GENOTYPE\n",
      "Wtap KO 1131 1138 GENOTYPE\n",
      "Ptipf/f;Sp7-cre 270 285 GENOTYPE\n",
      "Ptipf/f;Sp7-cre 745 760 GENOTYPE\n",
      "Ptipf/ 270 278 GENOTYPE\n",
      "Ptipf/ 745 753 GENOTYPE\n",
      "IL-15Tg-Xbp1s cKO 1006 1023 GENOTYPE\n",
      "Skiv2lfl/fl-Cd79a-Cre 138 159 GENOTYPE\n",
      "Skiv2lfl/fl 138 149 GENOTYPE\n",
      "Skiv2lfl/fl 171 182 GENOTYPE\n",
      "MafF-/-::MafG-/-::MafK-/- 304 329 GENOTYPE\n",
      "WT 423 425 GENOTYPE\n",
      "knockout 104 112 GENOTYPE\n",
      "ChemR23 290 306 GENOTYPE\n",
      "ShhCre Cenpj-/lox 241 258 GENOTYPE\n",
      "ShhCre Cenpj+/lox 189 206 GENOTYPE\n",
      "ShhCre Ift88-/lox 293 310 GENOTYPE\n",
      "Nfil3-/- 1740 1748 GENOTYPE\n",
      "Nfil3GFP/GFP 1646 1658 GENOTYPE\n",
      "Nfil3GFP/GFP 1753 1765 GENOTYPE\n",
      "Tia1 Tial1 double conditional knock out 1078 1117 GENOTYPE\n",
      "WT 593 595 GENOTYPE\n",
      "hnRNPF knockout (HFKO) 601 623 GENOTYPE\n",
      "PU.1 fl/fl 514 524 GENOTYPE\n",
      "PU.1 fl/fl 539 549 GENOTYPE\n",
      "wild type 84 93 GENOTYPE\n",
      "wild type 314 323 GENOTYPE\n",
      "wild type 654 663 GENOTYPE\n",
      "WT 175 177 GENOTYPE\n",
      "WT 1280 1282 GENOTYPE\n",
      "Ythdf2 cKO 1287 1297 GENOTYPE\n",
      "NFAM1-/- 623 631 GENOTYPE\n",
      "NFAM1-/- 696 704 GENOTYPE\n",
      "NFAM1-/- 833 841 GENOTYPE\n",
      "NFAM1-/- 1839 1847 GENOTYPE\n",
      "WT 1258 1260 GENOTYPE\n",
      "ACC1-deficient 1265 1279 GENOTYPE\n",
      "ACC1-deficient 1580 1594 GENOTYPE\n",
      "ACC1-deficient 1835 1849 GENOTYPE\n",
      "Ppp1r13lflox/flox;Cre+ERT 657 682 GENOTYPE\n",
      "LERKO 1127 1132 GENOTYPE\n",
      "ERa floxed 1112 1122 GENOTYPE\n",
      "db/db 195 200 GENOTYPE\n",
      "db/db 488 493 GENOTYPE\n",
      "db/db 659 664 GENOTYPE\n",
      "IXa 45 48 GENOTYPE\n",
      "control 208 215 GENOTYPE\n",
      "None 1099 1106 GENOTYPE\n",
      "Skipping entity\n",
      "Lmo2-transgenic 514 529 GENOTYPE\n",
      "db/db 441 446 GENOTYPE\n",
      "db/db 749 754 GENOTYPE\n",
      "db/db 1620 1625 GENOTYPE\n",
      "WT 150 152 GENOTYPE\n",
      "WT 301 303 GENOTYPE\n",
      "WT 337 339 GENOTYPE\n",
      "Tnfr2-/- 156 164 GENOTYPE\n",
      "Tnfr2-/- 316 324 GENOTYPE\n",
      "WT 57 59 GENOTYPE\n",
      "WT 211 213 GENOTYPE\n",
      "Tnfr2-/- 64 72 GENOTYPE\n",
      "Tnfr2-/- 218 226 GENOTYPE\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 83%|███████████████████████████████▌      | 1496/1804 [00:05<00:00, 352.41it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "HrasG12V/+ 1338 1348 GENOTYPE\n",
      "Hras+/+ 1416 1423 GENOTYPE\n",
      "Toxf/fPdcd1Cre 237 251 GENOTYPE\n",
      "Toxf/fPdcd1Cre 342 356 GENOTYPE\n",
      "Toxf/fPdcd1Cre 795 809 GENOTYPE\n",
      "Toxf/fPdcd1Cre 970 984 GENOTYPE\n",
      "Toxf/fPdcd1Cre 1560 1574 GENOTYPE\n",
      "Toxf/f 226 232 GENOTYPE\n",
      "Toxf/ 237 243 GENOTYPE\n",
      "Toxf/f 331 337 GENOTYPE\n",
      "Toxf/ 342 348 GENOTYPE\n",
      "Toxf/f 784 790 GENOTYPE\n",
      "Toxf/ 795 801 GENOTYPE\n",
      "Toxf/f 959 965 GENOTYPE\n",
      "Toxf/ 970 976 GENOTYPE\n",
      "Toxf/f 1549 1555 GENOTYPE\n",
      "Toxf/ 1560 1566 GENOTYPE\n",
      "WT 845 847 GENOTYPE\n",
      "Shp-2fl/ 97 107 GENOTYPE\n",
      "Shp-2fl/ 133 143 GENOTYPE\n",
      "Shp-2fl/ 213 223 GENOTYPE\n",
      "Shp-2fl/ 307 317 GENOTYPE\n",
      "Shp-2fl/fl 851 861 GENOTYPE\n",
      "Shp-2fl/ 866 876 GENOTYPE\n",
      "Shp-2fl/flLysMCre 133 150 GENOTYPE\n",
      "Shp-2fl/flLysMCre 307 324 GENOTYPE\n",
      "Shp-2fl/flLysMCre 866 883 GENOTYPE\n",
      "CDH5-CreER(T): ERG fl/fl 473 497 GENOTYPE\n",
      "ERG fl/fl 459 468 GENOTYPE\n",
      "ERG fl/fl 488 497 GENOTYPE\n",
      "control 52 59 GENOTYPE\n",
      "control 346 353 GENOTYPE\n",
      "Angptl4-/- 644 654 GENOTYPE\n",
      "Angptl4-/- 745 755 GENOTYPE\n",
      "Angptl4-/- 1225 1235 GENOTYPE\n",
      "WT 423 425 GENOTYPE\n",
      "Stag2f/y 1286 1294 GENOTYPE\n",
      "Stag2f/y 1299 1307 GENOTYPE\n",
      "Stag2f/y; Cre 1299 1312 GENOTYPE\n",
      "Stag2f/y 1242 1250 GENOTYPE\n",
      "Stag2f/y 1255 1263 GENOTYPE\n",
      "Stag2f/y; Cre 1255 1268 GENOTYPE\n",
      "Apolipoprotein E ko 1231 1250 GENOTYPE\n",
      "WT 425 427 GENOTYPE\n",
      "WT 2076 2078 GENOTYPE\n",
      "WT 2377 2379 GENOTYPE\n",
      "Furin KO 387 395 GENOTYPE\n",
      "Furin KO 2063 2071 GENOTYPE\n",
      "Furin KO 2364 2372 GENOTYPE\n",
      "WT 425 427 GENOTYPE\n",
      "WT 1615 1617 GENOTYPE\n",
      "WT 1700 1702 GENOTYPE\n",
      "Furin KO 387 395 GENOTYPE\n",
      "Furin KO 1677 1685 GENOTYPE\n",
      "hepatic Atg5 KO 88 103 GENOTYPE\n",
      "C57BL/6 1416 1423 GENOTYPE\n",
      "floxed VMP1 348 359 GENOTYPE\n",
      "floxed VMP1 434 445 GENOTYPE\n",
      "control 1860 1867 GENOTYPE\n",
      "hepatocyte-specific Ascl1 KO 1890 1918 GENOTYPE\n",
      "Mettl5-KO 1028 1037 GENOTYPE\n",
      "Ctrl 1019 1023 GENOTYPE\n",
      "Ctrl 1079 1083 GENOTYPE\n",
      "Ctrl 1128 1132 GENOTYPE\n",
      "WT 1149 1151 GENOTYPE\n",
      "transgenic Col1a1*2,3-GFP 465 490 GENOTYPE\n",
      "Smad3-deficient (Smad3-/-) 1881 1907 GENOTYPE\n",
      "RBM24-/- 96 104 GENOTYPE\n",
      "RBM24-/- 965 973 GENOTYPE\n",
      "Knock out 141 150 GENOTYPE\n",
      "KLF2-KO 172 179 GENOTYPE\n",
      "wild type 201 210 GENOTYPE\n",
      "wild type 331 340 GENOTYPE\n",
      "KO 1775 1777 GENOTYPE\n",
      "KO 1922 1924 GENOTYPE\n",
      "KO 1945 1947 GENOTYPE\n",
      "KO 1957 1959 GENOTYPE\n",
      "KO 2038 2040 GENOTYPE\n",
      "KO 2050 2052 GENOTYPE\n",
      "WT 1894 1896 GENOTYPE\n",
      "Bhlhe40-/- 1344 1354 GENOTYPE\n",
      "Bhlhe40-/- 1389 1399 GENOTYPE\n",
      "Bhlhe40-/- 1406 1416 GENOTYPE\n",
      "Bhlhe40-/- 1451 1461 GENOTYPE\n",
      "Bhlhe40-/- 1468 1478 GENOTYPE\n",
      "Bhlhe40-/- 1513 1523 GENOTYPE\n",
      "Bhlhe40-/- 1530 1540 GENOTYPE\n",
      "Bhlhe40-/- 1575 1585 GENOTYPE\n",
      "Bhlhe40-/- 1592 1602 GENOTYPE\n",
      "Angptl2fl/fl 112 124 GENOTYPE\n",
      "/fl 137 149 GENOTYPE\n",
      "Angptl2fl/fl 266 278 GENOTYPE\n",
      "/fl 291 303 GENOTYPE\n",
      "Cdh5-Cre;Angptl2fl/fl 128 149 GENOTYPE\n",
      "Cdh5-Cre;Angptl2fl/fl 282 303 GENOTYPE\n",
      "WT 239 241 GENOTYPE\n",
      "WT 345 347 GENOTYPE\n",
      "CD19-/- 126 133 GENOTYPE\n",
      "CD19-/- 246 253 GENOTYPE\n",
      "CD19-/- 362 369 GENOTYPE\n",
      "WT 240 242 GENOTYPE\n",
      "WT 346 348 GENOTYPE\n",
      "CD19-/- 247 254 GENOTYPE\n",
      "CD19-/- 353 360 GENOTYPE\n",
      "E2a-/- 70 76 GENOTYPE\n",
      "None 83 89 GENOTYPE\n",
      "Skipping entity\n",
      "Atad3 nKO 355 364 GENOTYPE\n",
      "Atad3 nKO 1164 1173 GENOTYPE\n",
      "CDK8-/- 1128 1135 GENOTYPE\n",
      "None 1151 1158 GENOTYPE\n",
      "Skipping entity\n",
      "CDK19-/- 1137 1145 GENOTYPE\n",
      "None 1158 1166 GENOTYPE\n",
      "Skipping entity\n",
      "WT 1351 1353 GENOTYPE\n",
      "Asxl2KO 1358 1365 GENOTYPE\n",
      "miR-29a KO 415 425 GENOTYPE\n",
      "Oasl1+/+Apoe-/- 192 207 GENOTYPE\n",
      "Oasl1+/+Apoe-/- 393 408 GENOTYPE\n",
      "Oasl1-/-Apoe-/- 212 227 GENOTYPE\n",
      "Oasl1-/-Apoe-/- 412 427 GENOTYPE\n",
      "MDR 1221 1224 GENOTYPE\n",
      "Slc-HKO 564 571 GENOTYPE\n",
      "Slc-HKO 912 919 GENOTYPE\n",
      "wild 445 454 GENOTYPE\n",
      "ApoE-/-/SAHHWT 330 344 GENOTYPE\n",
      "ApoE-/-/SAHH+/- 346 361 GENOTYPE\n",
      "ApoE-/-/SAHH+/- 371 386 GENOTYPE\n",
      "ApoE-/-/SAHH+/- +Betaine 371 395 GENOTYPE\n",
      "Sftpc-creER;Rosa26-mTmG 1914 1937 GENOTYPE\n",
      "Sftpc-creER;Rosa26-mTmG 2043 2066 GENOTYPE\n",
      "HMCES-KO 1401 1409 GENOTYPE\n",
      "Ifnar1-fl/fl 1068 1080 GENOTYPE\n",
      "cKO (Ncr1Cre-Rorafl/fl) 399 422 GENOTYPE\n",
      "WT 357 359 GENOTYPE\n",
      "rTg4510 66 73 GENOTYPE\n",
      "rTg4510 186 193 GENOTYPE\n",
      "rTg4510 397 404 GENOTYPE\n",
      "non-transgenic 409 423 GENOTYPE\n",
      "WT 442 444 GENOTYPE\n",
      "WT 729 731 GENOTYPE\n",
      "WT 424 426 GENOTYPE\n",
      "WT 672 674 GENOTYPE\n",
      "control 304 311 GENOTYPE\n",
      "control littermates 324 343 GENOTYPE\n",
      "K14Cre 205 211 GENOTYPE\n",
      "K14Cre 779 785 GENOTYPE\n",
      "K14Cre 795 801 GENOTYPE\n",
      "Gpr109a-/- 85 95 GENOTYPE\n",
      "Adamtsl2 knockdown 833 851 GENOTYPE\n",
      "control 583 590 GENOTYPE\n",
      "control 821 828 GENOTYPE\n",
      "SFPQ-ABL1 46 55 GENOTYPE\n",
      "SFPQ-ABL1 168 177 GENOTYPE\n",
      "SFPQ-ABL1 531 540 GENOTYPE\n",
      "SFPQ-ABL1 629 638 GENOTYPE\n",
      "SFPQ-ABL1 879 888 GENOTYPE\n",
      "SFPQ- 1190 1199 GENOTYPE\n",
      "SFPQ-ABL1 1241 1250 GENOTYPE\n",
      "SFPQ-ABL1 1454 1463 GENOTYPE\n",
      "SFPQ-ABL1 1478 1487 GENOTYPE\n",
      "SFPQ-ABL1 1842 1851 GENOTYPE\n",
      "BCR-ABL1 211 219 GENOTYPE\n",
      "BCR-ABL1 519 527 GENOTYPE\n",
      "BCR-ABL1 619 627 GENOTYPE\n",
      "BCR-ABL1 869 877 GENOTYPE\n",
      "BCR-ABL1 1214 1222 GENOTYPE\n",
      "BCR-ABL1 1441 1449 GENOTYPE\n",
      "BCR-ABL1 1832 1840 GENOTYPE\n",
      "MSCV 666 670 GENOTYPE\n",
      "MSCV 1871 1875 GENOTYPE\n",
      "wild type 515 524 GENOTYPE\n",
      "Trim11+/- 1264 1273 GENOTYPE\n",
      "Wt 1199 1201 GENOTYPE\n",
      "Wt 1354 1356 GENOTYPE\n",
      "miR-KO 1228 1234 GENOTYPE\n",
      "miR-KO 1361 1367 GENOTYPE\n",
      "non-targeting 1816 1837 GENOTYPE\n",
      "KLF2-KO 194 201 GENOTYPE\n",
      "wild type 1152 1161 GENOTYPE\n",
      "ASIC1a-/- 181 190 GENOTYPE\n",
      "ASIC1a-/- 1042 1051 GENOTYPE\n",
      "ASIC1a-/- 1171 1180 GENOTYPE\n",
      "Fbl+/- 140 146 GENOTYPE\n",
      "WT 317 319 GENOTYPE\n",
      "March1-/- 920 929 GENOTYPE\n",
      "March1-/- 2467 2476 GENOTYPE\n",
      "wild type 901 910 GENOTYPE\n",
      "wild type 2448 2457 GENOTYPE\n",
      "miR-204-/- 399 409 GENOTYPE\n",
      "Xkr8 KO 141 148 GENOTYPE\n",
      "Xkr8 KO 192 199 GENOTYPE\n",
      "wild-type 148 157 GENOTYPE\n",
      "wild-type 1193 1202 GENOTYPE\n",
      "Lrg1 knockout 544 557 GENOTYPE\n",
      "wild type 530 539 GENOTYPE\n",
      "Sox6-FSF-Cre, Th-2A-Flpo, RC-Frepe 179 213 GENOTYPE\n",
      "/f 237 245 GENOTYPE\n",
      "Cdc42f/f 339 347 GENOTYPE\n",
      "/f 358 366 GENOTYPE\n",
      "Cdc42f/f 436 444 GENOTYPE\n",
      "/f 457 465 GENOTYPE\n",
      "WT 181 183 GENOTYPE\n",
      "WT 330 332 GENOTYPE\n",
      "cGAS-/- 188 195 GENOTYPE\n",
      "cGAS-/- 336 343 GENOTYPE\n",
      "NL4-/- 433 439 GENOTYPE\n",
      "NL4-/- 709 715 GENOTYPE\n",
      "NL4-/- 843 849 GENOTYPE\n",
      "None 70 77 GENOTYPE\n",
      "Skipping entity\n",
      "VE-PTPiKO 334 343 GENOTYPE\n",
      "VE-PTPiKO 416 425 GENOTYPE\n",
      "ITK KO 1551 1557 GENOTYPE\n",
      "WT 1615 1617 GENOTYPE\n",
      "Wnt1cre; Rosa26Dlx2/- 711 732 GENOTYPE\n",
      "Wnt1cre; Rosa26Dlx2/- 840 861 GENOTYPE\n",
      "Wnt1cre; Rosa26Dlx2/- 1553 1574 GENOTYPE\n",
      "ICOS-deficient 461 475 GENOTYPE\n",
      "ICOS-deficient 1367 1381 GENOTYPE\n",
      "ICOS-deficient 1667 1681 GENOTYPE\n",
      "ICOS WT 1655 1662 GENOTYPE\n",
      "Lepr+/+, Lyz2-Cre 135 152 GENOTYPE\n",
      "Leprfl/fl, Lyz2-Cre 174 193 GENOTYPE\n",
      "Control 1667 1674 GENOTYPE\n",
      "Control 1702 1709 GENOTYPE\n",
      "circSry KO 259 269 GENOTYPE\n",
      "WT 342 344 GENOTYPE\n",
      "WT 744 746 GENOTYPE\n",
      "Fbln7 WT 830 838 GENOTYPE\n",
      "FNIP1-AKO 585 594 GENOTYPE\n",
      "FNIP1-AKO 898 907 GENOTYPE\n",
      "FNIP1-AKO 999 1008 GENOTYPE\n",
      "FNIP1-AKO 1294 1303 GENOTYPE\n",
      "FNIP1-AKO 1517 1526 GENOTYPE\n",
      "FapKO 983 988 GENOTYPE\n",
      "FapKO 1099 1104 GENOTYPE\n",
      "Extl1-deficient 247 262 GENOTYPE\n",
      "wild-type 233 242 GENOTYPE\n",
      "Extl1-dificient 79 94 GENOTYPE\n",
      "Extl1-dificient 290 305 GENOTYPE\n",
      "KO 1246 1248 GENOTYPE\n",
      "WT 1237 1239 GENOTYPE\n",
      "HET 1241 1244 GENOTYPE\n",
      "wild type 731 740 GENOTYPE\n",
      "WT 113 115 GENOTYPE\n",
      "WT 309 311 GENOTYPE\n",
      "WT 443 445 GENOTYPE\n",
      "WT 896 898 GENOTYPE\n",
      "WT 911 913 GENOTYPE\n",
      "AdipoR1-/- 58 68 GENOTYPE\n",
      "AdipoR1-/- 316 326 GENOTYPE\n",
      "AdipoR1-/- 450 460 GENOTYPE\n",
      "AdipoR1-/- 855 865 GENOTYPE\n",
      "AdipoR1-/- 877 887 GENOTYPE\n",
      "Extl1-dificient 81 96 GENOTYPE\n",
      "Extl1-dificient 292 307 GENOTYPE\n",
      "CCR2-/- 477 484 GENOTYPE\n",
      "CCR2-/- 694 701 GENOTYPE\n",
      "WT 688 690 GENOTYPE\n",
      "WT 1001 1003 GENOTYPE\n",
      "WT 1048 1050 GENOTYPE\n",
      "dKO 1008 1011 GENOTYPE\n",
      "dKO 1055 1058 GENOTYPE\n",
      "db/db mice 389 399 GENOTYPE\n",
      "db/m mice 406 415 GENOTYPE\n",
      "Lats1f/fLats2f/fAdipoqCre 1084 1109 GENOTYPE\n",
      "Lats1f/fLats2f/ 1084 1100 GENOTYPE\n",
      "Lats1f/fLats2f/f 1137 1153 GENOTYPE\n",
      "Treg KO 82 89 GENOTYPE\n",
      "WT 76 78 GENOTYPE\n",
      "WT 136 138 GENOTYPE\n",
      "WT 374 376 GENOTYPE\n",
      "WT 235 237 GENOTYPE\n",
      "WT 288 290 GENOTYPE\n",
      "WT 414 416 GENOTYPE\n",
      "Sema3b-/- 102 111 GENOTYPE\n",
      "Sema3b-/- 266 275 GENOTYPE\n",
      "Sema3b-/- 311 320 GENOTYPE\n",
      "Sema3b-/- 422 431 GENOTYPE\n",
      "Xist knockout 1274 1287 GENOTYPE\n",
      "Slc16a1 flox/flox 894 911 GENOTYPE\n",
      "Slc16a1 flox/flox 947 964 GENOTYPE\n",
      "KO 686 688 GENOTYPE\n",
      "KO 705 707 GENOTYPE\n",
      "KO 1141 1143 GENOTYPE\n",
      "KO 686 688 GENOTYPE\n",
      "KO 705 707 GENOTYPE\n",
      "KO 1127 1129 GENOTYPE\n",
      "Bmal1-/- 224 232 GENOTYPE\n",
      "Bmal1-/- 503 511 GENOTYPE\n",
      "Bmal1-/- 945 953 GENOTYPE\n",
      "wild-type 210 219 GENOTYPE\n",
      "wild-type 530 539 GENOTYPE\n",
      "wild-type 931 940 GENOTYPE\n",
      "MCK-TD 49 55 GENOTYPE\n",
      "MCK-TD 161 167 GENOTYPE\n",
      "Con-TD 38 44 GENOTYPE\n",
      "Con-TD 150 156 GENOTYPE\n",
      "Tet2 knockout 925 938 GENOTYPE\n",
      "Tet2 knockout 985 998 GENOTYPE\n",
      "Tet2 knockout 1163 1176 GENOTYPE\n",
      "Tet2 knockout 1382 1395 GENOTYPE\n",
      "Tet2 knockout 1442 1455 GENOTYPE\n",
      "Tet2 knockout 1620 1633 GENOTYPE\n",
      "Tet2 knockout 1682 1695 GENOTYPE\n",
      "Tet2 knockout 1744 1757 GENOTYPE\n",
      "wild-type (WT) 445 459 GENOTYPE\n",
      "control 1295 1302 GENOTYPE\n",
      "KD 150 152 GENOTYPE\n",
      "KD 181 183 GENOTYPE\n",
      "KD 887 889 GENOTYPE\n",
      "KD 963 965 GENOTYPE\n",
      "WT 135 137 GENOTYPE\n",
      "WT 166 168 GENOTYPE\n",
      "WT 198 200 GENOTYPE\n",
      "WT 583 585 GENOTYPE\n",
      "WT 880 882 GENOTYPE\n",
      "WT 956 958 GENOTYPE\n",
      "hemogen KO 205 215 GENOTYPE\n",
      "hemogen KO 590 600 GENOTYPE\n",
      "WT 58 60 GENOTYPE\n",
      "WT 1079 1081 GENOTYPE\n",
      "control 139 146 GENOTYPE\n",
      "wild type 791 800 GENOTYPE\n",
      "WT 58 60 GENOTYPE\n",
      "WT 1427 1429 GENOTYPE\n",
      "WT 1478 1480 GENOTYPE\n",
      "KO 1439 1441 GENOTYPE\n",
      "KO 1490 1492 GENOTYPE\n",
      "Anp32b-KO 950 959 GENOTYPE\n",
      "WT 942 944 GENOTYPE\n",
      "Terc-/- 188 195 GENOTYPE\n",
      "Skiv2lf/fl;K14-Cre 132 150 GENOTYPE\n",
      "Skiv2l ctl 181 191 GENOTYPE\n",
      "Per2def 1612 1619 GENOTYPE\n",
      "Per2wt 1584 1590 GENOTYPE\n",
      "Acan-Cre+/Cre;Stat3fl/fl 1146 1170 GENOTYPE\n",
      "Stat3fl/fl 1121 1131 GENOTYPE\n",
      "/fl 1160 1170 GENOTYPE\n",
      "2C 2048 2050 GENOTYPE\n",
      "None 2058 2060 GENOTYPE\n",
      "Skipping entity\n",
      "None 2088 2090 GENOTYPE\n",
      "Skipping entity\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 87%|█████████████████████████████████▏    | 1573/1804 [00:05<00:00, 347.26it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "hIGFBP5 514 521 GENOTYPE\n",
      "wild-type 450 459 GENOTYPE\n",
      "WT 607 609 GENOTYPE\n",
      "WT 866 868 GENOTYPE\n",
      "WT 940 942 GENOTYPE\n",
      "WT 159 161 GENOTYPE\n",
      "WT 351 353 GENOTYPE\n",
      "WT 567 569 GENOTYPE\n",
      "SCD1-/- 1170 1177 GENOTYPE\n",
      "SCD1-/- 1268 1275 GENOTYPE\n",
      "SCD1-/- 1502 1509 GENOTYPE\n",
      "WT 1495 1497 GENOTYPE\n",
      "Dnah9 knockout 54 68 GENOTYPE\n",
      "Dnah9 knockout 195 209 GENOTYPE\n",
      "Hez 91 94 GENOTYPE\n",
      "WT 1093 1095 GENOTYPE\n",
      "Trem2 -/- 58 67 GENOTYPE\n",
      "Trem2 +/+ 34 43 GENOTYPE\n",
      "APOE3 87 92 GENOTYPE\n",
      "APOE2 80 85 GENOTYPE\n",
      "APOE2 256 261 GENOTYPE\n",
      "APOE4 98 103 GENOTYPE\n",
      "APOE4 266 271 GENOTYPE\n",
      "APOE2 80 85 GENOTYPE\n",
      "APOE2 352 357 GENOTYPE\n",
      "APOE3 87 92 GENOTYPE\n",
      "APOE4 98 103 GENOTYPE\n",
      "APOE4 362 367 GENOTYPE\n",
      "Glast-CreER:LNL-tTA:tetO-mAscl1-tetO-Atoh1-ires-GFP mice 2099 2155 GENOTYPE\n",
      "infected with pLOC-hAscl1-IRES-turboGFP virus 2005 2050 GENOTYPE\n",
      "infected with pLOC-hATOH7-IRES-copGFP virus 1906 1949 GENOTYPE\n",
      "infected with a lentivirus containing RFP 1803 1844 GENOTYPE\n",
      "None 535 539 GENOTYPE\n",
      "Skipping entity\n",
      "None 611 615 GENOTYPE\n",
      "Skipping entity\n",
      "Smc3 718 722 GENOTYPE\n",
      "None 879 883 GENOTYPE\n",
      "Skipping entity\n",
      "None 1004 1008 GENOTYPE\n",
      "Skipping entity\n",
      "None 1067 1071 GENOTYPE\n",
      "Skipping entity\n",
      "None 1155 1159 GENOTYPE\n",
      "Skipping entity\n",
      "Npm1 357 361 GENOTYPE\n",
      "None 518 522 GENOTYPE\n",
      "Skipping entity\n",
      "None 602 606 GENOTYPE\n",
      "Skipping entity\n",
      "None 662 666 GENOTYPE\n",
      "Skipping entity\n",
      "None 710 714 GENOTYPE\n",
      "Skipping entity\n",
      "None 870 874 GENOTYPE\n",
      "Skipping entity\n",
      "None 995 999 GENOTYPE\n",
      "Skipping entity\n",
      "None 1058 1062 GENOTYPE\n",
      "Skipping entity\n",
      "None 1079 1083 GENOTYPE\n",
      "Skipping entity\n",
      "None 1146 1150 GENOTYPE\n",
      "Skipping entity\n",
      "WT 102 104 GENOTYPE\n",
      "WT 498 500 GENOTYPE\n",
      "WT 635 637 GENOTYPE\n",
      "WT 696 698 GENOTYPE\n",
      "WT 755 757 GENOTYPE\n",
      "DYRK1B-Tg 504 513 GENOTYPE\n",
      "DYRK1B-Tg 556 565 GENOTYPE\n",
      "WT 362 364 GENOTYPE\n",
      "WT 489 491 GENOTYPE\n",
      "WT 541 543 GENOTYPE\n",
      "XY 65 67 GENOTYPE\n",
      "XY 454 456 GENOTYPE\n",
      "XY 557 559 GENOTYPE\n",
      "XY 630 632 GENOTYPE\n",
      "XY 765 767 GENOTYPE\n",
      "XO 58 60 GENOTYPE\n",
      "XO 494 496 GENOTYPE\n",
      "XO 586 588 GENOTYPE\n",
      "XO 681 683 GENOTYPE\n",
      "XO 758 760 GENOTYPE\n",
      "XX 54 56 GENOTYPE\n",
      "XX 478 480 GENOTYPE\n",
      "XX 580 582 GENOTYPE\n",
      "XX 675 677 GENOTYPE\n",
      "XX 755 757 GENOTYPE\n",
      "EsrrbKO 75 82 GENOTYPE\n",
      "EsrrbKO 283 290 GENOTYPE\n",
      "EsrrbKO 402 409 GENOTYPE\n",
      "EsrrbKO 571 578 GENOTYPE\n",
      "EsrrbKO 625 632 GENOTYPE\n",
      "WT 69 71 GENOTYPE\n",
      "WT 309 311 GENOTYPE\n",
      "WT 619 621 GENOTYPE\n",
      "IL6 KO 82 88 GENOTYPE\n",
      "WT 178 180 GENOTYPE\n",
      "-b mutant 95 107 GENOTYPE\n",
      "Dst-b mutant 183 195 GENOTYPE\n",
      "None 86 88 GENOTYPE\n",
      "Skipping entity\n",
      "None 173 175 GENOTYPE\n",
      "Skipping entity\n",
      "Dcir-/- 196 203 GENOTYPE\n",
      "Dcir-/- 508 515 GENOTYPE\n",
      "Dcir-/- 864 871 GENOTYPE\n",
      "p40-/-IL-2Ra+/- 146 161 GENOTYPE\n",
      "p40-/-IL-2Ra+/- 351 366 GENOTYPE\n",
      "WT 1265 1267 GENOTYPE\n",
      "CM 123 125 GENOTYPE\n",
      "CM 406 408 GENOTYPE\n",
      "CM 595 597 GENOTYPE\n",
      "CM 1209 1211 GENOTYPE\n",
      "CM 1226 1228 GENOTYPE\n",
      "DENR KO 56 63 GENOTYPE\n",
      "DENR KO 131 138 GENOTYPE\n",
      "DENR KO 277 284 GENOTYPE\n",
      "DENR KO 414 421 GENOTYPE\n",
      "anti-Act1 133 142 GENOTYPE\n",
      "anti-Act1 294 303 GENOTYPE\n",
      "anti-Act1 417 426 GENOTYPE\n",
      "anti-Act1 610 619 GENOTYPE\n",
      "anti-Act1 830 839 GENOTYPE\n",
      "anti-Act1 895 904 GENOTYPE\n",
      "anti-Act1 1051 1060 GENOTYPE\n",
      "CCRL2 KO 178 186 GENOTYPE\n",
      "CCRL2 WT 351 359 GENOTYPE\n",
      "Kras mutant 37 48 GENOTYPE\n",
      "normal 24 30 GENOTYPE\n",
      "Kras mutant 35 46 GENOTYPE\n",
      "5xFAD 202 207 GENOTYPE\n",
      "miR-21fl/flmTmG/Myh11ERT2 338 363 GENOTYPE\n",
      "miR-21fl/flmTmG/Myh11ERT2 838 863 GENOTYPE\n",
      "miR-21fl/flmTmG/Myh11ERT2 1850 1875 GENOTYPE\n",
      "miR-21fl/flmTmG/Myh11ERT2 425 450 GENOTYPE\n",
      "miR-21fl/flmTmG/Myh11ERT2 891 916 GENOTYPE\n",
      "miR-21fl/flmTmG/Myh11ERT2 1829 1854 GENOTYPE\n",
      "BMAd-Pnpla2-/- 277 291 GENOTYPE\n",
      "BMAd-Pnpla2-/- 1317 1331 GENOTYPE\n",
      "knockout 186 194 GENOTYPE\n",
      "wild type 243 252 GENOTYPE\n",
      "Abcb11-/- 136 145 GENOTYPE\n",
      "Abcb11-/- 410 419 GENOTYPE\n",
      "Cpt1a L-/- 1569 1579 GENOTYPE\n",
      "WT 947 949 GENOTYPE\n",
      "WT 293 295 GENOTYPE\n",
      "Pax7-CreER;ROSA26YFP;Drp1fl/fl 346 376 GENOTYPE\n",
      "Pax7- 346 366 GENOTYPE\n",
      "Myc KO 429 435 GENOTYPE\n",
      "Arid1a KO 507 516 GENOTYPE\n",
      "wild type 1932 1941 GENOTYPE\n",
      "wild type 1995 2004 GENOTYPE\n",
      "MafA-S64F heterozygous 1489 1511 GENOTYPE\n",
      "MafA-S64F heterozygous 1540 1562 GENOTYPE\n",
      "Il10-/- 484 491 GENOTYPE\n",
      "Il10-/- 535 542 GENOTYPE\n",
      "Pear1-/- 968 976 GENOTYPE\n",
      "Pear1-/- 1242 1250 GENOTYPE\n",
      "WT 2593 2595 GENOTYPE\n",
      "WT 2752 2754 GENOTYPE\n",
      "WT 2950 2952 GENOTYPE\n",
      "LysMDCommd10 1983 1995 GENOTYPE\n",
      "Commd10fl/fl 2026 2038 GENOTYPE\n",
      "LysMDCommd10 2472 2484 GENOTYPE\n",
      "Commd10fl/fl 2515 2527 GENOTYPE\n",
      "Commd10fl/fl 1809 1821 GENOTYPE\n",
      "Commd10fl/fl 2224 2236 GENOTYPE\n",
      "WT 1140 1142 GENOTYPE\n",
      "Dars2 KO 1147 1155 GENOTYPE\n",
      "Dnmt3a KO 42 51 GENOTYPE\n",
      "Dnmt3a KO 186 195 GENOTYPE\n",
      "Dnmt3a KO 375 384 GENOTYPE\n",
      "Dnmt3a KO 547 556 GENOTYPE\n",
      "Dnmt3a KO 750 759 GENOTYPE\n",
      "WT 179 181 GENOTYPE\n",
      "WT 389 391 GENOTYPE\n",
      "WT 540 542 GENOTYPE\n",
      "WT 743 745 GENOTYPE\n",
      "Mx1Cre; Jak2VF/VF 1073 1090 GENOTYPE\n",
      "renal ischemia reperfusion 96 122 GENOTYPE\n",
      "renal ischemia reperfusion 226 252 GENOTYPE\n",
      "wild-type 753 762 GENOTYPE\n",
      "wild-type 1433 1442 GENOTYPE\n",
      "Adap-KO 1609 1616 GENOTYPE\n",
      "ApoE KO 1295 1302 GENOTYPE\n",
      "Pitx2 mutant 576 588 GENOTYPE\n",
      "Pitx2 mutant 961 973 GENOTYPE\n",
      "Pitx2 mutant 1444 1456 GENOTYPE\n",
      "Pitx2 mutant 2020 2032 GENOTYPE\n",
      "Sidt1/Sidt2 double knock- 387 415 GENOTYPE\n",
      "Il1rn-/- 47 55 GENOTYPE\n",
      "Il1rn-/- 125 133 GENOTYPE\n",
      "Il1rn-/- 259 267 GENOTYPE\n",
      "Il1rn-/- 698 706 GENOTYPE\n",
      "Il1rn-/- 981 989 GENOTYPE\n",
      "Il1rn-/- 1307 1315 GENOTYPE\n",
      "Il1rn-/- 1418 1426 GENOTYPE\n",
      "Il1rn-/- 1556 1564 GENOTYPE\n",
      "Il1rn-/- 1736 1744 GENOTYPE\n",
      "wild type 1717 1726 GENOTYPE\n",
      "wild type 829 838 GENOTYPE\n",
      "rd10 644 648 GENOTYPE\n",
      "rd10 775 779 GENOTYPE\n",
      "wild type 1100 1109 GENOTYPE\n",
      "None 475 478 GENOTYPE\n",
      "Skipping entity\n",
      "rd1 479 482 GENOTYPE\n",
      "rd1 484 487 GENOTYPE\n",
      "rd1 640 643 GENOTYPE\n",
      "rd1 774 777 GENOTYPE\n",
      "rd1 1323 1326 GENOTYPE\n",
      "rd1 1740 1743 GENOTYPE\n",
      "None 1772 1775 GENOTYPE\n",
      "Skipping entity\n",
      "rd1 1776 1779 GENOTYPE\n",
      "L1 KO 354 359 GENOTYPE\n",
      "WT 393 395 GENOTYPE\n",
      "p53 wild-type (Albumin-Cre; p53 WT) 268 303 GENOTYPE\n",
      "p53-deficient (Albumin-Cre; p53FL/FL) 314 351 GENOTYPE\n",
      "None 549 553 GENOTYPE\n",
      "Skipping entity\n",
      "wild type 530 539 GENOTYPE\n",
      "BMKO 405 409 GENOTYPE\n",
      "BMCtr 395 400 GENOTYPE\n",
      "prostate-specific Pten-/- 86 111 GENOTYPE\n",
      "prostate-specific Pten-/- 401 426 GENOTYPE\n",
      "Rbpms KO 1331 1339 GENOTYPE\n",
      "Rbpms KO 1418 1426 GENOTYPE\n",
      "WT 1325 1327 GENOTYPE\n",
      "WT 1525 1527 GENOTYPE\n",
      "Sftpc-GFP 215 224 GENOTYPE\n",
      "LysM-cre, Mettl3fl/fl 226 247 GENOTYPE\n",
      "LysM-cre, Mettl3fl/fl 366 387 GENOTYPE\n",
      "Mettl3fl/fl 205 216 GENOTYPE\n",
      "Mettl3fl/fl 236 247 GENOTYPE\n",
      "Mettl3fl/fl 350 361 GENOTYPE\n",
      "Mettl3fl/fl 376 387 GENOTYPE\n",
      "LysM-cre, Mettl3fl/fl 200 221 GENOTYPE\n",
      "LysM-cre, Mettl3fl/fl 314 335 GENOTYPE\n",
      "Mettl3fl/fl 179 190 GENOTYPE\n",
      "Mettl3fl/fl 210 221 GENOTYPE\n",
      "Mettl3fl/fl 298 309 GENOTYPE\n",
      "Mettl3fl/fl 324 335 GENOTYPE\n",
      "Egln1/Hif2aTie2Cre mice 87 110 GENOTYPE\n",
      "Egln1/Hif2aTie2Cre mice 204 227 GENOTYPE\n",
      "wildtype 193 201 GENOTYPE\n",
      "Siah2 KO 129 137 GENOTYPE\n",
      "CHD8+/- 632 639 GENOTYPE\n",
      "wild type 618 627 GENOTYPE\n",
      "Gpr65_WT 52 60 GENOTYPE\n",
      "Gpr65_WT 114 122 GENOTYPE\n",
      "Gpr65_WT 202 210 GENOTYPE\n",
      "Lats1/2 double conditional knock out (Lats1 flox/flox; Lats2 flox/flox; E2f1 Tg(Wnt1-cre)2Sor/+, DCKO) 1434 1536 GENOTYPE\n",
      "control (no E2f1 Tg(Wnt1-cre)2Sor/+) 1545 1581 GENOTYPE\n",
      "wildtype 120 128 GENOTYPE\n",
      "NOD.Prdm1 flox/flox 133 152 GENOTYPE\n",
      "NOD.Prdm1 flox/ 158 177 GENOTYPE\n",
      "NOD.Prdm1 flox/flox 558 577 GENOTYPE\n",
      "NOD.Prdm1 flox/ 583 602 GENOTYPE\n",
      "NOD.CD4-Cre+ 116 128 GENOTYPE\n",
      "NOD.CD4-Cre+ 541 553 GENOTYPE\n",
      "NOD.Prdm1 flox/floxCD4-Cre+ 158 185 GENOTYPE\n",
      "NOD.Prdm1 flox/floxCD4-Cre+ 583 610 GENOTYPE\n",
      "control 1628 1635 GENOTYPE\n",
      "control 1402 1409 GENOTYPE\n",
      "control 1687 1694 GENOTYPE\n",
      "None 1772 1779 GENOTYPE\n",
      "Skipping entity\n",
      "control 2079 2086 GENOTYPE\n",
      "control 2140 2147 GENOTYPE\n",
      "control 2370 2377 GENOTYPE\n",
      "Smg6-cKO 295 303 GENOTYPE\n",
      "Smg6-cKO 1025 1033 GENOTYPE\n",
      "Smg6-cKO 1517 1525 GENOTYPE\n",
      "Smg6-cKO 2066 2074 GENOTYPE\n",
      "Smg6-cKO 2166 2174 GENOTYPE\n",
      "Smg6-cKO 2348 2356 GENOTYPE\n",
      "WT 97 99 GENOTYPE\n",
      "WT 216 218 GENOTYPE\n",
      "Bmi1 conditional know-out in B cells 203 239 GENOTYPE\n",
      "KB1P 1128 1132 GENOTYPE\n",
      "KP 1124 1126 GENOTYPE\n",
      "KB2P 1138 1142 GENOTYPE\n",
      "Neat1KO 1498 1505 GENOTYPE\n",
      "Pspc1KO 1486 1493 GENOTYPE\n",
      "Baf180 shRNAs 151 164 GENOTYPE\n",
      "Baf180 shRNAs 485 498 GENOTYPE\n",
      "wildtype 220 228 GENOTYPE\n",
      "FOXA2Venus 1334 1344 GENOTYPE\n",
      "FOXA2Venus 1530 1540 GENOTYPE\n",
      "WT 161 163 GENOTYPE\n",
      "WT 346 348 GENOTYPE\n",
      "WT 449 451 GENOTYPE\n",
      "WT 187 189 GENOTYPE\n",
      "WT 409 411 GENOTYPE\n",
      "H2b/Xbp1fl/fl 347 360 GENOTYPE\n",
      "WT 990 992 GENOTYPE\n",
      "WT 1151 1153 GENOTYPE\n",
      "lncRNA-GM-/- 145 157 GENOTYPE\n",
      "wild type 126 135 GENOTYPE\n",
      "knocked down 491 503 GENOTYPE\n",
      "WT 89 91 GENOTYPE\n",
      "WT 116 118 GENOTYPE\n",
      "WT 285 287 GENOTYPE\n",
      "Cd34-CreERT2;R26-tdTomato 621 646 GENOTYPE\n",
      "wildtype 668 676 GENOTYPE\n",
      "wildtype 1950 1958 GENOTYPE\n",
      "PS19(P301S) 197 208 GENOTYPE\n",
      "(C57BL/6 x C3H) 236 251 GENOTYPE\n",
      "wild-type 1448 1457 GENOTYPE\n",
      "Grem1-deleted 53 66 GENOTYPE\n",
      "Grem1-deleted 397 410 GENOTYPE\n",
      "MttpIKO 173 180 GENOTYPE\n",
      "WT 602 604 GENOTYPE\n",
      "Btn1a1-/- 95 104 GENOTYPE\n",
      "None 262 271 GENOTYPE\n",
      "Skipping entity\n",
      "Ifnar1-SA 39 48 GENOTYPE\n",
      "Ifnar1-SA 162 171 GENOTYPE\n",
      "WT 23 25 GENOTYPE\n",
      "WT 154 156 GENOTYPE\n",
      "wild type 841 850 GENOTYPE\n",
      "wild type 969 978 GENOTYPE\n",
      "wild type 1271 1280 GENOTYPE\n",
      "B6 1703 1705 GENOTYPE\n",
      "Il5-tg 1693 1699 GENOTYPE\n",
      "WT 1161 1163 GENOTYPE\n",
      "WT 1259 1261 GENOTYPE\n",
      "DhhCre;Nf1fl/fl 36 51 GENOTYPE\n",
      "/fl 43 51 GENOTYPE\n",
      "WT 645 647 GENOTYPE\n",
      "Dpy30-KO 345 353 GENOTYPE\n",
      "Dpy30-KO 1200 1208 GENOTYPE\n",
      "CTRL 1212 1216 GENOTYPE\n",
      "WT 1266 1268 GENOTYPE\n",
      "Rag2-/- 1273 1280 GENOTYPE\n",
      "Odc_Delta_Epi 89 102 GENOTYPE\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 91%|██████████████████████████████████▋   | 1645/1804 [00:05<00:00, 342.53it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "B6 control 249 259 GENOTYPE\n",
      "wild type 645 654 GENOTYPE\n",
      "Dsg2-W2A homozygouse mutant 664 691 GENOTYPE\n",
      "Y334C MT 2163 2171 GENOTYPE\n",
      "C57BL6/J 241 249 GENOTYPE\n",
      "C57BL6/J 301 309 GENOTYPE\n",
      "TauPS2APP 189 198 GENOTYPE\n",
      "TauPS2APP 885 894 GENOTYPE\n",
      "PS2APP 141 147 GENOTYPE\n",
      "None 192 198 GENOTYPE\n",
      "Skipping entity\n",
      "PS2APP 867 873 GENOTYPE\n",
      "None 888 894 GENOTYPE\n",
      "Skipping entity\n",
      "CTCF-K20R 388 397 GENOTYPE\n",
      "wild type 374 383 GENOTYPE\n",
      "Parva-fl/fl*hNPHS2Cre 43 64 GENOTYPE\n",
      "Parva-fl/fl*hNPHS2Cre 381 402 GENOTYPE\n",
      "Parva-fl/ 43 54 GENOTYPE\n",
      "Parva-fl/ 381 392 GENOTYPE\n",
      "WT 170 172 GENOTYPE\n",
      "WT 221 223 GENOTYPE\n",
      "RUNX1-RUNX1T1 94 107 GENOTYPE\n",
      "WT 1659 1661 GENOTYPE\n",
      "HSC-Spred1 wt CML mice 2550 2572 GENOTYPE\n",
      "Spred1 wt CML mice 2554 2572 GENOTYPE\n",
      "Spred1 wt CML mice 2628 2646 GENOTYPE\n",
      "Spred1 wt CML mice 2703 2721 GENOTYPE\n",
      "wild type 1025 1034 GENOTYPE\n",
      "ARF-null 278 286 GENOTYPE\n",
      "wild type 260 269 GENOTYPE\n",
      "wild type 339 348 GENOTYPE\n",
      "wild type 434 443 GENOTYPE\n",
      "NRAS G12D 406 415 GENOTYPE\n",
      "wild type 1130 1139 GENOTYPE\n",
      "wild type 4831 4840 GENOTYPE\n",
      "NOX4-/- 214 221 GENOTYPE\n",
      "WT 47 49 GENOTYPE\n",
      "WT 383 385 GENOTYPE\n",
      "WT mESCs 681 689 GENOTYPE\n",
      "DKO 509 512 GENOTYPE\n",
      "DKO 743 746 GENOTYPE\n",
      "DKO 1011 1014 GENOTYPE\n",
      "R6/2 70 74 GENOTYPE\n",
      "R6/2 232 236 GENOTYPE\n",
      "zQ175 94 99 GENOTYPE\n",
      "zQ175 256 261 GENOTYPE\n",
      "EPHA2 null 1684 1694 GENOTYPE\n",
      "wild type 1668 1677 GENOTYPE\n",
      "wildtype 215 223 GENOTYPE\n",
      "wildtype 419 427 GENOTYPE\n",
      "St18(-/-) 432 441 GENOTYPE\n",
      "None 2049 2056 GENOTYPE\n",
      "Skipping entity\n",
      "None 2287 2294 GENOTYPE\n",
      "Skipping entity\n",
      "None 2442 2449 GENOTYPE\n",
      "Skipping entity\n",
      "None 2576 2583 GENOTYPE\n",
      "Skipping entity\n",
      "None 2808 2815 GENOTYPE\n",
      "Skipping entity\n",
      "Zfp280c knock-out 581 598 GENOTYPE\n",
      "SCA7 31 35 GENOTYPE\n",
      "SCA7 137 141 GENOTYPE\n",
      "SCA7 221 225 GENOTYPE\n",
      "SCA7 344 348 GENOTYPE\n",
      "SCA7 693 697 GENOTYPE\n",
      "SCA7 711 715 GENOTYPE\n",
      "SCA7 1058 1062 GENOTYPE\n",
      "SCA7 1182 1186 GENOTYPE\n",
      "None 1209 1213 GENOTYPE\n",
      "Skipping entity\n",
      "SCA7 1266 1270 GENOTYPE\n",
      "SCA7 1367 1371 GENOTYPE\n",
      "SCA7 1539 1543 GENOTYPE\n",
      "SCA7 2277 2281 GENOTYPE\n",
      "SCA7 2450 2454 GENOTYPE\n",
      "None 509 516 GENOTYPE\n",
      "Skipping entity\n",
      "TLR9P915H 310 319 GENOTYPE\n",
      "TLR9P915H 855 864 GENOTYPE\n",
      "TLR9P915H 1000 1009 GENOTYPE\n",
      "TLR9WT 869 875 GENOTYPE\n",
      "control 306 313 GENOTYPE\n",
      "Nme1+/- / Nme2+/- 1437 1454 GENOTYPE\n",
      "Vegfr2 knockout 92 107 GENOTYPE\n",
      "Vegfr2 knockout 236 251 GENOTYPE\n",
      "control 68 75 GENOTYPE\n",
      "control 212 219 GENOTYPE\n",
      "WT 288 290 GENOTYPE\n",
      "Vsir-/- 313 320 GENOTYPE\n",
      "Vsir-/- 1428 1435 GENOTYPE\n",
      "AdC3KO 1201 1207 GENOTYPE\n",
      "WT 1157 1159 GENOTYPE\n",
      "wild type 970 979 GENOTYPE\n",
      "Mettl3-conditional knockout 238 265 GENOTYPE\n",
      "WT 215 217 GENOTYPE\n",
      "Apoe4 265 270 GENOTYPE\n",
      "Apoe3 305 310 GENOTYPE\n",
      "wild type 311 320 GENOTYPE\n",
      "wild type 1092 1101 GENOTYPE\n",
      "WT 737 739 GENOTYPE\n",
      "ATXN1[30Q]D776/Cck-/- 1220 1241 GENOTYPE\n",
      "None 613 619 GENOTYPE\n",
      "Skipping entity\n",
      "Cck-/- 1209 1215 GENOTYPE\n",
      "Cck-/- 1235 1241 GENOTYPE\n",
      "KrasG12D/+Lkb1fl/ 667 686 GENOTYPE\n",
      "KrasG12D/+ 653 663 GENOTYPE\n",
      "None 667 677 GENOTYPE\n",
      "Skipping entity\n",
      "CD4-Cre 62 69 GENOTYPE\n",
      "CD4- 90 97 GENOTYPE\n",
      "CD4- 125 132 GENOTYPE\n",
      "CD4-Cre 153 160 GENOTYPE\n",
      "CD4- 1049 1056 GENOTYPE\n",
      "CD4-Cre 1069 1076 GENOTYPE\n",
      "CD4-CrexBcl6fl/+ 90 106 GENOTYPE\n",
      "CD4-CrexBcl6fl/+ 125 141 GENOTYPE\n",
      "CD4-CrexBcl6fl/+ 1049 1065 GENOTYPE\n",
      "wild type 1019 1028 GENOTYPE\n",
      "WT 1519 1521 GENOTYPE\n",
      "None 1512 1514 GENOTYPE\n",
      "Skipping entity\n",
      "WT 1519 1521 GENOTYPE\n",
      "None 1512 1514 GENOTYPE\n",
      "Skipping entity\n",
      "Ren1c-/-;Myh11-CreERT2;tdTomato;Ren1c-YFP mice 151 197 GENOTYPE\n",
      "wild-type 482 491 GENOTYPE\n",
      "Cox10-/- 506 514 GENOTYPE\n",
      "Sox9 KO 295 302 GENOTYPE\n",
      "Sox9 KO 618 625 GENOTYPE\n",
      "Sox9 KO 689 696 GENOTYPE\n",
      "WT 608 610 GENOTYPE\n",
      "WT 1268 1270 GENOTYPE\n",
      "Cat-Tg 595 601 GENOTYPE\n",
      "Cat-Tg 658 664 GENOTYPE\n",
      "Cat-Tg 704 710 GENOTYPE\n",
      "Cat-Tg 765 771 GENOTYPE\n",
      "-Tg 1534 1540 GENOTYPE\n",
      "Cat-Tg 1590 1596 GENOTYPE\n",
      "ApoE-/- 291 298 GENOTYPE\n",
      "ApoE-/- 2241 2248 GENOTYPE\n",
      "delEx44 1382 1389 GENOTYPE\n",
      "delEx44 1405 1412 GENOTYPE\n",
      "delEx44 1522 1529 GENOTYPE\n",
      "delEx44 1545 1552 GENOTYPE\n",
      "WT 1378 1380 GENOTYPE\n",
      "WT 1518 1520 GENOTYPE\n",
      "wildtype 691 699 GENOTYPE\n",
      "Wild Type 1435 1444 GENOTYPE\n",
      "TNFR2 KO 1912 1920 GENOTYPE\n",
      "KO 309 311 GENOTYPE\n",
      "Fmo5(-/-) 167 176 GENOTYPE\n",
      "WT 1452 1454 GENOTYPE\n",
      "Cdh1-/- 618 625 GENOTYPE\n",
      "Cdh1-/- 770 777 GENOTYPE\n",
      "Cdh1-/- 1004 1011 GENOTYPE\n",
      "Cdh1-/- 1469 1476 GENOTYPE\n",
      "None 551 553 GENOTYPE\n",
      "Skipping entity\n",
      "None 558 560 GENOTYPE\n",
      "Skipping entity\n",
      "NOD.AIRE GW/+ 1219 1232 GENOTYPE\n",
      "wildtype 2353 2361 GENOTYPE\n",
      "Mx1-Cre x Adam10f/f Rag2KO 409 435 GENOTYPE\n",
      "None 474 477 GENOTYPE\n",
      "Skipping entity\n",
      "wild type 104 113 GENOTYPE\n",
      "wild type 294 303 GENOTYPE\n",
      "Ldlr-/- 116 123 GENOTYPE\n",
      "Ldlr-/- 323 330 GENOTYPE\n",
      "Apoe-/- 129 136 GENOTYPE\n",
      "Apoe-/- 310 317 GENOTYPE\n",
      "Plcg2+/+ 1691 1699 GENOTYPE\n",
      "Plcg2-/- 1170 1178 GENOTYPE\n",
      "Plcg2-/- 1704 1712 GENOTYPE\n",
      "wild-type 2201 2210 GENOTYPE\n",
      "IL13TG 2220 2226 GENOTYPE\n",
      "sgPrkci 1171 1178 GENOTYPE\n",
      "APP/PS1 230 237 GENOTYPE\n",
      "APP/PS1 330 337 GENOTYPE\n",
      "APP/PS1 339 346 GENOTYPE\n",
      "APP/PS1/Hras-/- 339 354 GENOTYPE\n",
      "(C57BL/6 x C3H) 238 253 GENOTYPE\n",
      "PS19(P301S) 199 210 GENOTYPE\n",
      "Mettl3fl/fl (WT) 131 147 GENOTYPE\n",
      "Mettl3fl/fl (WT) 598 614 GENOTYPE\n",
      "Ncr1Cre-Mettl3fl/fl 101 120 GENOTYPE\n",
      "Ncr1Cre-Mettl3fl/fl 541 560 GENOTYPE\n",
      "WT 1626 1628 GENOTYPE\n",
      "OTULIN-KO 339 348 GENOTYPE\n",
      "WT 350 352 GENOTYPE\n",
      "wild-type 1427 1436 GENOTYPE\n",
      "ApoE-/- 65 72 GENOTYPE\n",
      "ApoE-/- 241 248 GENOTYPE\n",
      "ApoE-/- 551 558 GENOTYPE\n",
      "ApoE-/- 1105 1112 GENOTYPE\n",
      "Pkd2-/- 2949 2956 GENOTYPE\n",
      "wild-type 2926 2935 GENOTYPE\n",
      "Pkd1-/- 2937 2944 GENOTYPE\n",
      "Col4a3-/- 171 180 GENOTYPE\n",
      "Col4a3-/- 267 276 GENOTYPE\n",
      "Col4a3-/- 286 295 GENOTYPE\n",
      "Col4a3-/- 321 330 GENOTYPE\n",
      "WT 113 115 GENOTYPE\n",
      "WT 261 263 GENOTYPE\n",
      "Surf1+/+ 426 434 GENOTYPE\n",
      "Surf1-/- 439 447 GENOTYPE\n",
      "p75 floxed mice (p75fl/fl) 1256 1282 GENOTYPE\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 97%|████████████████████████████████████▉ | 1755/1804 [00:05<00:00, 347.16it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "WT 467 469 GENOTYPE\n",
      "WT 518 520 GENOTYPE\n",
      "WT 856 858 GENOTYPE\n",
      "Siglec-H +/+ 394 406 GENOTYPE\n",
      "Siglec-H -/- 423 435 GENOTYPE\n",
      "p53- 296 300 GENOTYPE\n",
      "p53- 883 887 GENOTYPE\n",
      "p53- 1578 1582 GENOTYPE\n",
      "p53- 1735 1739 GENOTYPE\n",
      "p53+ 334 338 GENOTYPE\n",
      "p53+ 438 442 GENOTYPE\n",
      "p53+ 677 681 GENOTYPE\n",
      "p53+ 1033 1037 GENOTYPE\n",
      "p53+ 1294 1298 GENOTYPE\n",
      "p53+ 1616 1620 GENOTYPE\n",
      "p53+ 1768 1772 GENOTYPE\n",
      "WT 1135 1137 GENOTYPE\n",
      "Cre 4 7 GENOTYPE\n",
      "Cre 147 150 GENOTYPE\n",
      "Cre 195 198 GENOTYPE\n",
      "Cre 260 263 GENOTYPE\n",
      "Cre 304 307 GENOTYPE\n",
      "Cre 439 442 GENOTYPE\n",
      "None 476 479 GENOTYPE\n",
      "Skipping entity\n",
      "None 616 619 GENOTYPE\n",
      "Skipping entity\n",
      "None 858 861 GENOTYPE\n",
      "Skipping entity\n",
      "None 975 978 GENOTYPE\n",
      "Skipping entity\n",
      "Cre 1040 1043 GENOTYPE\n",
      "None 1167 1170 GENOTYPE\n",
      "Skipping entity\n",
      "None 1172 1175 GENOTYPE\n",
      "Skipping entity\n",
      "WT 1125 1127 GENOTYPE\n",
      "Cre 4 7 GENOTYPE\n",
      "Cre 147 150 GENOTYPE\n",
      "Cre 195 198 GENOTYPE\n",
      "Cre 260 263 GENOTYPE\n",
      "Cre 304 307 GENOTYPE\n",
      "None 460 463 GENOTYPE\n",
      "Skipping entity\n",
      "Cre 532 535 GENOTYPE\n",
      "None 746 749 GENOTYPE\n",
      "Skipping entity\n",
      "Cre 1030 1033 GENOTYPE\n",
      "None 1164 1167 GENOTYPE\n",
      "Skipping entity\n",
      "shControl 1255 1264 GENOTYPE\n",
      "WT 105 107 GENOTYPE\n",
      "WT 215 217 GENOTYPE\n",
      "WT 1344 1346 GENOTYPE\n",
      "CH 1357 1359 GENOTYPE\n",
      "CH 1369 1371 GENOTYPE\n",
      "Parental 1616 1624 GENOTYPE\n",
      "WT 617 619 GENOTYPE\n",
      "WT 715 717 GENOTYPE\n",
      "BAG3cKO 907 914 GENOTYPE\n",
      "BAG3cKO 1078 1085 GENOTYPE\n",
      "None 1782 1789 GENOTYPE\n",
      "Skipping entity\n",
      "None 1799 1806 GENOTYPE\n",
      "Skipping entity\n",
      "None 1774 1776 GENOTYPE\n",
      "Skipping entity\n",
      "Elavl4 -/- (HuD KO) 1181 1200 GENOTYPE\n",
      "Npas3+/- 153 161 GENOTYPE\n",
      "Npas3+/- 227 235 GENOTYPE\n",
      "Npas3+/- 577 585 GENOTYPE\n",
      "Npas3+/- 627 635 GENOTYPE\n",
      "Npas3+/- 742 750 GENOTYPE\n",
      "Npas3-/- 286 294 GENOTYPE\n",
      "Npas3-/- 572 580 GENOTYPE\n",
      "None 599 606 GENOTYPE\n",
      "Skipping entity\n",
      "Ino80WT 1514 1521 GENOTYPE\n",
      "Ino80cKO 212 220 GENOTYPE\n",
      "Ino80cKO 453 461 GENOTYPE\n",
      "Ino80cKO 927 935 GENOTYPE\n",
      "Ino80cKO 1256 1264 GENOTYPE\n",
      "Ino80cKO 1526 1534 GENOTYPE\n",
      "WT 385 387 GENOTYPE\n",
      "WT 711 713 GENOTYPE\n",
      "RetDVav1 1406 1414 GENOTYPE\n",
      "RetMEN2B 1416 1424 GENOTYPE\n",
      "WT 1251 1253 GENOTYPE\n",
      "wild-type 236 245 GENOTYPE\n",
      "PTEN T cell-specific knockout 265 294 GENOTYPE\n",
      "Mex3a-/- 1386 1394 GENOTYPE\n",
      "wild type 94 103 GENOTYPE\n",
      "wild type 157 166 GENOTYPE\n",
      "wild type 658 667 GENOTYPE\n",
      "wild type 1004 1013 GENOTYPE\n",
      "Brd4-/- 108 115 GENOTYPE\n",
      "Brd4-/- 171 178 GENOTYPE\n",
      "Brd4-/- 672 679 GENOTYPE\n",
      "Brd4-/- 1023 1030 GENOTYPE\n",
      "wild type (WT) 342 356 GENOTYPE\n",
      "Nsd2 Treg conditional knock out 361 392 GENOTYPE\n",
      "WT 865 867 GENOTYPE\n",
      "WT 1264 1266 GENOTYPE\n",
      "WT 520 522 GENOTYPE\n",
      "Whsc1 -/- 528 537 GENOTYPE\n",
      "WT 61 63 GENOTYPE\n",
      "WT 145 147 GENOTYPE\n",
      "Stim1R304W/+ 149 161 GENOTYPE\n",
      "Tnmd WT ScxGFP+ 1563 1578 GENOTYPE\n",
      "Tnmd-/- ScxGFP+ 1543 1558 GENOTYPE\n",
      "WT 168 170 GENOTYPE\n",
      "WT 188 190 GENOTYPE\n",
      "LIRKO 193 198 GENOTYPE\n",
      "LIRKO 201 206 GENOTYPE\n",
      "WT 632 634 GENOTYPE\n",
      "Ahr HKO 639 646 GENOTYPE\n",
      "ASXL1-MT-KI 202 213 GENOTYPE\n",
      "ASXL1-MT-KI 338 349 GENOTYPE\n",
      "WT 372 374 GENOTYPE\n",
      "I{mu}BCL6 346 355 GENOTYPE\n",
      "wild type 1017 1026 GENOTYPE\n",
      "wild type 417 426 GENOTYPE\n",
      "control 1156 1163 GENOTYPE\n",
      "LAMA4 knockout 208 222 GENOTYPE\n",
      "LAMA4 knockout 362 376 GENOTYPE\n",
      "LAMA4 knockout 480 494 GENOTYPE\n",
      "Btg1 wild-type 297 311 GENOTYPE\n",
      "Btg1 wild-type 840 854 GENOTYPE\n",
      "Btg1 knockout 315 328 GENOTYPE\n",
      "Btg1 knockout 404 417 GENOTYPE\n",
      "Btg1 knockout 613 626 GENOTYPE\n",
      "Btg1 knockout 691 704 GENOTYPE\n",
      "Btg1 knockout 858 871 GENOTYPE\n",
      "St3gal1 flox/flox Pf4/cre 1887 1912 GENOTYPE\n",
      "St3gal1 flox/flox Pf4/ 2060 2085 GENOTYPE\n",
      "St3gal1 flox/flox 1887 1904 GENOTYPE\n",
      "St3gal1 flox/flox 1917 1934 GENOTYPE\n",
      "St3gal1 flox/flox 2030 2047 GENOTYPE\n",
      "St3gal1 flox/flox 2060 2077 GENOTYPE\n",
      "16p11.2+/- 87 97 GENOTYPE\n",
      "16p11.2+/- 141 151 GENOTYPE\n",
      "heterozygous (fro/+) 884 904 GENOTYPE\n",
      "Gli1_Cre_ERT2/WT; Rosa_mTmG/WT mouse 88 124 GENOTYPE\n",
      "RIPK1D325A/D325A; RIPK3-/- 1350 1376 GENOTYPE\n",
      "RIPK1D325A/D325A; RIPK3-/- 1453 1479 GENOTYPE\n",
      "knockout 133 141 GENOTYPE\n",
      "knockout 203 211 GENOTYPE\n",
      "WT 1147 1149 GENOTYPE\n",
      "WT 1231 1233 GENOTYPE\n",
      "CXCR4C1013G 179 190 GENOTYPE\n",
      "G 805 816 GENOTYPE\n",
      "CXCR4C1013G 1151 1162 GENOTYPE\n",
      "G 1184 1195 GENOTYPE\n",
      "CXCR4C1013G 1235 1246 GENOTYPE\n",
      "G 1265 1276 GENOTYPE\n",
      "Rspo1 GOF 321 330 GENOTYPE\n",
      "control 337 344 GENOTYPE\n",
      "control 431 438 GENOTYPE\n",
      "control 1621 1628 GENOTYPE\n",
      "control 1637 1644 GENOTYPE\n",
      "miR182 overexpression 133 154 GENOTYPE\n",
      "miR182 overexpression 210 231 GENOTYPE\n",
      "WT 99 101 GENOTYPE\n",
      "WT 690 692 GENOTYPE\n",
      "NF-kB1 deficient mice 703 724 GENOTYPE\n",
      "NF-kB1 deficient mice 893 914 GENOTYPE\n",
      "CaMK-Cre;Aki2fl/fl 261 279 GENOTYPE\n",
      "CaMK-Cre;Aki2fl/fl 686 704 GENOTYPE\n",
      "DKO 282 285 GENOTYPE\n",
      "DKO 407 410 GENOTYPE\n",
      "DKO 524 527 GENOTYPE\n",
      "DKO 582 585 GENOTYPE\n",
      "DKO 688 691 GENOTYPE\n",
      "DKO 1019 1022 GENOTYPE\n",
      "DKO 1382 1385 GENOTYPE\n",
      "DKO 1411 1414 GENOTYPE\n",
      "DKO 1431 1434 GENOTYPE\n",
      "DKO 1618 1621 GENOTYPE\n",
      "WT 330 332 GENOTYPE\n",
      "WT 467 469 GENOTYPE\n",
      "WT 528 530 GENOTYPE\n",
      "WT 1013 1015 GENOTYPE\n",
      "WT 1375 1377 GENOTYPE\n",
      "WT 1404 1406 GENOTYPE\n",
      "WT 1424 1426 GENOTYPE\n",
      "WT 1626 1628 GENOTYPE\n",
      "WT 1374 1376 GENOTYPE\n",
      "TET KO 1378 1384 GENOTYPE\n",
      "WT 1504 1506 GENOTYPE\n",
      "Auts2-/- 1511 1519 GENOTYPE\n",
      "SOD1(G93A) 2585 2595 GENOTYPE\n",
      "WT 51 53 GENOTYPE\n",
      "WT 282 284 GENOTYPE\n",
      "HAT1 WT 239 246 GENOTYPE\n",
      "wild type (ctrl) 860 876 GENOTYPE\n",
      "Ift140-/-(cko) 881 895 GENOTYPE\n",
      "Foxo1f/f 894 902 GENOTYPE\n",
      "Foxo1f/f 916 924 GENOTYPE\n",
      "Wnt7acre 894 902 GENOTYPE\n",
      "None 904 912 GENOTYPE\n",
      "Skipping entity\n",
      "None 927 935 GENOTYPE\n",
      "Skipping entity\n",
      "WT 25 27 GENOTYPE\n",
      "WT 163 165 GENOTYPE\n",
      "KO 18 20 GENOTYPE\n",
      "KO 156 158 GENOTYPE\n",
      "Mrgpra2 dKO 547 558 GENOTYPE\n",
      "Def cKO 407 414 GENOTYPE\n",
      "Def cKO 1145 1152 GENOTYPE\n",
      "WT 640 642 GENOTYPE\n",
      "WT 1141 1143 GENOTYPE\n",
      "wild type 48 57 GENOTYPE\n",
      "15Q 362 365 GENOTYPE\n",
      "15Q 760 763 GENOTYPE\n",
      "84Q 159 162 GENOTYPE\n",
      "None 376 379 GENOTYPE\n",
      "Skipping entity\n",
      "None 395 398 GENOTYPE\n",
      "Skipping entity\n",
      "84Q 464 467 GENOTYPE\n",
      "84Q 798 801 GENOTYPE\n",
      "84Q 831 834 GENOTYPE\n",
      "Asgr1 knockout 275 289 GENOTYPE\n",
      "QKO 655 658 GENOTYPE\n",
      "QKO 1371 1374 GENOTYPE\n",
      "QKO 1477 1480 GENOTYPE\n",
      "QKO 1989 1992 GENOTYPE\n",
      "None 1925 1932 GENOTYPE\n",
      "Skipping entity\n",
      "MIRKO 409 414 GENOTYPE\n",
      "MIRKO 1239 1244 GENOTYPE\n",
      "MIRKO 1457 1462 GENOTYPE\n",
      "MIRKO 1975 1980 GENOTYPE\n",
      "MIGF1RKO 1264 1272 GENOTYPE\n",
      "Qlox 1998 2002 GENOTYPE\n",
      "MIGIRKO 447 454 GENOTYPE\n",
      "MIGIRKO 842 849 GENOTYPE\n",
      "MIGIRKO 1303 1310 GENOTYPE\n",
      "MIGIRKO 1464 1471 GENOTYPE\n",
      "MIGIRKO 1911 1918 GENOTYPE\n",
      "IRIGFRlox 1923 1932 GENOTYPE\n",
      "IRlox 1982 1987 GENOTYPE\n",
      "Nmnat1 fl/fl 1171 1183 GENOTYPE\n",
      "Nmnat1-/- 1138 1147 GENOTYPE\n",
      "WT 1663 1665 GENOTYPE\n",
      "Cnp-/- 1087 1093 GENOTYPE\n",
      "Cnp-/- 1667 1673 GENOTYPE\n",
      "None 1682 1688 GENOTYPE\n",
      "Skipping entity\n",
      "5xFAD 1675 1680 GENOTYPE\n",
      "None 1689 1694 GENOTYPE\n",
      "Skipping entity\n",
      "Cnp-/- 1180 1186 GENOTYPE\n",
      "WT 984 986 GENOTYPE\n",
      "WT 1367 1369 GENOTYPE\n",
      "WT 2449 2451 GENOTYPE\n",
      "WT 2832 2834 GENOTYPE\n",
      "TLR4 -/- 23 31 GENOTYPE\n",
      "TLR4 -/- 324 332 GENOTYPE\n",
      "wild type 390 399 GENOTYPE\n",
      "TnnT2Cre;Wt1flox 409 425 GENOTYPE\n",
      "tKO 227 230 GENOTYPE\n",
      "WT 223 225 GENOTYPE\n",
      "nKO 232 235 GENOTYPE\n",
      "aKO 241 244 GENOTYPE\n",
      "Foxf1 S52F heterozygous 70 93 GENOTYPE\n",
      "Ptf1a cre/+ 203 214 GENOTYPE\n",
      "Ptf1a 251 262 GENOTYPE\n",
      "Ptf1a cre/+;CAG-loxP-STOP-loxP-Prox1-LacZ 251 292 GENOTYPE\n",
      "wildtype 170 178 GENOTYPE\n",
      "Prox1-nullyzygous 85 102 GENOTYPE\n",
      "Prox1-nullyzygous 184 201 GENOTYPE\n",
      "wild type 656 665 GENOTYPE\n",
      "wild type 1620 1629 GENOTYPE\n",
      "restf/f 1253 1260 GENOTYPE\n",
      "Prdm1fl/flFoxP3YFP-Cre (KO) 474 501 GENOTYPE\n",
      "FoxP3YFP-Cre (WT) 506 523 GENOTYPE\n",
      "wild-type control (C57Bl/6) 181 208 GENOTYPE\n",
      "Apln-/- 105 112 GENOTYPE\n",
      "Apln-/- 212 219 GENOTYPE\n",
      "None 277 284 GENOTYPE\n",
      "Skipping entity\n",
      "Eldr cKI 310 318 GENOTYPE\n",
      "Eldr cKO 310 318 GENOTYPE\n",
      "None 277 284 GENOTYPE\n",
      "Skipping entity\n",
      "control 1244 1251 GENOTYPE\n",
      "Restf/f PR+/Cre 1220 1235 GENOTYPE\n",
      "IL-2-KO 747 754 GENOTYPE\n",
      "IL-2-KO 1142 1149 GENOTYPE\n",
      "IL-2-KO 1190 1197 GENOTYPE\n",
      "Rpl10a +/+ 1252 1262 GENOTYPE\n",
      "Rpl10a LOF/LOF 1267 1281 GENOTYPE\n",
      "(knockout) 264 282 GENOTYPE\n",
      "Mll1-/- (knockout) 1085 1103 GENOTYPE\n",
      "(control) 212 229 GENOTYPE\n",
      "Mll1+/- (control) 1046 1063 GENOTYPE\n",
      "Lifrfl/fl 195 204 GENOTYPE\n",
      "Lifrfl/ 209 218 GENOTYPE\n",
      "Lifrfl/fl;Crealb/alb 209 229 GENOTYPE\n",
      "WT 1218 1220 GENOTYPE\n",
      "Fabp3-null 1225 1235 GENOTYPE\n",
      "Wildtype 1864 1872 GENOTYPE\n",
      "EC-Myc KO 579 588 GENOTYPE\n",
      "EC-Myc KO 735 744 GENOTYPE\n",
      "EC-Myc KO 1622 1631 GENOTYPE\n",
      "Cyfip2 CKO 872 882 GENOTYPE\n",
      "Cyfip2 CKO 983 993 GENOTYPE\n",
      "Cyfip2 CKO 1114 1124 GENOTYPE\n",
      "Cyfip2 CKO 1258 1268 GENOTYPE\n",
      "Cyfip2 CKO 1836 1846 GENOTYPE\n",
      "wild type 198 207 GENOTYPE\n",
      "Tgfbr2-/- 254 263 GENOTYPE\n",
      "Tgfbr2-/- 302 311 GENOTYPE\n",
      "wildtype 62 70 GENOTYPE\n",
      "wildtype 236 244 GENOTYPE\n",
      "wildtype 356 364 GENOTYPE\n",
      "wild type 332 341 GENOTYPE\n",
      "wild type 760 769 GENOTYPE\n",
      "Arid1a (f/f) cre- 1094 1111 GENOTYPE\n",
      "Arid1a (f/f) cre+ 1072 1089 GENOTYPE\n",
      "WT 471 473 GENOTYPE\n",
      "WT 170 172 GENOTYPE\n",
      "Ddb1-KO 158 165 GENOTYPE\n",
      "dKO 1064 1067 GENOTYPE\n",
      "WT 1054 1056 GENOTYPE\n",
      "Nude 1058 1062 GENOTYPE\n",
      "Foxp3GFP 1709 1717 GENOTYPE\n",
      "Foxp3CNS1- 1722 1732 GENOTYPE\n",
      "Alb-Cre 489 496 GENOTYPE\n",
      "Alb-R3 526 532 GENOTYPE\n",
      "Tcfl5+/+ 423 431 GENOTYPE\n",
      "Tcfl5+/+ 654 662 GENOTYPE\n",
      "Tcfl5+/- 436 444 GENOTYPE\n",
      "Tcfl5+/- 667 675 GENOTYPE\n",
      "overexpressing p140Cap 1382 1404 GENOTYPE\n",
      "Stat1/Stat2 dKO 620 635 GENOTYPE\n",
      "Stat1/Stat2 dKO 691 706 GENOTYPE\n",
      "Nlrc5 KO 664 672 GENOTYPE\n",
      "Ahi1-/- 326 333 GENOTYPE\n",
      "Ahi1-/- 522 529 GENOTYPE\n",
      "WT 94 96 GENOTYPE\n",
      "Sox2 T258A mutant 327 344 GENOTYPE\n",
      "WT 70 72 GENOTYPE\n",
      "Sox2 T258A mutant 331 348 GENOTYPE\n",
      "Clp1+/+ 1751 1758 GENOTYPE\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "100%|██████████████████████████████████████| 1804/1804 [00:06<00:00, 297.52it/s]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "KC 706 708 GENOTYPE\n",
      "KC 834 836 GENOTYPE\n",
      "KC 902 904 GENOTYPE\n",
      "KC 1165 1167 GENOTYPE\n",
      "KC 1177 1179 GENOTYPE\n",
      "KC 1375 1377 GENOTYPE\n",
      "KC 1393 1395 GENOTYPE\n",
      "KC 2053 2055 GENOTYPE\n",
      "KC 2070 2072 GENOTYPE\n",
      "KC/PD 2070 2075 GENOTYPE\n",
      "Dusp1-/- 665 673 GENOTYPE\n",
      "Dusp1-/- 723 731 GENOTYPE\n",
      "Dusp1-/- 932 940 GENOTYPE\n",
      "Dusp1-/- 1469 1477 GENOTYPE\n",
      "wild type 1450 1459 GENOTYPE\n",
      "p53 wt 1237 1243 GENOTYPE\n",
      "wild-type 216 225 GENOTYPE\n",
      "wild-type 279 288 GENOTYPE\n",
      "wild-type 697 706 GENOTYPE\n",
      "Dusp6 knockout 711 725 GENOTYPE\n",
      "Nix-/-Bnip3-/- 1161 1175 GENOTYPE\n",
      "WT 706 708 GENOTYPE\n",
      "Alk5iEKO 1083 1091 GENOTYPE\n",
      "wild type 1358 1367 GENOTYPE\n",
      "MAVS -/- 350 358 GENOTYPE\n",
      "WT 84 86 GENOTYPE\n",
      "wild type 921 930 GENOTYPE\n",
      "HdhQ7/Q111 (KI) 2308 2323 GENOTYPE\n",
      "HdhQ7/7 (wild-type) 2284 2303 GENOTYPE\n",
      "Ino80 cKO 1568 1577 GENOTYPE\n",
      "control 400 407 GENOTYPE\n",
      "control 1429 1436 GENOTYPE\n",
      "control 347 354 GENOTYPE\n",
      "None 782 789 GENOTYPE\n",
      "Skipping entity\n",
      "control 890 897 GENOTYPE\n",
      "Acsl6-/- 902 910 GENOTYPE\n",
      "Adcy10-/- 504 513 GENOTYPE\n",
      "Adcy10-/- 767 776 GENOTYPE\n",
      "Adcy10-/- 1452 1461 GENOTYPE\n",
      "WT 368 370 GENOTYPE\n",
      "WT 489 491 GENOTYPE\n",
      "WT 533 535 GENOTYPE\n",
      "Gclc KO 629 636 GENOTYPE\n",
      "Gclc KO 1702 1709 GENOTYPE\n",
      "Gclc KO 1762 1769 GENOTYPE\n",
      "Gclc KO 1891 1898 GENOTYPE\n",
      "Gclc KO 1974 1981 GENOTYPE\n",
      "Gclc KO 2445 2452 GENOTYPE\n",
      "WT 132 134 GENOTYPE\n",
      "WT 250 252 GENOTYPE\n",
      "LIPTER KO 138 147 GENOTYPE\n"
     ]
    }
   ],
   "source": [
    "from spacy.util import filter_spans\n",
    "\n",
    "for training_example  in tqdm(training_data['annotations']): \n",
    "    text = training_example['text']\n",
    "    labels = training_example['entities']\n",
    "    doc = nlp.make_doc(text) \n",
    "    ents = []\n",
    "    for start, end, label in labels:\n",
    "        span = doc.char_span(start, end, label=label, alignment_mode=\"contract\")\n",
    "        if span is None:\n",
    "            print(\"Skipping entity\")\n",
    "        else:\n",
    "            ents.append(span)\n",
    "    filtered_ents = filter_spans(ents)\n",
    "    doc.ents = filtered_ents \n",
    "    doc_bin.add(doc)\n",
    "\n",
    "doc_bin.to_disk(\"training_data.spacy\") # save the docbin object"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "7873884f",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 99,
   "id": "962708d3",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(WT, LIPTER KO, WT)"
      ]
     },
     "execution_count": 99,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "doc.ents"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 115,
   "id": "a8acaa19",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Inhibitory control is a core cognitive function preventing the expression of maladaptive behaviors. This function is overridden in mental illnesses including bipolar disorder, autism spectrum disorders and schizophrenia, causing maladaptive psychomotor agitation. We developed intersectional CRISPR/Cas9 targeting approaches in adult mice to identify receptor/circuit selective mechanisms responsible for inactivation of inhibitory control in two models of amphetamine-induced psychomotor agitation. Inactivation of dopamine D2-receptors in the medial-prefrontal cortex (mPFC) abolished amphetamine-induced: psychomotor agitation, motor sensitization, and cAMP-associated transcriptome changes in the striatum. Further characterization using intersectional projection preparations indicate that these deficits implicate D2-receptors expressed on projection neurons innervating cholinergic interneurons of the dorsomedial striatum (DMS). Trans-synaptic targeting of acetylcholine production in DMS neurons receiving these projections restored psychomotor agitation in mice lacking mPFC D2- receptor expression. These findings identify a circuit by which activation of cortical D2-receptor circumvent an acetylcholine-mediated inhibition of DMS functions by the mPFC to promote psychomotor activation. Overall design: To investigate the impact of mPFC D2R deficit on striatal responses to amphetamine, we performed RNA-seq analysis using mice that D2R was targeting using CRISPR/Cas9 strategy and received an acute administration of amphetamine (3mg/kg, i.p). D2RCre_floxCas9-EGFP mice received virus injection in mPFC of AAV5 D2R-sgRNA (\n",
       "<mark class=\"entity\" style=\"background: #F67DE3; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PFC_D2R-KO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">GENOTYPE</span>\n",
       "</mark>\n",
       " mice) or AAV5 syn-mCherry (control mice). Treatment: 5 days of saline injection (ip) followed by a single injection of amphetamine (3mg/kg, ip) on the challenge day (day9). 4 groups: VV-\n",
       "<mark class=\"entity\" style=\"background: #F67DE3; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Control\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">GENOTYPE</span>\n",
       "</mark>\n",
       " = vehicle treatment in control mice; VA-\n",
       "<mark class=\"entity\" style=\"background: #F67DE3; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Control\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">GENOTYPE</span>\n",
       "</mark>\n",
       " = amphetamine treatment in control mice; VV-KO = vehicle treatment in \n",
       "<mark class=\"entity\" style=\"background: #F67DE3; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PFC_D2R-KO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">GENOTYPE</span>\n",
       "</mark>\n",
       " mice; VA-KO = amphetamine treatment in \n",
       "<mark class=\"entity\" style=\"background: #F67DE3; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PFC_D2R-KO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">GENOTYPE</span>\n",
       "</mark>\n",
       " mice.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "colors = {\"GENOTYPE\": \"#F67DE3\"}\n",
    "options = {\"colors\": colors}\n",
    "\n",
    "# Convert the generator object to a list\n",
    "doc_list = list(doc_bin.get_docs(nlp.vocab))\n",
    "doc = doc_list[9]\n",
    "\n",
    "spacy.displacy.render(doc, style=\"ent\", options= options, jupyter=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "230daca8",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
