{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 30,
   "id": "1b354873",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import numpy as np\n",
    "import torch\n",
    "from transformers import BertTokenizer, BertForTokenClassification\n",
    "from transformers import BertTokenizerFast\n",
    "from torch.utils.data import TensorDataset, DataLoader, RandomSampler, SequentialSampler\n",
    "from sklearn.metrics import f1_score\n",
    "import matplotlib.pyplot as plt\n",
    "from tensorflow.keras.preprocessing.sequence import pad_sequences\n",
    "\n",
    "\n",
    "if torch.cuda.is_available():\n",
    "    device = torch.device('cuda')\n",
    "else:\n",
    "    device = torch.device('cpu')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "id": "4b6227aa",
   "metadata": {},
   "outputs": [],
   "source": [
    "train_df = pd.read_pickle(\"../data/train.df\")\n",
    "test_df = pd.read_pickle(\"../data/test.df\")\n",
    "val_df = pd.read_pickle(\"../data/val.df\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "id": "84dd156e",
   "metadata": {},
   "outputs": [],
   "source": [
    "ignore_genotypes = ['WT', 'wt', 'Wt', 'wild type', 'control', 'KO',\n",
    "                    'wild-type', 'wildtype', 'Control', 'Wild type', \n",
    "                    'Wild-type', 'Wildtype', 'Wild Type', 'knockout']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "id": "6de48109",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Initialize the BERT tokenizer\n",
    "tokenizer = BertTokenizerFast.from_pretrained('microsoft/BiomedNLP-PubMedBERT-base-uncased-abstract')\n",
    "\n",
    "# Define the BIO labels\n",
    "BIO_LABELS = ['B-GENOTYPE', 'I-GENOTYPE', 'O']\n",
    "\n",
    "\n",
    "def split_list(lst, size):\n",
    "    return [lst[i:i+size] for i in range(0, len(lst), size)]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "id": "78af89ec",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Tokenize the text and do BIO labeling\n",
    "def offset_tokenization(df):\n",
    "    tokens = []\n",
    "    offsets = []\n",
    "    index_offsets = []\n",
    "    labels = []\n",
    "    token_label_pairs = []\n",
    "    for i, row in df.iterrows():\n",
    "        text = row['Description']\n",
    "        genotype = row['genotype']\n",
    "        indexes = row['index_dict']\n",
    "        train_encodings = tokenizer(text, return_offsets_mapping=True)\n",
    "        tokenized_text = tokenizer.tokenize(text)\n",
    "        token_labels = ['O'] * len(tokenized_text)\n",
    "        # Get the character level offsets\n",
    "        offsets = train_encodings.offset_mapping[1:-1]\n",
    "        for key, value in indexes.items():\n",
    "            if key not in ignore_genotypes:\n",
    "                for v in value:\n",
    "                    start = v[0]\n",
    "                    end = v[1]\n",
    "                    # Loop through each character level offset in the offsets\n",
    "                    for k in range(len(offsets)):\n",
    "                        offset_start = offsets[k][0]\n",
    "                        offset_end = offsets[k][1]\n",
    "                        if start <= offset_start and end >= offset_end:\n",
    "        #                     print(row['tokens'][k], offsets[k], start, offset_start, end, offset_end)\n",
    "                            if offset_start == start:\n",
    "                                token_labels[k] = 'B-GENOTYPE'\n",
    "                            else:\n",
    "                                token_labels[k] = 'I-GENOTYPE'\n",
    "        token_chunks = split_list(tokenized_text, 500)\n",
    "        tokens.extend(token_chunks)\n",
    "        label_chunks = split_list(token_labels, 500)\n",
    "        labels.extend(label_chunks)\n",
    "        token_label_pairs.extend([[(a, b) for a, b in zip(sublist_A, sublist_B)] for sublist_A, sublist_B in zip(token_chunks, label_chunks)])\n",
    "\n",
    "\n",
    "    # Create a new dataframe with tokenized text and BIO labels\n",
    "    new_df = pd.DataFrame({'tokens': tokens, 'labels': labels, 'token_label_pairs': token_label_pairs})\n",
    "    return new_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "id": "f05e8d33",
   "metadata": {},
   "outputs": [],
   "source": [
    "tok_train_df = offset_tokenization(train_df)\n",
    "tok_test_df = offset_tokenization(test_df)\n",
    "tok_val_df = offset_tokenization(val_df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "id": "30f1e2aa",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[('purpose', 'O'),\n",
       " (':', 'O'),\n",
       " ('next', 'O'),\n",
       " ('-', 'O'),\n",
       " ('generation', 'O'),\n",
       " ('sequencing', 'O'),\n",
       " ('(', 'O'),\n",
       " ('ngs', 'O'),\n",
       " (')', 'O'),\n",
       " ('has', 'O'),\n",
       " ('revolution', 'O'),\n",
       " ('##ized', 'O'),\n",
       " ('systems', 'O'),\n",
       " ('-', 'O'),\n",
       " ('based', 'O'),\n",
       " ('analysis', 'O'),\n",
       " ('of', 'O'),\n",
       " ('cellular', 'O'),\n",
       " ('pathways', 'O'),\n",
       " ('.', 'O'),\n",
       " ('the', 'O'),\n",
       " ('goals', 'O'),\n",
       " ('of', 'O'),\n",
       " ('this', 'O'),\n",
       " ('study', 'O'),\n",
       " ('are', 'O'),\n",
       " ('to', 'O'),\n",
       " ('compare', 'O'),\n",
       " ('transcriptome', 'O'),\n",
       " ('##s', 'O'),\n",
       " ('of', 'O'),\n",
       " ('sirt', 'O'),\n",
       " ('##3', 'O'),\n",
       " ('wt', 'O'),\n",
       " ('and', 'O'),\n",
       " ('kr', 'O'),\n",
       " ('macrophages', 'O'),\n",
       " ('with', 'O'),\n",
       " ('high', 'O'),\n",
       " ('-', 'O'),\n",
       " ('throughput', 'O'),\n",
       " ('data', 'O'),\n",
       " ('analysis', 'O'),\n",
       " ('methods', 'O'),\n",
       " (':', 'O'),\n",
       " ('macrophages', 'O'),\n",
       " (\"'\", 'O'),\n",
       " ('mrna', 'O'),\n",
       " ('profiles', 'O'),\n",
       " ('of', 'O'),\n",
       " ('8', 'O'),\n",
       " ('weeks', 'O'),\n",
       " ('-', 'O'),\n",
       " ('old', 'O'),\n",
       " ('wild', 'O'),\n",
       " ('-', 'O'),\n",
       " ('type', 'O'),\n",
       " ('(', 'O'),\n",
       " ('wt', 'O'),\n",
       " (')', 'O'),\n",
       " ('and', 'O'),\n",
       " ('sirt', 'O'),\n",
       " ('##3', 'O'),\n",
       " ('k2', 'O'),\n",
       " ('##23', 'O'),\n",
       " ('##r', 'O'),\n",
       " ('mice', 'O'),\n",
       " ('were', 'O'),\n",
       " ('generated', 'O'),\n",
       " ('by', 'O'),\n",
       " ('deep', 'O'),\n",
       " ('sequencing', 'O'),\n",
       " (',', 'O'),\n",
       " ('in', 'O'),\n",
       " ('quadrup', 'O'),\n",
       " ('##lic', 'O'),\n",
       " ('##ate', 'O'),\n",
       " (',', 'O'),\n",
       " ('using', 'O'),\n",
       " ('total', 'O'),\n",
       " ('rna', 'O'),\n",
       " ('from', 'O'),\n",
       " ('each', 'O'),\n",
       " ('sample', 'O'),\n",
       " ('was', 'O'),\n",
       " ('quantified', 'O'),\n",
       " ('using', 'O'),\n",
       " ('a', 'O'),\n",
       " ('nanod', 'O'),\n",
       " ('##rop', 'O'),\n",
       " ('nd', 'O'),\n",
       " ('-', 'O'),\n",
       " ('1000', 'O'),\n",
       " ('instrument', 'O'),\n",
       " ('.', 'O'),\n",
       " ('1', 'O'),\n",
       " ('##Î¼g', 'O'),\n",
       " ('total', 'O'),\n",
       " ('rna', 'O'),\n",
       " ('was', 'O'),\n",
       " ('used', 'O'),\n",
       " ('to', 'O'),\n",
       " ('prepare', 'O'),\n",
       " ('the', 'O'),\n",
       " ('sequencing', 'O'),\n",
       " ('library', 'O'),\n",
       " ('in', 'O'),\n",
       " ('the', 'O'),\n",
       " ('following', 'O'),\n",
       " ('steps', 'O'),\n",
       " (':', 'O'),\n",
       " ('1', 'O'),\n",
       " ('.', 'O'),\n",
       " ('total', 'O'),\n",
       " ('rna', 'O'),\n",
       " ('is', 'O'),\n",
       " ('enriched', 'O'),\n",
       " ('by', 'O'),\n",
       " ('oligo', 'O'),\n",
       " ('(', 'O'),\n",
       " ('dt', 'O'),\n",
       " (')', 'O'),\n",
       " ('magnetic', 'O'),\n",
       " ('beads', 'O'),\n",
       " ('(', 'O'),\n",
       " ('rrna', 'O'),\n",
       " ('removed', 'O'),\n",
       " (')', 'O'),\n",
       " (';', 'O'),\n",
       " ('2', 'O'),\n",
       " ('.', 'O'),\n",
       " ('rna', 'O'),\n",
       " ('-', 'O'),\n",
       " ('seq', 'O'),\n",
       " ('library', 'O'),\n",
       " ('preparation', 'O'),\n",
       " ('using', 'O'),\n",
       " ('kap', 'O'),\n",
       " ('##a', 'O'),\n",
       " ('stranded', 'O'),\n",
       " ('rna', 'O'),\n",
       " ('-', 'O'),\n",
       " ('seq', 'O'),\n",
       " ('library', 'O'),\n",
       " ('prep', 'O'),\n",
       " ('kit', 'O'),\n",
       " ('(', 'O'),\n",
       " ('illumina', 'O'),\n",
       " (')', 'O'),\n",
       " (',', 'O'),\n",
       " ('which', 'O'),\n",
       " ('incorporates', 'O'),\n",
       " ('dutp', 'O'),\n",
       " ('into', 'O'),\n",
       " ('the', 'O'),\n",
       " ('second', 'O'),\n",
       " ('cdna', 'O'),\n",
       " ('strand', 'O'),\n",
       " ('and', 'O'),\n",
       " ('renders', 'O'),\n",
       " ('the', 'O'),\n",
       " ('rna', 'O'),\n",
       " ('-', 'O'),\n",
       " ('seq', 'O'),\n",
       " ('library', 'O'),\n",
       " ('strand', 'O'),\n",
       " ('-', 'O'),\n",
       " ('specific', 'O'),\n",
       " ('.', 'O'),\n",
       " ('the', 'O'),\n",
       " ('completed', 'O'),\n",
       " ('libraries', 'O'),\n",
       " ('were', 'O'),\n",
       " ('qualified', 'O'),\n",
       " ('with', 'O'),\n",
       " ('ag', 'O'),\n",
       " ('##ile', 'O'),\n",
       " ('##nt', 'O'),\n",
       " ('210', 'O'),\n",
       " ('##0', 'O'),\n",
       " ('bio', 'O'),\n",
       " ('##analy', 'O'),\n",
       " ('##zer', 'O'),\n",
       " ('and', 'O'),\n",
       " ('quantified', 'O'),\n",
       " ('by', 'O'),\n",
       " ('absolute', 'O'),\n",
       " ('quantification', 'O'),\n",
       " ('qpcr', 'O'),\n",
       " ('method', 'O'),\n",
       " ('.', 'O'),\n",
       " ('to', 'O'),\n",
       " ('sequence', 'O'),\n",
       " ('the', 'O'),\n",
       " ('libraries', 'O'),\n",
       " ('on', 'O'),\n",
       " ('the', 'O'),\n",
       " ('illumina', 'O'),\n",
       " ('his', 'O'),\n",
       " ('##e', 'O'),\n",
       " ('##q', 'O'),\n",
       " ('4000', 'O'),\n",
       " ('instrument', 'O'),\n",
       " (',', 'O'),\n",
       " ('the', 'O'),\n",
       " ('barc', 'O'),\n",
       " ('##oded', 'O'),\n",
       " ('libraries', 'O'),\n",
       " ('were', 'O'),\n",
       " ('mixed', 'O'),\n",
       " (',', 'O'),\n",
       " ('denatured', 'O'),\n",
       " ('to', 'O'),\n",
       " ('single', 'O'),\n",
       " ('stranded', 'O'),\n",
       " ('dna', 'O'),\n",
       " ('in', 'O'),\n",
       " ('naoh', 'O'),\n",
       " (',', 'O'),\n",
       " ('captured', 'O'),\n",
       " ('on', 'O'),\n",
       " ('illumina', 'O'),\n",
       " ('flow', 'O'),\n",
       " ('cell', 'O'),\n",
       " (',', 'O'),\n",
       " ('amplified', 'O'),\n",
       " ('in', 'O'),\n",
       " ('situ', 'O'),\n",
       " (',', 'O'),\n",
       " ('and', 'O'),\n",
       " ('subsequently', 'O'),\n",
       " ('sequenced', 'O'),\n",
       " ('for', 'O'),\n",
       " ('150', 'O'),\n",
       " ('cycles', 'O'),\n",
       " ('for', 'O'),\n",
       " ('both', 'O'),\n",
       " ('ends', 'O'),\n",
       " ('on', 'O'),\n",
       " ('illumina', 'O'),\n",
       " ('his', 'O'),\n",
       " ('##e', 'O'),\n",
       " ('##q', 'O'),\n",
       " ('instrument', 'O'),\n",
       " ('.', 'O'),\n",
       " ('results', 'O'),\n",
       " (':', 'O'),\n",
       " ('image', 'O'),\n",
       " ('analysis', 'O'),\n",
       " ('and', 'O'),\n",
       " ('base', 'O'),\n",
       " ('calling', 'O'),\n",
       " ('were', 'O'),\n",
       " ('performed', 'O'),\n",
       " ('using', 'O'),\n",
       " ('sole', 'O'),\n",
       " ('##xa', 'O'),\n",
       " ('pipeline', 'O'),\n",
       " ('v1', 'O'),\n",
       " ('.', 'O'),\n",
       " ('8', 'O'),\n",
       " ('(', 'O'),\n",
       " ('off', 'O'),\n",
       " ('-', 'O'),\n",
       " ('line', 'O'),\n",
       " ('base', 'O'),\n",
       " ('call', 'O'),\n",
       " ('##er', 'O'),\n",
       " ('software', 'O'),\n",
       " (',', 'O'),\n",
       " ('v1', 'O'),\n",
       " ('.', 'O'),\n",
       " ('8', 'O'),\n",
       " (')', 'O'),\n",
       " ('.', 'O'),\n",
       " ('sequence', 'O'),\n",
       " ('quality', 'O'),\n",
       " ('was', 'O'),\n",
       " ('examined', 'O'),\n",
       " ('using', 'O'),\n",
       " ('the', 'O'),\n",
       " ('fast', 'O'),\n",
       " ('##qc', 'O'),\n",
       " ('software', 'O'),\n",
       " ('.', 'O'),\n",
       " ('the', 'O'),\n",
       " ('trim', 'O'),\n",
       " ('##med', 'O'),\n",
       " ('reads', 'O'),\n",
       " ('(', 'O'),\n",
       " ('trim', 'O'),\n",
       " ('##med', 'O'),\n",
       " ('5', 'O'),\n",
       " ('â', 'O'),\n",
       " (',', 'O'),\n",
       " ('3', 'O'),\n",
       " ('â', 'O'),\n",
       " ('-', 'O'),\n",
       " ('adaptor', 'O'),\n",
       " ('bases', 'O'),\n",
       " ('using', 'O'),\n",
       " ('cut', 'O'),\n",
       " ('##adap', 'O'),\n",
       " ('##t', 'O'),\n",
       " (')', 'O'),\n",
       " ('were', 'O'),\n",
       " ('aligned', 'O'),\n",
       " ('to', 'O'),\n",
       " ('reference', 'O'),\n",
       " ('genome', 'O'),\n",
       " ('using', 'O'),\n",
       " ('his', 'O'),\n",
       " ('##at', 'O'),\n",
       " ('##2', 'O'),\n",
       " ('software', 'O'),\n",
       " ('.', 'O'),\n",
       " ('the', 'O'),\n",
       " ('transcript', 'O'),\n",
       " ('abundances', 'O'),\n",
       " ('for', 'O'),\n",
       " ('each', 'O'),\n",
       " ('sample', 'O'),\n",
       " ('was', 'O'),\n",
       " ('estimated', 'O'),\n",
       " ('with', 'O'),\n",
       " ('string', 'O'),\n",
       " ('##ti', 'O'),\n",
       " ('##e', 'O'),\n",
       " (',', 'O'),\n",
       " ('and', 'O'),\n",
       " ('the', 'O'),\n",
       " ('fp', 'O'),\n",
       " ('##k', 'O'),\n",
       " ('##m', 'O'),\n",
       " ('value', 'O'),\n",
       " ('for', 'O'),\n",
       " ('gene', 'O'),\n",
       " ('and', 'O'),\n",
       " ('transcript', 'O'),\n",
       " ('level', 'O'),\n",
       " ('were', 'O'),\n",
       " ('calculated', 'O'),\n",
       " ('with', 'O'),\n",
       " ('r', 'O'),\n",
       " ('package', 'O'),\n",
       " ('ball', 'O'),\n",
       " ('##gow', 'O'),\n",
       " ('##n', 'O'),\n",
       " ('.', 'O'),\n",
       " ('the', 'O'),\n",
       " ('differentially', 'O'),\n",
       " ('expressed', 'O'),\n",
       " ('genes', 'O'),\n",
       " ('and', 'O'),\n",
       " ('transcripts', 'O'),\n",
       " ('were', 'O'),\n",
       " ('filtered', 'O'),\n",
       " ('using', 'O'),\n",
       " ('r', 'O'),\n",
       " ('package', 'O'),\n",
       " ('ball', 'O'),\n",
       " ('##gow', 'O'),\n",
       " ('##n', 'O'),\n",
       " ('.', 'O'),\n",
       " ('the', 'O'),\n",
       " ('novel', 'O'),\n",
       " ('genes', 'O'),\n",
       " ('and', 'O'),\n",
       " ('transcripts', 'O'),\n",
       " ('were', 'O'),\n",
       " ('predicted', 'O'),\n",
       " ('from', 'O'),\n",
       " ('assembled', 'O'),\n",
       " ('results', 'O'),\n",
       " ('by', 'O'),\n",
       " ('comparing', 'O'),\n",
       " ('to', 'O'),\n",
       " ('the', 'O'),\n",
       " ('reference', 'O'),\n",
       " ('annotation', 'O'),\n",
       " ('using', 'O'),\n",
       " ('string', 'O'),\n",
       " ('##ti', 'O'),\n",
       " ('##e', 'O'),\n",
       " ('and', 'O'),\n",
       " ('ball', 'O'),\n",
       " ('##gow', 'O'),\n",
       " ('##n', 'O'),\n",
       " (',', 'O'),\n",
       " ('then', 'O'),\n",
       " ('use', 'O'),\n",
       " ('cpa', 'O'),\n",
       " ('##t', 'O'),\n",
       " ('to', 'O'),\n",
       " ('assess', 'O'),\n",
       " ('the', 'O'),\n",
       " ('coding', 'O'),\n",
       " ('potential', 'O'),\n",
       " ('of', 'O'),\n",
       " ('those', 'O'),\n",
       " ('sequences', 'O'),\n",
       " ('.', 'O'),\n",
       " ('then', 'O'),\n",
       " ('use', 'O'),\n",
       " ('rm', 'O'),\n",
       " ('##ats', 'O'),\n",
       " ('to', 'O'),\n",
       " ('detecting', 'O'),\n",
       " ('alternative', 'O'),\n",
       " ('splicing', 'O'),\n",
       " ('events', 'O'),\n",
       " ('and', 'O'),\n",
       " ('plots', 'O'),\n",
       " ('.', 'O'),\n",
       " ('principle', 'O'),\n",
       " ('component', 'O'),\n",
       " ('analysis', 'O'),\n",
       " ('(', 'O'),\n",
       " ('pca', 'O'),\n",
       " (')', 'O'),\n",
       " ('and', 'O'),\n",
       " ('correlation', 'O'),\n",
       " ('analysis', 'O'),\n",
       " ('were', 'O'),\n",
       " ('based', 'O'),\n",
       " ('on', 'O'),\n",
       " ('gene', 'O'),\n",
       " ('expression', 'O'),\n",
       " ('level', 'O'),\n",
       " (',', 'O'),\n",
       " ('hierarchical', 'O'),\n",
       " ('clustering', 'O'),\n",
       " (',', 'O'),\n",
       " ('gene', 'O'),\n",
       " ('ontology', 'O'),\n",
       " (',', 'O'),\n",
       " ('pathway', 'O'),\n",
       " ('analysis', 'O'),\n",
       " (',', 'O'),\n",
       " ('gene', 'O'),\n",
       " ('ontology', 'O'),\n",
       " (',', 'O'),\n",
       " ('pathway', 'O'),\n",
       " ('analysis', 'O'),\n",
       " (',', 'O'),\n",
       " ('scatter', 'O'),\n",
       " ('plots', 'O'),\n",
       " ('and', 'O'),\n",
       " ('vol', 'O'),\n",
       " ('##can', 'O'),\n",
       " ('##o', 'O'),\n",
       " ('plots', 'O'),\n",
       " ('were', 'O'),\n",
       " ('performed', 'O'),\n",
       " ('with', 'O'),\n",
       " ('the', 'O'),\n",
       " ('differentially', 'O'),\n",
       " ('expressed', 'O'),\n",
       " ('genes', 'O'),\n",
       " ('in', 'O'),\n",
       " ('r', 'O'),\n",
       " (',', 'O'),\n",
       " ('py', 'O'),\n",
       " ('##th', 'O'),\n",
       " ('##on', 'O'),\n",
       " ('or', 'O'),\n",
       " ('shell', 'O'),\n",
       " ('environment', 'O'),\n",
       " ('for', 'O'),\n",
       " ('statistical', 'O'),\n",
       " ('computing', 'O'),\n",
       " ('and', 'O'),\n",
       " ('graph', 'O'),\n",
       " ('##ics', 'O'),\n",
       " ('.', 'O'),\n",
       " ('conclusions', 'O'),\n",
       " (':', 'O'),\n",
       " ('our', 'O'),\n",
       " ('study', 'O'),\n",
       " ('represents', 'O'),\n",
       " ('the', 'O'),\n",
       " ('first', 'O'),\n",
       " ('detailed', 'O'),\n",
       " ('analysis', 'O'),\n",
       " ('of', 'O'),\n",
       " ('retinal', 'O'),\n",
       " ('transcriptome', 'O'),\n",
       " ('##s', 'O'),\n",
       " (',', 'O'),\n",
       " ('with', 'O'),\n",
       " ('4', 'O'),\n",
       " ('biologic', 'O'),\n",
       " ('replicates', 'O'),\n",
       " (',', 'O'),\n",
       " ('generated', 'O'),\n",
       " ('by', 'O'),\n",
       " ('rna', 'O'),\n",
       " ('-', 'O'),\n",
       " ('seq', 'O'),\n",
       " ('technology', 'O')]"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "tok_train_df.iloc[2].token_label_pairs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "id": "9abc143d",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array(['wild type', 'Sirt3K223R'], dtype=object)"
      ]
     },
     "execution_count": 44,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "train_df.iloc[2].genotype"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "id": "e2e9b2a2",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>tokens</th>\n",
       "      <th>labels</th>\n",
       "      <th>token_label_pairs</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>[we, performed, rnase, ##q, experiments, to, e...</td>\n",
       "      <td>[O, O, O, O, O, O, O, O, O, O, O, O, O, O, O, ...</td>\n",
       "      <td>[(we, O), (performed, O), (rnase, O), (##q, O)...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>[lin, ##28, ##b, suppresses, ml, ##l, -, enl, ...</td>\n",
       "      <td>[O, O, O, O, O, O, O, O, O, O, O, O, O, O, O, ...</td>\n",
       "      <td>[(lin, O), (##28, O), (##b, O), (suppresses, O...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>[purpose, :, next, -, generation, sequencing, ...</td>\n",
       "      <td>[O, O, O, O, O, O, O, O, O, O, O, O, O, O, O, ...</td>\n",
       "      <td>[(purpose, O), (:, O), (next, O), (-, O), (gen...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>[., the, optimized, data, analysis, workflow, ...</td>\n",
       "      <td>[O, O, O, O, O, O, O, O, O, O, O, O, O, O, O, ...</td>\n",
       "      <td>[(., O), (the, O), (optimized, O), (data, O), ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>[stress, granule, and, inflammasome, assembly,...</td>\n",
       "      <td>[O, O, O, O, O, O, O, O, O, O, O, O, O, O, O, ...</td>\n",
       "      <td>[(stress, O), (granule, O), (and, O), (inflamm...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1116</th>\n",
       "      <td>[the, trem, ##2, -, dap, ##12, receptor, compl...</td>\n",
       "      <td>[O, O, O, O, O, O, O, O, O, O, O, O, O, O, O, ...</td>\n",
       "      <td>[(the, O), (trem, O), (##2, O), (-, O), (dap, ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1117</th>\n",
       "      <td>[the, striatum, is, the, main, input, structur...</td>\n",
       "      <td>[O, O, O, O, O, O, O, O, O, O, O, O, O, O, O, ...</td>\n",
       "      <td>[(the, O), (striatum, O), (is, O), (the, O), (...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1118</th>\n",
       "      <td>[/, +, mice, and, dl, ##x, ##5, /, 6, -, ci, #...</td>\n",
       "      <td>[I-GENOTYPE, I-GENOTYPE, O, O, B-GENOTYPE, I-G...</td>\n",
       "      <td>[(/, I-GENOTYPE), (+, I-GENOTYPE), (mice, O), ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1119</th>\n",
       "      <td>[this, study, was, performed, to, compare, the...</td>\n",
       "      <td>[O, O, O, O, O, O, O, O, O, O, O, O, O, O, O, ...</td>\n",
       "      <td>[(this, O), (study, O), (was, O), (performed, ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1120</th>\n",
       "      <td>[total, gallbladder, /, extrahepatic, duct, ti...</td>\n",
       "      <td>[O, O, O, O, O, O, O, O, B-GENOTYPE, I-GENOTYP...</td>\n",
       "      <td>[(total, O), (gallbladder, O), (/, O), (extrah...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1121 rows Ã 3 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                 tokens  \\\n",
       "0     [we, performed, rnase, ##q, experiments, to, e...   \n",
       "1     [lin, ##28, ##b, suppresses, ml, ##l, -, enl, ...   \n",
       "2     [purpose, :, next, -, generation, sequencing, ...   \n",
       "3     [., the, optimized, data, analysis, workflow, ...   \n",
       "4     [stress, granule, and, inflammasome, assembly,...   \n",
       "...                                                 ...   \n",
       "1116  [the, trem, ##2, -, dap, ##12, receptor, compl...   \n",
       "1117  [the, striatum, is, the, main, input, structur...   \n",
       "1118  [/, +, mice, and, dl, ##x, ##5, /, 6, -, ci, #...   \n",
       "1119  [this, study, was, performed, to, compare, the...   \n",
       "1120  [total, gallbladder, /, extrahepatic, duct, ti...   \n",
       "\n",
       "                                                 labels  \\\n",
       "0     [O, O, O, O, O, O, O, O, O, O, O, O, O, O, O, ...   \n",
       "1     [O, O, O, O, O, O, O, O, O, O, O, O, O, O, O, ...   \n",
       "2     [O, O, O, O, O, O, O, O, O, O, O, O, O, O, O, ...   \n",
       "3     [O, O, O, O, O, O, O, O, O, O, O, O, O, O, O, ...   \n",
       "4     [O, O, O, O, O, O, O, O, O, O, O, O, O, O, O, ...   \n",
       "...                                                 ...   \n",
       "1116  [O, O, O, O, O, O, O, O, O, O, O, O, O, O, O, ...   \n",
       "1117  [O, O, O, O, O, O, O, O, O, O, O, O, O, O, O, ...   \n",
       "1118  [I-GENOTYPE, I-GENOTYPE, O, O, B-GENOTYPE, I-G...   \n",
       "1119  [O, O, O, O, O, O, O, O, O, O, O, O, O, O, O, ...   \n",
       "1120  [O, O, O, O, O, O, O, O, B-GENOTYPE, I-GENOTYP...   \n",
       "\n",
       "                                      token_label_pairs  \n",
       "0     [(we, O), (performed, O), (rnase, O), (##q, O)...  \n",
       "1     [(lin, O), (##28, O), (##b, O), (suppresses, O...  \n",
       "2     [(purpose, O), (:, O), (next, O), (-, O), (gen...  \n",
       "3     [(., O), (the, O), (optimized, O), (data, O), ...  \n",
       "4     [(stress, O), (granule, O), (and, O), (inflamm...  \n",
       "...                                                 ...  \n",
       "1116  [(the, O), (trem, O), (##2, O), (-, O), (dap, ...  \n",
       "1117  [(the, O), (striatum, O), (is, O), (the, O), (...  \n",
       "1118  [(/, I-GENOTYPE), (+, I-GENOTYPE), (mice, O), ...  \n",
       "1119  [(this, O), (study, O), (was, O), (performed, ...  \n",
       "1120  [(total, O), (gallbladder, O), (/, O), (extrah...  \n",
       "\n",
       "[1121 rows x 3 columns]"
      ]
     },
     "execution_count": 45,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "tok_train_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "id": "f9348653",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Length of train dataframe: 1121\n",
      "Length of validation dataframe: 372\n",
      "Length of test dataframe: 369\n"
     ]
    }
   ],
   "source": [
    "print(\"Length of train dataframe:\", len(tok_train_df))\n",
    "print(\"Length of validation dataframe:\", len(tok_test_df))\n",
    "print(\"Length of test dataframe:\", len(tok_val_df))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "id": "8060dd04",
   "metadata": {},
   "outputs": [],
   "source": [
    "tok_train_df.to_pickle(\"../data/pubMed/offset/train.df\")\n",
    "tok_test_df.to_pickle(\"../data/pubMed/offset/test.df\")\n",
    "tok_val_df.to_pickle(\"../data/pubMed/offset/val.df\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "4af9fec5",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "2ceba341",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
